<!--lint disable awesome-contributing awesome-license awesome-list-item match-punctuation no-repeat-punctuation no-undefined-references awesome-spell-check-->
# Awesome Stars [![Awesome](https://awesome.re/badge.svg)](https://github.com/sindresorhus/awesome)

> A curated list of my GitHub stars! Generated by [starred](https://github.com/maguowei/starred).

## Contents

- [AMPL](#ampl)
- [Assembly](#assembly)
- [Batchfile](#batchfile)
- [C](#c)
- [C#](#c#)
- [C++](#c++)
- [CSS](#css)
- [CoffeeScript](#coffeescript)
- [Common Workflow Language](#common-workflow-language)
- [Cython](#cython)
- [D](#d)
- [Dart](#dart)
- [Dockerfile](#dockerfile)
- [EJS](#ejs)
- [GAP](#gap)
- [GDScript](#gdscript)
- [Go](#go)
- [Groovy](#groovy)
- [HTML](#html)
- [Haskell](#haskell)
- [Java](#java)
- [JavaScript](#javascript)
- [Julia](#julia)
- [Jupyter Notebook](#jupyter-notebook)
- [Kotlin](#kotlin)
- [M](#m)
- [MATLAB](#matlab)
- [Makefile](#makefile)
- [Nextflow](#nextflow)
- [Nim](#nim)
- [OpenEdge ABL](#openedge-abl)
- [Others](#others)
- [PHP](#php)
- [PLpgSQL](#plpgsql)
- [Pascal](#pascal)
- [Pep8](#pep8)
- [Perl](#perl)
- [PostScript](#postscript)
- [Python](#python)
- [R](#r)
- [Roff](#roff)
- [Ruby](#ruby)
- [Rust](#rust)
- [Shell](#shell)
- [Smalltalk](#smalltalk)
- [TeX](#tex)
- [TypeScript](#typescript)
- [UnrealScript](#unrealscript)
- [Vue](#vue)
- [WDL](#wdl)
- [sed](#sed)

## AMPL 

- [ablab/quast](https://github.com/ablab/quast) - Genome assembly evaluation tool
- [zm-git-dev/quast](https://github.com/zm-git-dev/quast) - Genome assembly evaluation tool

## Assembly 

- [nh13/pbgzip](https://github.com/nh13/pbgzip) - Parallel Block GZIP

## Batchfile 

- [zm-git-dev/ImageMagick](https://github.com/zm-git-dev/ImageMagick) - The Guide to ImageMagick

## C 

- [jesvedberg/Ancestry_HMM-S](https://github.com/jesvedberg/Ancestry_HMM-S) - Ancestry_HMM-S is program designed to infer adaptive introgression from population genomic data
- [JinfengChen/Scripts](https://github.com/JinfengChen/Scripts) - 
- [fenderglass/Flye](https://github.com/fenderglass/Flye) - De novo assembler for single molecule sequencing reads using repeat graphs
- [ZhihaoXie/Scripts](https://github.com/ZhihaoXie/Scripts) - 
- [divonlan/genozip](https://github.com/divonlan/genozip) - A modern compressor for genomic files (FASTQ, SAM/BAM/CRAM, VCF, FASTA, GFF/GTF/GVF, 23andMe...), up to 5x better than gzip and faster too
- [bpp/bpp](https://github.com/bpp/bpp) - Bayesian analysis of genomic sequence data under the multispecies coalescent model
- [multiz/multiz](https://github.com/multiz/multiz) - DNA multiple sequence aligner, official version from Penn State's Miller Lab
- [marbl/Winnowmap](https://github.com/marbl/Winnowmap) - Long read / genome alignment software
- [hillerlab/GenomeAlignmentTools](https://github.com/hillerlab/GenomeAlignmentTools) - Tools for improving the sensitivity and specificity of genome alignments
- [kiminsigne-ucla/Motif-Scan-Plus](https://github.com/kiminsigne-ucla/Motif-Scan-Plus) - 185 Project
- [dentearl/mafTools](https://github.com/dentearl/mafTools) - Bioinformatics tools for dealing with Multiple Alignment Format (MAF) files.
- [dfguan/purge_dups](https://github.com/dfguan/purge_dups) - haplotypic duplication identification tool
- [xmuyulab/spark-scRNASeq-Analysis](https://github.com/xmuyulab/spark-scRNASeq-Analysis) - use spark to improve scRNASeq analysis
- [chhylp123/BitMapperBS](https://github.com/chhylp123/BitMapperBS) - BitMapperBS: a fast and accurate read aligner for whole-genome bisulfite sequencing
- [arangrhie/T2T-Polish](https://github.com/arangrhie/T2T-Polish) - Evaluation and polishing workflows for T2T genome assemblies
- [ElementoLab/ChIPseeqer](https://github.com/ElementoLab/ChIPseeqer) - A comprehensive framework for the analysis of ChIP-seq data
- [stjude/ChIP-Seq](https://github.com/stjude/ChIP-Seq) - 
- [eblancoga/seqcode](https://github.com/eblancoga/seqcode) - Suite of command-line software for high-performance graphical analysis of ChIP-seq/RNA-seq/ATAC-seq data
- [deeptools/pyBigWig](https://github.com/deeptools/pyBigWig) - A python extension for quick access to bigWig and bigBed files
- [ruanjue/pgzf](https://github.com/ruanjue/pgzf) - Parallel blocked gzip file IO
- [xia-lab/Seq2Fun](https://github.com/xia-lab/Seq2Fun) - 
- [Shicheng-Guo/PileOMeth](https://github.com/Shicheng-Guo/PileOMeth) - A (mostly) universal methylation extractor for BS-seq experiments.
- [Shicheng-Guo/methpipe](https://github.com/Shicheng-Guo/methpipe) - A pipeline for analyzing DNA methylation data from bisulfite sequencing.
- [zm-git-dev/MPI-blastn](https://github.com/zm-git-dev/MPI-blastn) - A high performance parallel version of nucleotide BLAST+ for sequence search
- [zm-git-dev/Scripts-2](https://github.com/zm-git-dev/Scripts-2) - 
- [zm-git-dev/treebest](https://github.com/zm-git-dev/treebest) - TreeBeST: Tree Building guided by Species Tree
- [zm-git-dev/scanPAV](https://github.com/zm-git-dev/scanPAV) - Pipeline to detect PAVs (presence/absence variations) in genome comparison using whole genome alignment.
- [zm-git-dev/BatMeth2](https://github.com/zm-git-dev/BatMeth2) - BS-seq analysis pipeline
- [zm-git-dev/mbs_simulator](https://github.com/zm-git-dev/mbs_simulator) - A workflow to simulate and analyze mapping-by-sequencing data of different experimental setups
- [zm-git-dev/fxstat](https://github.com/zm-git-dev/fxstat) - Fast and easy stats for FASTA and FASTQ files
- [zm-git-dev/snp-sites](https://github.com/zm-git-dev/snp-sites) - Finds SNP sites from a multi-FASTA alignment file
- [zm-git-dev/GSAlign](https://github.com/zm-git-dev/GSAlign) - GSAlign: an ultra-fast sequence alignment algorithm for intra-species genome comparison
- [zm-git-dev/MAESTRO](https://github.com/zm-git-dev/MAESTRO) - Single-cell Transcriptome and Regulome Analysis Pipeline
- [zm-git-dev/pblat](https://github.com/zm-git-dev/pblat) - parallelized blat with multi-threads support
- [zm-git-dev/mixer](https://github.com/zm-git-dev/mixer) - Causal Mixture Model for GWAS summary statistics
- [zm-git-dev/vcf-split](https://github.com/zm-git-dev/vcf-split) - Split combined-sample VCF stream into single-sample VCF files
- [zm-git-dev/miniwfa](https://github.com/zm-git-dev/miniwfa) - A reimplementation of the WaveFront Alignment algorithm at low memory
- [zm-git-dev/BLINK](https://github.com/zm-git-dev/BLINK) - BLINK: A Package for Next Level of Genome Wide Association Studies with Both Individuals and Markers in Millions
- [zm-git-dev/genometools](https://github.com/zm-git-dev/genometools) - GenomeTools genome analysis system.
- [zm-git-dev/fastq-filter](https://github.com/zm-git-dev/fastq-filter) - A fast FASTQ filter program.
- [zm-git-dev/GMMAT](https://github.com/zm-git-dev/GMMAT) - Generalized linear Mixed Model Association Tests
- [zm-git-dev/T2T-Polish](https://github.com/zm-git-dev/T2T-Polish) - Evaluation and polishing workflows for T2T genome assemblies
- [zm-git-dev/f5c](https://github.com/zm-git-dev/f5c) - Ultra-fast methylation calling and event alignment tool for nanopore sequencing data (supports CUDA acceleration)
- [zm-git-dev/Flye](https://github.com/zm-git-dev/Flye) - De novo assembler for single molecule sequencing reads using repeat graphs
- [zm-git-dev/olego](https://github.com/zm-git-dev/olego) - 
- [zm-git-dev/dysgu](https://github.com/zm-git-dev/dysgu) - dysgu-SV is a collection of tools for calling structural variants using short or long reads
- [shulp2211/BAMreport](https://github.com/shulp2211/BAMreport) - Report on a BAM file
- [refresh-bio/FAMSA](https://github.com/refresh-bio/FAMSA) - Algorithm for ultra-scale multiple sequence alignments (3M protein sequences in 5 minutes and 24 GB of RAM)
- [sanger-pathogens/snp-sites](https://github.com/sanger-pathogens/snp-sites) - Finds SNP sites from a multi-FASTA alignment file
- [brentp/bio-playground](https://github.com/brentp/bio-playground) - miscellaneous scripts for bioinformatics/genomics that dont merit their own repo.
- [chaimol/bio_code](https://github.com/chaimol/bio_code) - this project is content about my bioinformation analysis code .it may have Rï¼ŒShellï¼ŒPython.
- [alshai/vcf2msa](https://github.com/alshai/vcf2msa) - convert a multi-sample VCF file to a multiple sequence alignment (C)
- [ToolsVanBox/PrimerDesign](https://github.com/ToolsVanBox/PrimerDesign) - Tool to design primers on a vcf file (using primer3 software)
- [eastmallingresearch/crosslink](https://github.com/eastmallingresearch/crosslink) - genetic mapping software for outbreeding populations with additional features suitable for allo-polyploids
- [lh3/bedtk](https://github.com/lh3/bedtk) - A simple toolset for BED files (warning: CLI may change before bedtk becomes stable)

## C# # 

- [ZhiGroup/IBD_benchmark](https://github.com/ZhiGroup/IBD_benchmark) - 
- [1357310795/MyComputerManager](https://github.com/1357310795/MyComputerManager) - ç®¡ç†â€œæ­¤ç”µè„‘â€é‡Œåˆ ä¸æ‰çš„æµæ°“â€œå¿«æ·æ–¹å¼â€ï¼ˆåŒ…æ‹¬ä¾§è¾¹æ ï¼‰ï¼ŒåŒæ—¶å¯è‡ªå·±æ·»åŠ è¿™ç±»â€œå¿«æ·æ–¹å¼â€

## C++ 

- [delehef/treerecs](https://github.com/delehef/treerecs) - 
- [amkozlov/raxml-ng](https://github.com/amkozlov/raxml-ng) - RAxML Next Generation: faster, easier-to-use and more flexible
- [hahnlab/CAFE](https://github.com/hahnlab/CAFE) - Analyze changes in gene family size and provide a statistical foundation for evolutionary inferences.
- [idanefroni/Conservatory](https://github.com/idanefroni/Conservatory) - Identification of conserved non-coding sequences in plants
- [BGI-shenzhen/BamDeal](https://github.com/BGI-shenzhen/BamDeal) - BamDeal: a comprehensive toolkit for bam manipulation
- [chhylp123/hifiasm](https://github.com/chhylp123/hifiasm) - Hifiasm: a haplotype-resolved assembler for accurate Hifi reads
- [bcm-uga/Loter](https://github.com/bcm-uga/Loter) - A software package for local ancestry inference and haplotype phasing
- [halasadi/MAPS](https://github.com/halasadi/MAPS) - software for the MAPS method
- [tidyverse/vroom](https://github.com/tidyverse/vroom) - Fast reading of delimited files
- [Illumina/paragraph](https://github.com/Illumina/paragraph) - Graph realignment tools for structural variants
- [williamslab/ibis](https://github.com/williamslab/ibis) - Algorithm for rapid, phase-free detection of long identical by descent segments
- [marbl/MashMap](https://github.com/marbl/MashMap) - A fast approximate aligner for long DNA sequences
- [maickrau/MBG](https://github.com/maickrau/MBG) - 
- [jacobbierstedt/fastqc-summary](https://github.com/jacobbierstedt/fastqc-summary) - Summarize fastqc results by parsing the fastqc_data.txt files output by a fastqc run.
- [hahnlab/CAGEE](https://github.com/hahnlab/CAGEE) - Computational Analysis of Gene Expression Evolution
- [dellytools/sansa](https://github.com/dellytools/sansa) - Structural variant annotation
- [OpenGene/fastp](https://github.com/OpenGene/fastp) - An ultra-fast all-in-one FASTQ preprocessor (QC/adapters/trimming/filtering/splitting/merging...)
- [gsneha26/SegAlign](https://github.com/gsneha26/SegAlign) - A Scalable GPU-Based Whole Genome Aligner, published in SC20: https://doi.ieeecomputersociety.org/10.1109/SC41405.2020.00043
- [voichek/fetch_reads_with_kmers](https://github.com/voichek/fetch_reads_with_kmers) - A small program to fetch reads from paired-end sequences which contain a predefined set of k-mers
- [voichek/kmersGWAS](https://github.com/voichek/kmersGWAS) - A library for running k-mers based GWAS
- [akolesnikov/bamtools](https://github.com/akolesnikov/bamtools) - C++ API & command-line toolkit for working with BAM data
- [DecodeGenetics/popSTR](https://github.com/DecodeGenetics/popSTR) - PopSTR - A Population based microsatellite genotyper
- [jacobbierstedt/gfftobed](https://github.com/jacobbierstedt/gfftobed) - Convert GFF3/GTF to BED
- [kcleal/gw](https://github.com/kcleal/gw) - Genome browser and variant annotation
- [c-guzman/cipher-workflow-platform](https://github.com/c-guzman/cipher-workflow-platform) - A data processing platform for ChIP-seq, RNA-seq, MNase-seq, DNase-seq, ATAC-seq and GRO-seq datasets. Please ignore information on cipher.readthedocs.io, it is currently out of date. Follow informati
- [sunyumail93/fastqtl](https://github.com/sunyumail93/fastqtl) - Enhanced version of the FastQTL QTL mapper
- [eblerjana/pangenie](https://github.com/eblerjana/pangenie) - Pangenome-based genome inference
- [paoloshasta/shasta](https://github.com/paoloshasta/shasta) - De novo assembly from Oxford Nanopore reads.
- [jianyangqt/gcta](https://github.com/jianyangqt/gcta) - GCTA software
- [Illumina/manta](https://github.com/Illumina/manta) - Structural variant and indel caller for mapped sequencing data
- [rrwick/Unicycler](https://github.com/rrwick/Unicycler) - hybrid assembly pipeline for bacterial genomes
- [lyotvincent/CFMP](https://github.com/lyotvincent/CFMP) - 
- [BGI-Qingdao/TGS-GapCloser](https://github.com/BGI-Qingdao/TGS-GapCloser) - A gap-closing software tool that uses long reads to enhance genome assembly.
- [bwa-mem2/bwa-mem2](https://github.com/bwa-mem2/bwa-mem2) - The next version of bwa-mem
- [ANGSD/angsd](https://github.com/ANGSD/angsd) - Program for analysing NGS data.
- [millanek/Dsuite](https://github.com/millanek/Dsuite) - Fast calculation of Patterson's D (ABBA-BABA) and the f4-ratio statistics across many populations/species
- [fritzsedlazeck/SURVIVOR](https://github.com/fritzsedlazeck/SURVIVOR) - Toolset for SV simulation, comparison and filtering
- [gear-genomics/dicey](https://github.com/gear-genomics/dicey) - In-silico PCR, primer design and padlock design for in-situ sequencing
- [martin-steinegger/FAMSA](https://github.com/martin-steinegger/FAMSA) - Algorithm for large-scale multiple sequence alignments (400k proteins in 2 hours and 8BG of RAM)
- [waveygang/wfmash](https://github.com/waveygang/wfmash) - base-accurate DNA sequence alignments using WFA and mashmap2
- [broadinstitute/gamgee](https://github.com/broadinstitute/gamgee) - A C++14 library for NGS data formats
- [baoxingsong/AnchorWave](https://github.com/baoxingsong/AnchorWave) - sensitive alignment of genomes with high sequence diversity, extensive structural polymorphism and whole-genome duplication variation
- [rcedgar/urmap](https://github.com/rcedgar/urmap) - URMAP ultra-fast read mapper
- [jltsiren/vg](https://github.com/jltsiren/vg) - tools for working with genome variation graphs
- [ZhangZhenmiao/stLFR_Pipeline](https://github.com/ZhangZhenmiao/stLFR_Pipeline) - 
- [Shicheng-Guo/methylFlow](https://github.com/Shicheng-Guo/methylFlow) - Cell-specific methylation pattern reconstruction
- [Shicheng-Guo/bsmooth-align](https://github.com/Shicheng-Guo/bsmooth-align) - Alignment of bisulfite sequence reads and tabulation of read-level methylation measurements
- [zm-git-dev/mergeSVcallers](https://github.com/zm-git-dev/mergeSVcallers) - heuristics to merge structural variant calls in VCF format.
- [zm-git-dev/SA-SSR](https://github.com/zm-git-dev/SA-SSR) - A Novel Algorithm for SSR Discovery in Genetic Sequences
- [zm-git-dev/mplotter](https://github.com/zm-git-dev/mplotter) - Creates publication quality dot plot using ggplot
- [zm-git-dev/scallop](https://github.com/zm-git-dev/scallop) - Scallop is a reference-based transcriptome assembler for RNA-seq
- [zm-git-dev/stringtie](https://github.com/zm-git-dev/stringtie) - Transcript assembly and quantification for RNA-Seq
- [zm-git-dev/stacks-old](https://github.com/zm-git-dev/stacks-old) - 
- [zm-git-dev/vcftools](https://github.com/zm-git-dev/vcftools) - A set of tools written in Perl and C++ for working with VCF files, such as those generated by the 1000 Genomes Project.
- [zm-git-dev/Sibelia](https://github.com/zm-git-dev/Sibelia) - Genome comparison via de Bruijn graph. To get the latest stable version, please visit our site.
- [zm-git-dev/Conservatory](https://github.com/zm-git-dev/Conservatory) - Identification of conserved non-coding sequences in plants
- [zm-git-dev/Sorghum_pangenome](https://github.com/zm-git-dev/Sorghum_pangenome) - Sorghum pangenome analysis
- [zm-git-dev/SibeliaZ](https://github.com/zm-git-dev/SibeliaZ) - A fast whole-genome aligner based on de Bruijn graphs
- [zm-git-dev/maf2synteny](https://github.com/zm-git-dev/maf2synteny) - A tool for recovering synteny blocks from multiple alignment
- [zm-git-dev/Quast-1](https://github.com/zm-git-dev/Quast-1) - Quality Assessment Tool for Genome Assemblies (Gurevich et al, 2013)
- [zm-git-dev/CNVnator](https://github.com/zm-git-dev/CNVnator) - a tool for CNV discovery and genotyping from depth-of-coverage by mapped reads
- [zm-git-dev/graphtyper](https://github.com/zm-git-dev/graphtyper) - Population-scale genotyping using pangenome graphs
- [zm-git-dev/FAMSA](https://github.com/zm-git-dev/FAMSA) - Algorithm for ultra-scale multiple sequence alignments (3M protein sequences in 5 minutes and 24 GB of RAM)
- [zm-git-dev/MEGADOCK](https://github.com/zm-git-dev/MEGADOCK) - An ultra-high-performance protein-protein docking for heterogeneous supercomputers
- [zm-git-dev/DROMPAplus](https://github.com/zm-git-dev/DROMPAplus) - ChIP-seq pipeline tool for quality check, normalization, statistical analysis, and visualization of multiple ChIP-seq samples.
- [zm-git-dev/odgi](https://github.com/zm-git-dev/odgi) - Optimized Dynamic Genome/Graph Implementation: understanding pangenome graphs (https://doi.org/10.1093/bioinformatics/btac308)
- [zm-git-dev/BWA-MEME](https://github.com/zm-git-dev/BWA-MEME) - Faster BWA-MEM2 using learned-index
- [zm-git-dev/wally](https://github.com/zm-git-dev/wally) - Wally: Visualization of aligned sequencing reads and contigs
- [benwbooth/bam2bedgraph.cc](https://github.com/benwbooth/bam2bedgraph.cc) - Convert .bam alignment files to bedgraph or bigwig format.
- [bioinfologics/sdg](https://github.com/bioinfologics/sdg) - Sequence Distance Graph framework: graph + reads + mapping + analysis
- [mahmudz/HungryCloner](https://github.com/mahmudz/HungryCloner) - A simple GUI application to bulk download GitHub user's public repositories
- [corbinq/GAMBIT](https://github.com/corbinq/GAMBIT) - Tool for integrative gene-based association analysis using GWAS summary stats
- [lizzhao/Kaks_Calculator](https://github.com/lizzhao/Kaks_Calculator) - 
- [DecodeGenetics/LRcaller](https://github.com/DecodeGenetics/LRcaller) - 
- [xzhub/LTR_Finder](https://github.com/xzhub/LTR_Finder) - LTR_Finder is an efficient program for finding full-length LTR retrotranspsons in genome sequences.
- [urgi-anagen/TE_finder](https://github.com/urgi-anagen/TE_finder) - A suite of C++ programs developed for transposable element search and their annotation in large eukaryotic genome sequence. A part of the REPET package.
- [alshai/varcount](https://github.com/alshai/varcount) - Outputs alignment coverage over a predefined set of variants
- [Kingsford-Group/variantstore](https://github.com/Kingsford-Group/variantstore) - VariantStore: A Large-Scale Genomic Variant Search Index
- [wen-chen/GCEN](https://github.com/wen-chen/GCEN) - GCEN: an easy-to-use toolkit for Gene Co-Expression Network analysis and lncRNAs annotation
- [bbuchfink/diamond](https://github.com/bbuchfink/diamond) - Accelerated BLAST compatible local sequence aligner.
- [BGI-shenzhen/VCF2Dis](https://github.com/BGI-shenzhen/VCF2Dis) - VCF2Dis: A new simple and efficient software to calculate p-distance matrix  based Variant Call Format
- [qiao-xin/DupGen_finder](https://github.com/qiao-xin/DupGen_finder) - A pipeline used to identify different modes of duplicated gene pairs
- [inab/trimal](https://github.com/inab/trimal) - A tool for automated alignment trimming in large-scale phylogenetic analyses. Development version: 2.0
- [Iceylee/NGS-Pacbio](https://github.com/Iceylee/NGS-Pacbio) - Pipelines and scripts for analysing RNA-seqï¼Œgenome and other sequecing data.

## CSS 

- [Kinice/propose_mina](https://github.com/Kinice/propose_mina) - æ±‚å©šç”¨çš„å°ç¨‹åº
- [zouyaoji/wedding-invitation](https://github.com/zouyaoji/wedding-invitation) - å¾®ä¿¡å°ç¨‹åºå©šç¤¼è¯·æŸ¬
- [arungop/akhiarya](https://github.com/arungop/akhiarya) - Animated website for kerala wedding invitation(also contains timer, background music, Photo gallery and Pre-wedding video) | Save the Date: Apr 24, 2021
- [Kevinzjy/ManuGPT](https://github.com/Kevinzjy/ManuGPT) - Improve your manuscript using OpenAI GPT model
- [sharu725/online-cv](https://github.com/sharu725/online-cv) - A minimal Jekyll Theme to host your resume (CV)
- [RoyalKnights/PhotoShow](https://github.com/RoyalKnights/PhotoShow) - ä»…ç”¨html+csså®ç°çš„å„ç§ç›¸å†Œæ•ˆæœ
- [bestxtools/markdown-online-editor](https://github.com/bestxtools/markdown-online-editor) - ğŸ“åŸºäº Vueã€Vditorï¼Œæ‰€æ„å»ºçš„åœ¨çº¿ Markdown ç¼–è¾‘å™¨ï¼Œæ”¯æŒæµç¨‹å›¾ã€ç”˜ç‰¹å›¾ã€æ—¶åºå›¾ã€ä»»åŠ¡åˆ—è¡¨ã€HTML è‡ªåŠ¨è½¬æ¢ä¸º Markdown ç­‰åŠŸèƒ½ï¼›ğŸ‰æ–°å¢ã€Œæ‰€è§å³æ‰€å¾—ã€ç¼–è¾‘æ¨¡å¼ã€‚| å¥½å·¥å…·å‘¨åˆŠç¬¬ 17 æœŸ - https://discuss-cn.bestxtools.com/d/43
- [zm-git-dev/ccwang002.github.io](https://github.com/zm-git-dev/ccwang002.github.io) - My personal blog
- [zm-git-dev/biology-note](https://github.com/zm-git-dev/biology-note) - 
- [IagoLast/paper](https://github.com/IagoLast/paper) - Typora theme for the academy
- [blinkfox/typora-vue-theme](https://github.com/blinkfox/typora-vue-theme) - This is a typora theme inspired by Vue document style. ä¸€ä¸ªç±»ä¼¼äº Vue æ–‡æ¡£é£æ ¼çš„ Typora Markdown ç¼–è¾‘å™¨ä¸»é¢˜ã€‚
- [champagne-supernova/breast-cancer-database](https://github.com/champagne-supernova/breast-cancer-database) - 

## CoffeeScript 

- [wballard/git-friends](https://github.com/wballard/git-friends) - 

## Common Workflow Language 

- [Giulia-Calia/PLANTOGEN-A](https://github.com/Giulia-Calia/PLANTOGEN-A) - PLANTOGEN-A, an automated genome assembly pipeline designed to assemble symbiont microorganisms genome from Oxford Nanopore (ONT) data.
- [ncbi/pgap](https://github.com/ncbi/pgap) - NCBI Prokaryotic Genome Annotation Pipeline

## Cython 

- [23andMe/phasedibd](https://github.com/23andMe/phasedibd) - Identity-by-descent inference using the templated positional Burrows-Wheeler transform (TPBWT)
- [biocore-ntnu/epic2](https://github.com/biocore-ntnu/epic2) - Ultraperformant reimplementation of SICER
- [zm-git-dev/TIDDIT](https://github.com/zm-git-dev/TIDDIT) - TIDDIT - structural variant calling
- [schneebergerlab/syri](https://github.com/schneebergerlab/syri) - Synteny and Rearrangement Identifier
- [SciLifeLab/TIDDIT](https://github.com/SciLifeLab/TIDDIT) - TIDDIT - structural variant calling
- [macs3-project/MACS](https://github.com/macs3-project/MACS) - MACS -- Model-based Analysis of ChIP-Seq

## D 

- [genetics-statistics/faster_lmm_d](https://github.com/genetics-statistics/faster_lmm_d) - A faster lmm for GWAS. Supports GPU backend.

## Dart 

- [nslogx/flutter_easyloading](https://github.com/nslogx/flutter_easyloading) - âœ¨A clean and lightweight loading/toast widget for Flutter, easy to use without context, support iOSã€Android and Web

## Dockerfile 

- [Joyvalley/benchmark](https://github.com/Joyvalley/benchmark) - 
- [dceoy/docker-rstudio-server](https://github.com/dceoy/docker-rstudio-server) - Dockerfile for RStudio Server
- [zm-git-dev/microRNA_prediction_models_docker](https://github.com/zm-git-dev/microRNA_prediction_models_docker) - A docker file for training miRNA expression prediction models
- [zm-git-dev/latch_trinity](https://github.com/zm-git-dev/latch_trinity) - Implementation of Trinity de novo transcriptome assembly on the Latch Bio platform

## EJS 

- [Fechin/reference](https://github.com/Fechin/reference) - â­• Share quick reference cheat sheet for developers.
- [OmicsSuite/OmicsSuite.github.io](https://github.com/OmicsSuite/OmicsSuite.github.io) - OmicsSuite: a suite for multi-omics analysis and visualization.

## GAP 

- [zm-git-dev/LTDE](https://github.com/zm-git-dev/LTDE) - 

## GDScript 

- [LennonLab/LTDE](https://github.com/LennonLab/LTDE) - 

## Go 

- [nemoTyrant/goose](https://github.com/nemoTyrant/goose) - ä¸‹è½½å°é¹…é€šè§†é¢‘
- [shenwei356/taxonkit](https://github.com/shenwei356/taxonkit) - A Practical and Efficient NCBI Taxonomy Toolkit, also supports creating NCBI-style taxdump files for custom taxonomies like GTDB/ICTV
- [DeluxeOwl/gh-stars-backup](https://github.com/DeluxeOwl/gh-stars-backup) - 
- [gabrie30/ghorg](https://github.com/gabrie30/ghorg) - Quickly clone an entire org/users repositories into one directory - Supports GitHub, GitLab, Bitbucket, and more ğŸ¥š
- [zm-git-dev/pinfish](https://github.com/zm-git-dev/pinfish) - Tools to annotate genomes using long read transcriptomics data
- [zm-git-dev/lpp_Script](https://github.com/zm-git-dev/lpp_Script) - my bioinfomatics script
- [zm-git-dev/FastqCount-1](https://github.com/zm-git-dev/FastqCount-1) - Summary fastq reads, bases, Q20, Q30, N bases, GC
- [zm-git-dev/vcfanno](https://github.com/zm-git-dev/vcfanno) - annotate a VCF with other VCFs/BEDs/tabixed files
- [zm-git-dev/pgzip](https://github.com/zm-git-dev/pgzip) - Go parallel gzip (de)compression
- [zm-git-dev/ghorg](https://github.com/zm-git-dev/ghorg) - Quickly clone an entire org/users repositories into one directory - Supports GitHub, GitLab, Bitbucket, and more ğŸ¥š
- [shenwei356/rush](https://github.com/shenwei356/rush) - A cross-platform command-line tool for executing jobs in parallel
- [chenjiandongx/pprofsnapshot](https://github.com/chenjiandongx/pprofsnapshot) - ğŸ§¶ pprofsnapshot makes it easy to download profiling data to local.
- [Mikubill/cowtransfer-uploader](https://github.com/Mikubill/cowtransfer-uploader) - Simple Cowtransfer Uploader/Downloader in Golang
- [clindet/bget](https://github.com/clindet/bget) - Portable command-line tool to query bioinformatics APIs, data, databases and files.
- [fatedier/frp](https://github.com/fatedier/frp) - A fast reverse proxy to help you expose a local server behind a NAT or firewall to the internet.

## Groovy 

- [ssadedin/bpipe](https://github.com/ssadedin/bpipe) - Bpipe - a tool for running and managing bioinformatics pipelines
- [zm-git-dev/LncPipe](https://github.com/zm-git-dev/LncPipe) - See the update fork of this repository here &gt;&gt;&gt; https://github.com/nf-core/lncpipe/tree/dev

## HTML 

- [Vileu/EasyMetagenome](https://github.com/Vileu/EasyMetagenome) - Easy Metagenome Pipeline 2019 (1.0)
- [wynode/mapAlbum](https://github.com/wynode/mapAlbum) - ä¸€ä¸ªå¯ä»¥æ— é™å­˜å‚¨ç…§ç‰‡çš„åœ°å›¾ç›¸å†Œï¼Œè®°å½•ä½ çš„è¶³è¿¹ã€‚
- [rampatra/wedding-website](https://github.com/rampatra/wedding-website) - Our Wedding Website ğŸ‘«
- [CantuLab/cantulab.github.com](https://github.com/CantuLab/cantulab.github.com) - The lab website of Dario Cantu
- [lixiang117423/PlantKO](https://github.com/lixiang117423/PlantKO) - 
- [ewels/sra-explorer](https://github.com/ewels/sra-explorer) - Web application to explore the Sequence Read Archive.
- [bowhan/piPipes](https://github.com/bowhan/piPipes) - piRNA pipeline collection developed in the Zamore Lab and ZLab in UMass Med School
- [zhengshimao/my_blog](https://github.com/zhengshimao/my_blog) - å­¦ç”Ÿä¿¡çš„å¤§å”çš„åšå®¢
- [SziKayLeung/Whole_Transcriptome_Paper](https://github.com/SziKayLeung/Whole_Transcriptome_Paper) - 
- [sajuukLyu/Protocols-4pub](https://github.com/sajuukLyu/Protocols-4pub) - Multi-omics analysis protocols by Lyu.
- [StevenWingett/FastQ-Screen](https://github.com/StevenWingett/FastQ-Screen) - Detecting contamination in NGS data and multi-species analysis
- [stephens999/fiveMinuteStats](https://github.com/stephens999/fiveMinuteStats) - A repo of short "vignettes" illustrating statistical concepts
- [openbiox/weekly](https://github.com/openbiox/weekly) - ç”Ÿä¿¡çˆ±å¥½è€…å‘¨åˆŠï¼ˆæ¯å‘¨æ—¥å‘å¸ƒï¼‰
- [iansealy/biostar-handbook-web](https://github.com/iansealy/biostar-handbook-web) - 
- [Tong-Chen/Bioinfo_course_R](https://github.com/Tong-Chen/Bioinfo_course_R) - 
- [lixiang117423/ggmotif](https://github.com/lixiang117423/ggmotif) - 
- [Amz965/cs-self-learning](https://github.com/Amz965/cs-self-learning) - è®¡ç®—æœºè‡ªå­¦æŒ‡å—
- [eead-csic-compbio/coexpression_motif_discovery](https://github.com/eead-csic-compbio/coexpression_motif_discovery) - Discovering cis-regulatory motifs in proximal promoters of plants in RSAT::Plants
- [MerrimanLab/GWASPipeline](https://github.com/MerrimanLab/GWASPipeline) - A basic GWAS analyses workflow. Worked examples provided in rendered markdown.
- [mariakmejiaguerra/TF_binding_maize_leaf](https://github.com/mariakmejiaguerra/TF_binding_maize_leaf) - Repository to generate draft figures for the paper "Reconstructing the maize leaf regulatory network using ChIP-seq data of 104 transcription factors"  The R_notebook folder includes text in line with
- [leezx/bioinfo-pipelines](https://github.com/leezx/bioinfo-pipelines) - classical pipelines in bioinformatics
- [rrazaghi/modbamtools](https://github.com/rrazaghi/modbamtools) - Set of tools to manipulate and visualize modified base bam files
- [ashwini06/seqkit](https://github.com/ashwini06/seqkit) - test_approach for integrated ChiP-Seq + RNA-Seq analysis
- [agrosavia-bioinfo/development](https://github.com/agrosavia-bioinfo/development) - Development projects: dirs and files
- [related-sciences/gwas-analysis-manuscript](https://github.com/related-sciences/gwas-analysis-manuscript) - A comparison of GWAS analysis tools (Glow, Hail, and PLINK)
- [mdchambers/dissertation](https://github.com/mdchambers/dissertation) - My Doctoral Dissertation
- [hbctraining/In-depth-NGS-Data-Analysis-Course](https://github.com/hbctraining/In-depth-NGS-Data-Analysis-Course) - 
- [dnyansagar/gene_regulatory_network](https://github.com/dnyansagar/gene_regulatory_network) - Study gene regulatory network with a combination of ChIP-seq, RNA-seq, and comparative genomics
- [crahal/GWASReview](https://github.com/crahal/GWASReview) - A Scientometric Review of Genome Wide Association Studies
- [mulinlab/mulinlab-pip](https://github.com/mulinlab/mulinlab-pip) - mulinlab bioinformatics pipelines for reproducible, efficient and convenient bioinformatics research.
- [randel/MIND](https://github.com/randel/MIND) - Using Bulk Gene Expression to Estimate Cell-Type-Specific Gene Expression via Deconvolution
- [shulp2211/hic-bench](https://github.com/shulp2211/hic-bench) - A set of pipelines for Hi-C and ChIP-Seq analysis.
- [Shicheng-Guo/dMeth](https://github.com/Shicheng-Guo/dMeth) - deconvolution of gene expression using DNA methylation data
- [Shicheng-Guo/MethylNet](https://github.com/Shicheng-Guo/MethylNet) - Modular framework for deep learning predictions on methylation data.
- [Shicheng-Guo/methylation2020](https://github.com/Shicheng-Guo/methylation2020) - 
- [Shicheng-Guo/methylation-1](https://github.com/Shicheng-Guo/methylation-1) - A methylation workflow
- [gaospecial/bio-spring.info](https://github.com/gaospecial/bio-spring.info) - My Blogdown Site
- [GuangchuangYu/cv](https://github.com/GuangchuangYu/cv) - Guangchuang Yu's CV
- [johanneskoester/dna-seq-gatk-variant-calling](https://github.com/johanneskoester/dna-seq-gatk-variant-calling) - This Snakemake pipeline implements the GATK best-practices workflow
- [zm-git-dev/sed_and_awk](https://github.com/zm-git-dev/sed_and_awk) - A online presentation for sed & awk
- [zm-git-dev/ngsane](https://github.com/zm-git-dev/ngsane) - Analysis Framework for Biological Data from High Throughput Experiments
- [zm-git-dev/Tutorial-BSA-Python](https://github.com/zm-git-dev/Tutorial-BSA-Python) - Tutorial for bulk segregant analysis using Python
- [zm-git-dev/qtl_pipeline](https://github.com/zm-git-dev/qtl_pipeline) - software pipeline for molecular QTL analysis
- [zm-git-dev/BallGownTuto](https://github.com/zm-git-dev/BallGownTuto) - Knitr markdown tutorial file for RNAseq study using the tuxedo pipeline (HiSAT2, StringTie, BallGown)
- [zm-git-dev/my-R](https://github.com/zm-git-dev/my-R) - a small collection for my R  scripts
- [zm-git-dev/W3school-HTML](https://github.com/zm-git-dev/W3school-HTML) - practice the examples from W3school HTML
- [zm-git-dev/In-depth-NGS-Data-Analysis-Course](https://github.com/zm-git-dev/In-depth-NGS-Data-Analysis-Course) - Materials for 12-day course on analyzing RNA-Seq, ChIP-Seq and variant calling data.
- [zm-git-dev/FMR1CA1rnaseq](https://github.com/zm-git-dev/FMR1CA1rnaseq) - To better understand the effects silencing the Fmr1 gene, we conducted an integrative analysis of avoidance, cognitive discrimination, and hippocampal gene expression WT and FMR1-KO mice.
- [zm-git-dev/sophon](https://github.com/zm-git-dev/sophon) - 
- [zm-git-dev/supplementary](https://github.com/zm-git-dev/supplementary) - 
- [zm-git-dev/papers-1](https://github.com/zm-git-dev/papers-1) - Repository containing all scripts and documentation for published papers.
- [zm-git-dev/papers](https://github.com/zm-git-dev/papers) - Code for manuscripts published by Thompson and colleagues.
- [zm-git-dev/Zhang2019NBT](https://github.com/zm-git-dev/Zhang2019NBT) - Scripts for stat and plot figures in rice microbiome paper
- [zm-git-dev/bpipes](https://github.com/zm-git-dev/bpipes) - bioinformatic pipelines
- [zm-git-dev/brassica-snps](https://github.com/zm-git-dev/brassica-snps) - Scripts for bulk segregant analysis and genotyping of Brassica (T1000 & Jersey Kale)
- [zm-git-dev/PaperSearch](https://github.com/zm-git-dev/PaperSearch) - 
- [zm-git-dev/Population_Genetics_in_R](https://github.com/zm-git-dev/Population_Genetics_in_R) - A primer for computational tools to analyze genetics of populations in R
- [zm-git-dev/Shiny_Co](https://github.com/zm-git-dev/Shiny_Co) - Code to perform WGCNA analyses on bacterial datasets from SRA
- [zm-git-dev/CoExpNets](https://github.com/zm-git-dev/CoExpNets) - Co-expression network management based on WGCNA + k-means
- [zm-git-dev/NGSPipeDb](https://github.com/zm-git-dev/NGSPipeDb) - 
- [zm-git-dev/Metagenome_ICI](https://github.com/zm-git-dev/Metagenome_ICI) - 
- [zm-git-dev/Python-Data-Visualization](https://github.com/zm-git-dev/Python-Data-Visualization) - Data visualization with Python
- [zm-git-dev/Reference_transcriptome](https://github.com/zm-git-dev/Reference_transcriptome) - 
- [zm-git-dev/PopPipe](https://github.com/zm-git-dev/PopPipe) - 
- [zm-git-dev/myscripts](https://github.com/zm-git-dev/myscripts) - Collection of my scripts
- [zm-git-dev/Workshop](https://github.com/zm-git-dev/Workshop) - è¯¾é¢˜ç»„æ¯å‘¨ç ”è®¨ä¼š
- [zm-git-dev/rnaflow](https://github.com/zm-git-dev/rnaflow) - A simple RNA-Seq differential gene expression pipeline using Nextflow
- [zm-git-dev/msPIPE](https://github.com/zm-git-dev/msPIPE) - 
- [zm-git-dev/Timema_asex_genomes](https://github.com/zm-git-dev/Timema_asex_genomes) - Code for "Convergent consequences of parthenogenesis on stick insect genomes" manuscript
- [zm-git-dev/knotAnnotSV](https://github.com/zm-git-dev/knotAnnotSV) - A simple script to create a customizable html file from an AnnotSV output.
- [zm-git-dev/funRiceGenes](https://github.com/zm-git-dev/funRiceGenes) - The knowledge of cloned rice genes lost in the information of rice functional genomics studies
- [zm-git-dev/tspex-webapp](https://github.com/zm-git-dev/tspex-webapp) - tspex webapp
- [zm-git-dev/PopAT-expression-analysis](https://github.com/zm-git-dev/PopAT-expression-analysis) - Tissue specific expression analysis in order to identify new transcription factors that are the most representitive of specific tissue, and therefor suspected to have a strong role as a development re
- [linklink111/MyGithubModle](https://github.com/linklink111/MyGithubModle) - A project that try tto imitate github
- [hoelzer-lab/rnaflow](https://github.com/hoelzer-lab/rnaflow) - A simple RNA-Seq differential gene expression pipeline using Nextflow
- [tpoorten/dotPlotly](https://github.com/tpoorten/dotPlotly) - Generate an interactive dot plot from mummer or minimap alignments
- [Yixf-Education/course_Bioinformatics](https://github.com/Yixf-Education/course_Bioinformatics) - å¤©æ´¥åŒ»ç§‘å¤§å­¦ï¼Œç”Ÿç‰©åŒ»å­¦å·¥ç¨‹ä¸æŠ€æœ¯å­¦é™¢ï¼Œã€Šç”Ÿç‰©ä¿¡æ¯å­¦ã€‹è¯¾ç¨‹èµ„æ–™
- [kaustubhad/fastman](https://github.com/kaustubhad/fastman) - An R package for fast and efficient visualizing of GWAS results using Q-Q and Manhattan plots directly from PLINK output files.
- [pgpmartin/RNAseq_ChIPseq_course](https://github.com/pgpmartin/RNAseq_ChIPseq_course) - Introduction to RNA-seq and ChIP-seq data analysis using PAF1 Cell 2015 article
- [glarue/resume](https://github.com/glarue/resume) - My rÃ©sumÃ©
- [z3tt/beyond-bar-and-box-plots](https://github.com/z3tt/beyond-bar-and-box-plots) - Slides, recording and hands-on tutorial for the USGS Data Science webinar
- [abrahamalbert18/Single-SNP-Analysis-of-Plant-Phenotypes](https://github.com/abrahamalbert18/Single-SNP-Analysis-of-Plant-Phenotypes) - GWAS analysis using FaST-LMM.
- [mitsingh1/SNP-and-GWAS-of-rice-plant-varieties](https://github.com/mitsingh1/SNP-and-GWAS-of-rice-plant-varieties) - 
- [sjoleary/SFL_LinkageMap](https://github.com/sjoleary/SFL_LinkageMap) - 
- [MorrellLAB/DoesNaughtCompute](https://github.com/MorrellLAB/DoesNaughtCompute) - Code and example datasets for Does[0]Compute
- [zenbook/Jeremy-R](https://github.com/zenbook/Jeremy-R) - Learn R for data science
- [antronerds/r2-tutorials](https://github.com/antronerds/r2-tutorials) - This repository contains the files for the tutorials for the web-based molecular biology analysis platform
- [SISBID/All-Modules-Archive](https://github.com/SISBID/All-Modules-Archive) - 
- [bvieth/scRNA-seq-pipelines](https://github.com/bvieth/scRNA-seq-pipelines) - Compendium to "A Systematic Evaluation of Single Cell RNA-Seq Analysis Pipelines"
- [fei0810/LearnBasicStatisticsinR](https://github.com/fei0810/LearnBasicStatisticsinR) - ç»Ÿè®¡å­¦ä¸R
- [NTLx/blog](https://github.com/NTLx/blog) - My oldest blog build in Github
- [IgnatiusPang/Triplet_Motifs](https://github.com/IgnatiusPang/Triplet_Motifs) - 
- [reedliu/R-code](https://github.com/reedliu/R-code) - 
- [hbctraining/DGE_workshop](https://github.com/hbctraining/DGE_workshop) - 
- [anjitrue/Drosophila-Traumatic-Brain-Injury](https://github.com/anjitrue/Drosophila-Traumatic-Brain-Injury) - Shot gun proteomics on heads and hemolymph
- [YTLogos/Population_Genetics_in_R](https://github.com/YTLogos/Population_Genetics_in_R) - A primer for computational tools to analyze genetics of populations in R
- [jmzeng1314/5years](https://github.com/jmzeng1314/5years) - 
- [dongweiming/sed_and_awk](https://github.com/dongweiming/sed_and_awk) - A online presentation for sed & awk

## Haskell 

- [kaizhang/bioinformatics-toolkit](https://github.com/kaizhang/bioinformatics-toolkit) - A collection of bioinformatics algorithms

## Java 

- [isyangyuhao/book-mate](https://github.com/isyangyuhao/book-mate) - ä¸€æ¬¾ç”¨äºå›¾ä¹¦æ¨èã€æœç´¢ã€å€Ÿé˜…ã€äº¤æµçš„å¾®ä¿¡å°ç¨‹åº
- [GuoliangLi-HZAU/ChIA-PET_Tool_V3](https://github.com/GuoliangLi-HZAU/ChIA-PET_Tool_V3) - ChIA-PET, HiChIP, PLAC-Seq data analysis, with linker detection, linker filter, or restriction process for HiChIP, mapping, call cluster and visulization
- [eead-csic-compbio/vcf-filter](https://github.com/eead-csic-compbio/vcf-filter) - Java tool to filter an input VCF file a la carte, based on HTSJDK and Apache commons CLI.
- [BGI-flexlab/vcfstats](https://github.com/BGI-flexlab/vcfstats) - BGI-lowpass imputation vcf stats
- [dnaase/Bis-tools](https://github.com/dnaase/Bis-tools) - A collection of tools to deal with Bisulfite-seq/NOMe-seq (SNP/Methylation calling: BisSNP; HMM segmentation: DMNTools; Visualization/Clustering: alignWigToBed; )
- [Shicheng-Guo/Bis-tools](https://github.com/Shicheng-Guo/Bis-tools) - A collection of tools to deal with Bisulfite-seq/NOMe-seq (SNP/Methylation calling: BisSNP; HMM segmentation: DMNTools; Visualization/Clustering: alignWigToBed; )
- [rambaut/figtree](https://github.com/rambaut/figtree) - Automatically exported from code.google.com/p/figtree
- [zm-git-dev/systemsgenetics](https://github.com/zm-git-dev/systemsgenetics) - Generic Java genotype reader / writer, QTL mapping software, Strand alignment tool
- [zm-git-dev/FastQC](https://github.com/zm-git-dev/FastQC) - A quality control analysis tool for high throughput sequencing data
- [zm-git-dev/NLR-Parser](https://github.com/zm-git-dev/NLR-Parser) - Annotation Pipeline for NLR genes
- [zm-git-dev/VGTool](https://github.com/zm-git-dev/VGTool) - A tool for modelling graphs of genetic variation.
- [zm-git-dev/vcfstats-2](https://github.com/zm-git-dev/vcfstats-2) - BGI-lowpass imputation vcf stats
- [zm-git-dev/NGSEPcore](https://github.com/zm-git-dev/NGSEPcore) - NGSEP is an integrated framework for analysis of high throughput sequencing (HTS) reads. The main functionality of NGSEP is the variants detector, which allows to make integrated discovery and genotyp
- [zm-git-dev/eoulsan](https://github.com/zm-git-dev/eoulsan) - A pipeline and a framework for NGS analysis (RNA-Seq and soon Chip-Seq)

## JavaScript 

- [IkoSong/myownweddingapp](https://github.com/IkoSong/myownweddingapp) - å°ç°ç°å©šç¤¼è¯·æŸ¬å°ç¨‹åº
- [zerosoul/github-star-stats](https://github.com/zerosoul/github-star-stats) - awesome GitHub repo star statistics generator
- [zerosoul/tristan-wedding](https://github.com/zerosoul/tristan-wedding) - ç»™è‡ªå·±åšçš„å©šç¤¼é¡µé¢
- [KunChiang/wedding](https://github.com/KunChiang/wedding) - å©šç¤¼è¯·æŸ¬å¾®ä¿¡å°ç¨‹åº
- [ligenhw/invitation-mp](https://github.com/ligenhw/invitation-mp) - å¼€æºå…è´¹çš„ å©šç¤¼é‚€è¯·å‡½ å°ç¨‹åº
- [putyy/weddingInvitation](https://github.com/putyy/weddingInvitation) - ä¸€ä¸ªç®€å•çš„å©šç¤¼é‚€è¯·å°ç¨‹åº
- [zenmin/zm-WeddingInvitations](https://github.com/zenmin/zm-WeddingInvitations) - å©šç¤¼è¯·æŸ¬å°ç¨‹åº
- [ahao430/wedding-miniapp](https://github.com/ahao430/wedding-miniapp) - æˆ‘çš„å©šç¤¼è¯·æŸ¬å°ç¨‹åº
- [blogwy/Wedding-invitation](https://github.com/blogwy/Wedding-invitation) - å©šç¤¼é‚€è¯·å‡½å¾®ä¿¡å°ç¨‹åºï¼Œåç«¯ä½¿ç”¨äº‘å¼€å‘ã€‚ä¸éœ€è¦æœåŠ¡å™¨å’ŒåŸŸå
- [tanshuimaohenquejiao/WeWedding](https://github.com/tanshuimaohenquejiao/WeWedding) - å©šçº±å½±æ¥¼å°ç¨‹åºæä¾›äº†ä¸€ä¸ªè¿æ¥ç”¨æˆ·ä¸å½±æ¥¼çš„å¹³å°ï¼Œç›¸å½“äºå½±æ¥¼åœ¨å¾®ä¿¡çš„å®˜ç½‘ã€‚å®ƒèƒ½å¸®åŠ©å½±æ¥¼å±•ç¤ºæ‹æ‘„å®åŠ›ï¼Œè®°å½•è®¿å®¢æ•°æ®ï¼Œå®£ä¼ ä¼˜æƒ æ´»åŠ¨ã€‚ä½¿ç”¨é¢‘ç‡é«˜ï¼Œæ–¹ä¾¿ä¼ æ’­ï¼Œæ˜¯å½±æ¥¼åœ¨å¾®ä¿¡ç«¯å®£ä¼ è¥é”€çš„å¾—åŠ›åŠ©æ‰‹ã€‚å…¶é‡‡ç”¨è…¾è®¯æä¾›çš„å°ç¨‹åºäº‘å¼€å‘è§£å†³æ–¹æ¡ˆï¼Œæ— é¡»æœåŠ¡å™¨å’ŒåŸŸåã€‚  æ ·ç‰‡é¡µæ˜¯å½±æ¥¼å±•ç¤ºä¼˜ç§€æ‘„å½±æ ·ç‰‡æä¾›ç»™ç”¨æˆ·æ¬£èµå¹¶ä¸”å¸å¼•å®¢æˆ·çš„ã€‚å¥—ç³»é¡µæ˜¯å½±æ¥¼æ ¹æ®å¸‚åœºéœ€æ±‚æ¨å‡ºçš„ä¸åŒå¥—é¤ï¼Œç”¨æˆ·å¯ä»¥æŒ‰ç…§è‡ªå·±çš„å–œå¥½é¢„å®šå¥—ç³»ã€‚ä¸ªäººä¸­å¿ƒå¯ä»¥æŸ¥çœ‹ç”¨æˆ·é¢„çº¦çš„æ‹æ‘„
- [pengMaster/marry](https://github.com/pengMaster/marry) - :fire: å©šç¤¼é‚€è¯·å‡½ - å©šæŸ¬ - å¾®ä¿¡å°ç¨‹åº - å°ç¨‹åºç«¯    wedding
- [airufei/wechat-marry](https://github.com/airufei/wechat-marry) - å©šç¤¼ç›¸å…³å¾®ä¿¡å°ç¨‹åº
- [BakerYw/marry_nk](https://github.com/BakerYw/marry_nk) - â¤ï¸è‡ªå·±çš„å©šç¤¼å°ç¨‹åºæºç 
- [ChaseMillers/Eastside-Wedding-Videography](https://github.com/ChaseMillers/Eastside-Wedding-Videography) - 
- [sun0225SUN/Awesome-Love-Code](https://github.com/sun0225SUN/Awesome-Love-Code) - è¡¨ç™½ä»£ç æ”¶è—é¦†~è°è¯´ç¨‹åºçŒ¿ä¸æ‡‚æµªæ¼«â¤ï¸
- [SouthGreenPlatform/GeMo](https://github.com/SouthGreenPlatform/GeMo) - A mosaic genome painting tool for plant genomes
- [nslogx/Gitter](https://github.com/nslogx/Gitter) - Gitter for GitHub - å¯èƒ½æ˜¯ç›®å‰é¢œå€¼æœ€é«˜çš„GitHubå¾®ä¿¡å°ç¨‹åºå®¢æˆ·ç«¯
- [vega/vega](https://github.com/vega/vega) - A visualization grammar.
- [qiaoba0728/resequencing](https://github.com/qiaoba0728/resequencing) - 
- [wan230114/PythonNote](https://github.com/wan230114/PythonNote) - Pythonå­¦ä¹ ä¹‹æç®€ç¬”è®°â€”â€”ä»é›¶åˆ°å‚¬ç”ŸAGI
- [Liwu-di/PaperCrawlerUtil](https://github.com/Liwu-di/PaperCrawlerUtil) - ä¸€å¥—å·¥å…·ç»„ï¼ŒåŒ…æ‹¬è®¿é—®é“¾æ¥ï¼Œ è·å–å…ƒç´ ï¼ŒæŠ½å–æ–‡ä»¶ç­‰ç­‰ ä¹Ÿæœ‰å·²ç»å®ç°å¥½é€šè¿‡scihubè·å–è®ºæ–‡çš„å°å·¥å…·ï¼Œè¿˜æœ‰å¯¹äºpdfè½¬docï¼Œæ–‡æœ¬ç¿»è¯‘,ä»£ç†è¿æ¥è·å–ä»¥åŠé€šè¿‡apiè·å–ä»£ç†é“¾æ¥ï¼Œ PDFæ–‡ä»¶åˆå¹¶ï¼ŒPDFæ–‡ä»¶æˆªå–æŸäº›é¡µï¼ŒCSVï¼Œxlsæ–‡ä»¶å¤„ç†ç­‰
- [mhyfritz/maze](https://github.com/mhyfritz/maze) - maze: match visualizer | visually align dna sequences using exact matches
- [kiranbandi/synvisio](https://github.com/kiranbandi/synvisio) - SynVisio is an interactive multiscale visualization tool that lets you explore the results of McScanX a popular synteny and collinearity detection toolkit.
- [nvkelso/illustrator-scripts](https://github.com/nvkelso/illustrator-scripts) - JavaScript ExtendScript for automating and extending Adobe Illustrator
- [Przemol/seqplots](https://github.com/Przemol/seqplots) - :chart_with_upwards_trend:SeqPlots - An interactive tool for visualizing NGS signals and sequence motif densities along genomic features using average plots and heatmaps.
- [rjchallis/assembly-stats](https://github.com/rjchallis/assembly-stats) - Assembly statistic visualisation
- [vgteam/sequenceTubeMap](https://github.com/vgteam/sequenceTubeMap) - displays multiple genomic sequences in the form of a tube map
- [TF-Chan-Lab/panGraphViewer](https://github.com/TF-Chan-Lab/panGraphViewer) - Show pangenome graphs in an easy way
- [schatzlab/genomescope](https://github.com/schatzlab/genomescope) - Fast genome analysis from unassembled short reads
- [ZijieJin/zijiejin.github.io](https://github.com/ZijieJin/zijiejin.github.io) - Github Pages template for academic personal websites, forked from mmistakes/minimal-mistakes
- [jkimlab/mySyntenyPortal](https://github.com/jkimlab/mySyntenyPortal) - A stand-alone application for constructing websites for visualizing and browsing synteny blocks
- [plasma-umass/scalene](https://github.com/plasma-umass/scalene) - Scalene: a high-performance, high-precision CPU, GPU, and memory profiler for Python with AI-powered optimization proposals
- [stothard-group/cgview-js](https://github.com/stothard-group/cgview-js) - CGView.js creates interactive circular and linear genome maps.
- [resume/resume.github.com](https://github.com/resume/resume.github.com) - Resumes generated using the GitHub informations
- [zm-git-dev/Morris-Lab](https://github.com/zm-git-dev/Morris-Lab) - python snippets for analyzing short reads
- [zm-git-dev/ppatens_db](https://github.com/zm-git-dev/ppatens_db) - P patens gene model lookup database
- [zm-git-dev/synvisio](https://github.com/zm-git-dev/synvisio) - SynVisio is an interactive multiscale visualization tool that lets you explore the results of McScanX a popular synteny and collinearity detection toolkit.
- [zm-git-dev/msa_tubemap](https://github.com/zm-git-dev/msa_tubemap) - Visualize multiple sequence alignment as variation graph on Sequence Tube Map
- [neherlab/pan-genome-visualization](https://github.com/neherlab/pan-genome-visualization) - Visualization of the pan-genome output by panX
- [Sergiodiaz53/GeckoMGV](https://github.com/Sergiodiaz53/GeckoMGV) - Multigenome viewer in Django and JS - University of MÃ¡laga
- [JameyZhu/libpku](https://github.com/JameyZhu/libpku) - åŒ—äº¬å¤§å­¦è¯¾ç¨‹èµ„æ–™æ•´ç†
- [cehui0303/Tmall_Tickets](https://github.com/cehui0303/Tmall_Tickets) - å¤©çŒ«è¶…å¸‚èŒ…å°æŠ¢ç¥¨åŠŸèƒ½
- [iDataVisualizationLab/BioCluster](https://github.com/iDataVisualizationLab/BioCluster) - 
- [biodataprog/2017_programming-intro](https://github.com/biodataprog/2017_programming-intro) - Genetics 220-2017 - Computational Analysis of High Throughput Biological Data
- [rstudio/shiny-examples](https://github.com/rstudio/shiny-examples) - 
- [darencard/darencard.github.io](https://github.com/darencard/darencard.github.io) - Daren Card's professional website
- [ArchB1aZe/GalaxyTool](https://github.com/ArchB1aZe/GalaxyTool) - New Tool
- [zm-git-dev/Perl_programme](https://github.com/zm-git-dev/Perl_programme) - Working programme for Genedenovo

## Julia 

- [neherlab/pangraph](https://github.com/neherlab/pangraph) - A bioinformatic toolkit to align genome assemblies into pangenome graphs
- [cihga39871/Atria](https://github.com/cihga39871/Atria) - An accurate and ultra-fast adapter and quality trimming program for Illumina Next-Generation Sequencing (NGS) data.
- [ChelseaTrotter/fastlmm-gpu](https://github.com/ChelseaTrotter/fastlmm-gpu) - 
- [zm-git-dev/pangraph](https://github.com/zm-git-dev/pangraph) - A bioinformatic toolkit to align genome assemblies into pangenome graphs
- [ShadrackOdikara/GWAS_scrapping](https://github.com/ShadrackOdikara/GWAS_scrapping) - Parse through a GWAS summary statistics results identify obtain the significant SNPs and search the reference Genome 50 base pairs upstream and 50 base pair down stream for canditdate genes

## Jupyter Notebook 

- [ethanchen1301/SSDraw](https://github.com/ethanchen1301/SSDraw) - 
- [umccr/vcf_stuff](https://github.com/umccr/vcf_stuff) - ğŸ“ŠEvaluating, filtering, comparing, and visualising VCF
- [AliaksandrSiarohin/vitvin_genes](https://github.com/AliaksandrSiarohin/vitvin_genes) - Vitis vinifera pathway expansion
- [sex-group/wild-grapevine-genome](https://github.com/sex-group/wild-grapevine-genome) - 
- [gagneurlab/MMSplice_MTSplice](https://github.com/gagneurlab/MMSplice_MTSplice) - Tissue-specific variant effect predictions on splicing
- [jokergoo/PyComplexHeatmap](https://github.com/jokergoo/PyComplexHeatmap) - PyComplexHeatmap: A Python package to plot complex heatmap (clustermap)
- [fayjustin/computational_biology](https://github.com/fayjustin/computational_biology) - course material
- [theislab/single-cell-best-practices](https://github.com/theislab/single-cell-best-practices) - https://www.sc-best-practices.org
- [cakmakaf/fetch_PubMed_abstracts_by_keyword](https://github.com/cakmakaf/fetch_PubMed_abstracts_by_keyword) - It provides a method of how to fetch articles along with their features and text from PubMed API.
- [bmmoore43/Gene_coexpression_MR](https://github.com/bmmoore43/Gene_coexpression_MR) - 
- [eaton-lab/toytree](https://github.com/eaton-lab/toytree) - A minimalist tree plotting library using toyplot graphs
- [ZJUFanLab/bulk2space](https://github.com/ZJUFanLab/bulk2space) - a spatial deconvolution method based on deep learning frameworks, which converts bulk transcriptomes into spatially resolved single-cell expression profiles
- [NBISweden/workshop-scRNAseq](https://github.com/NBISweden/workshop-scRNAseq) - Single cell RNA sequencing analysis course
- [timeu/gwas-lecture](https://github.com/timeu/gwas-lecture) - Lectures on Genome Wide Association Studies (GWAS) using Python, Limix and Jupyter Noteboos
- [liyu95/Deep_learning_examples](https://github.com/liyu95/Deep_learning_examples) - Examples of using deep learning in Bioinformatics
- [aariani/bioinformatics_pipeline](https://github.com/aariani/bioinformatics_pipeline) - Space for development of bioinformatic pipelines
- [KULL-Centre/papers](https://github.com/KULL-Centre/papers) - Material from papers from KULL centre
- [ueser/FIDDLE](https://github.com/ueser/FIDDLE) - Flexible Integration of Data with Deep LEarning
- [GoekeLab/sg-nex-data](https://github.com/GoekeLab/sg-nex-data) - Nanopore RNA-Seq data from the Singapore Nanopore-Expression Project
- [Ran485/Integrated_proteogenomic_analysis_CCA](https://github.com/Ran485/Integrated_proteogenomic_analysis_CCA) - This repository is meant to accompany the manuscript "Proteogenomic Characterization of Cholangiocarcinoma." It includes all of the analysis and libraries to generate the figures and computations in t
- [mortazavilab/PyWGCNA](https://github.com/mortazavilab/PyWGCNA) - PyWGCNA is a Python package designed to do Weighted Gene Correlation Network analysis (WGCNA)
- [eporetsky/syntenyZ](https://github.com/eporetsky/syntenyZ) - Custom scripts to make comparative genomic plots using genoPlotR
- [zm-git-dev/Onion_BSA_RNASEQ_Workflow](https://github.com/zm-git-dev/Onion_BSA_RNASEQ_Workflow) - Workflow for Analysis of Onion Bulked Segregant Analysis RNASEQ
- [zm-git-dev/AdvancedPythonForBio](https://github.com/zm-git-dev/AdvancedPythonForBio) - Work from the book 'Advanced Python for Biologists'
- [zm-git-dev/pydata-notebook](https://github.com/zm-git-dev/pydata-notebook) - åˆ©ç”¨Pythonè¿›è¡Œæ•°æ®åˆ†æ ç¬¬äºŒç‰ˆ (2017) ä¸­æ–‡ç¿»è¯‘ç¬”è®°
- [zm-git-dev/rnacocktail](https://github.com/zm-git-dev/rnacocktail) - 
- [zm-git-dev/qbb2018-answers](https://github.com/zm-git-dev/qbb2018-answers) - 
- [zm-git-dev/Python-100-Days](https://github.com/zm-git-dev/Python-100-Days) - Python - 100å¤©ä»æ–°æ‰‹åˆ°å¤§å¸ˆ
- [zm-git-dev/talks](https://github.com/zm-git-dev/talks) - Repository of the talks at the [Ashworth Code Monkeys](https://groups.google.com/forum/#!forum/ashworth-code-monkeys) meetings
- [zm-git-dev/Cat-Does-Plant](https://github.com/zm-git-dev/Cat-Does-Plant) - Cat doing plant science.
- [zm-git-dev/syntenyZ](https://github.com/zm-git-dev/syntenyZ) - Custom scripts to make comparative genomic plots using genoPlotR
- [zm-git-dev/hgdp_tgp](https://github.com/zm-git-dev/hgdp_tgp) - 
- [zm-git-dev/philipempl.de](https://github.com/zm-git-dev/philipempl.de) - Netlify website
- [zm-git-dev/matplotlib-venn](https://github.com/zm-git-dev/matplotlib-venn) - Area-weighted venn-diagrams for Python/matplotlib
- [zm-git-dev/The-Python-Graph-Gallery](https://github.com/zm-git-dev/The-Python-Graph-Gallery) - A website displaying hundreds of charts made with Python
- [zm-git-dev/bioinformatics-repo](https://github.com/zm-git-dev/bioinformatics-repo) - 
- [zm-git-dev/my_binder](https://github.com/zm-git-dev/my_binder) - 
- [LankyCyril/pyvenn](https://github.com/LankyCyril/pyvenn) - Python module for plotting Venn diagrams of 2..6 sets
- [katewall/medium_tutorials](https://github.com/katewall/medium_tutorials) - 
- [DeadlineWasYesterday/Cat-Does-Plant](https://github.com/DeadlineWasYesterday/Cat-Does-Plant) - Cat doing plant science.
- [Uauy-Lab/WheatHomoeologExpression](https://github.com/Uauy-Lab/WheatHomoeologExpression) - 
- [danshea/IwateBiotechnologyResearchCenter](https://github.com/danshea/IwateBiotechnologyResearchCenter) - Python stuff from my current (circa 2019) work into analyzing rice GWAS data sets.
- [yevheniyc/Projects](https://github.com/yevheniyc/Projects) - Suit of projects to reinforce my self-study curriculum and other endeavors
- [jiankaiwang/sophia](https://github.com/jiankaiwang/sophia) - The sophia project is the coding-based tutorial for machine intelligence (MI) and artificial intelligence (AI).

## Kotlin 

- [2dust/v2rayNG](https://github.com/2dust/v2rayNG) - A V2Ray client for Android, support Xray core and v2fly core
- [bannedbook/fanqiang](https://github.com/bannedbook/fanqiang) - ç¿»å¢™-ç§‘å­¦ä¸Šç½‘

## M 

- [zm-git-dev/Notebook](https://github.com/zm-git-dev/Notebook) - 

## MATLAB 

- [Henrik86/Mow_Joe](https://github.com/Henrik86/Mow_Joe) - MowJoe is a software for the automatic phenotyping of compound leaves. The software automatically separates the whole leaf  into leaflets and calculates spatial statistics between the leaflets.

## Makefile 

- [zm-git-dev/TSNAD_update](https://github.com/zm-git-dev/TSNAD_update) - 

## Nextflow 

- [xmuyulab/scRNAseq_pipelines](https://github.com/xmuyulab/scRNAseq_pipelines) - 
- [nf-core/hic](https://github.com/nf-core/hic) - Analysis of Chromosome Conformation Capture data (Hi-C)
- [digenoma-lab/bam2cram](https://github.com/digenoma-lab/bam2cram) - A repository to convert BAM-&gt;CRAM files
- [nf-core/chipseq](https://github.com/nf-core/chipseq) - ChIP-seq peak-calling, QC and differential analysis pipeline.
- [zm-git-dev/rnafusion](https://github.com/zm-git-dev/rnafusion) - RNA-seq analysis pipeline for detection gene-fusions
- [zm-git-dev/meripseqpipe](https://github.com/zm-git-dev/meripseqpipe) - MeRIP-seq analysis pipeline arranged multiple alignment tools, peakCalling tools, Merge Peaks' methods and methylation analysis methods.
- [zm-git-dev/Maize_WGS_Build](https://github.com/zm-git-dev/Maize_WGS_Build) - Code and Data to include in WGS Build
- [zm-git-dev/crisprseq](https://github.com/zm-git-dev/crisprseq) - [Under construction] A pipeline for all crispr analyses
- [gencorefacility/variant-calling-pipeline-gatk4](https://github.com/gencorefacility/variant-calling-pipeline-gatk4) - Variant Calling Pipeline Using GATK4 and Nextflow
- [HuffordLab/Maize_WGS_Build](https://github.com/HuffordLab/Maize_WGS_Build) - Code and Data to include in WGS Build
- [nf-core/rnafusion](https://github.com/nf-core/rnafusion) - RNA-seq analysis pipeline for detection gene-fusions

## Nim 

- [telatin/seqfu2](https://github.com/telatin/seqfu2) - :rocket: seqfu - Sequece Fastx Utilities
- [brentp/slivar](https://github.com/brentp/slivar) - genetic variant expressions, annotation, and filtering for great good.

## OpenEdge ABL 

- [zm-git-dev/bioinf-python](https://github.com/zm-git-dev/bioinf-python) - Python for Bioinformatics

## Others 

- [Shenwei625/Length-variation-in-short-tandem-repeats-affects-gene-expression](https://github.com/Shenwei625/Length-variation-in-short-tandem-repeats-affects-gene-expression) - 
- [ayueme/ayueme.github.io](https://github.com/ayueme/ayueme.github.io) - Rè¯­è¨€å®æˆ˜åŒ»å­¦ç»Ÿè®¡
- [vdclab/vdclab-wiki](https://github.com/vdclab/vdclab-wiki) - 
- [Immortal2333/Telomeres_and_Centromeres](https://github.com/Immortal2333/Telomeres_and_Centromeres) - This is a method to find telomeres and centromeres in plants.
- [Immortal2333/Diploid_Genome_Assembly](https://github.com/Immortal2333/Diploid_Genome_Assembly) - Genome haplotype assembly process and evaluation method
- [zhouyflab/Grapevine_Adaptive_Maladaptive_Introgression](https://github.com/zhouyflab/Grapevine_Adaptive_Maladaptive_Introgression) - Adaptive and maladaptive introgression in grapevine domestication
- [zhouyflab/PN40024_T2T_Genome](https://github.com/zhouyflab/PN40024_T2T_Genome) - The complete reference genome for grapevine (Vitis vinifera L.) genetics and breeding
- [Aryan092/Advanced-Data-Analyisis-Project](https://github.com/Aryan092/Advanced-Data-Analyisis-Project) - Analyze Whole Genome Bi-Sulfite Sequencing for Grape evolution in response to grafting
- [jgrimplet/Grapevine_genome_annotation](https://github.com/jgrimplet/Grapevine_genome_annotation) - 
- [justjavac/awesome-wechat-weapp](https://github.com/justjavac/awesome-wechat-weapp) - å¾®ä¿¡å°ç¨‹åºå¼€å‘èµ„æºæ±‡æ€» :100:
- [weixuan2008/MyNotes](https://github.com/weixuan2008/MyNotes) - 
- [xx025/carrot](https://github.com/xx025/carrot) - Free ChatGPT Site List è¿™å„¿ä¸ºä½ å‡†å¤‡äº†ä¼—å¤šå…è´¹å¥½ç”¨çš„ChatGPTé•œåƒç«™ç‚¹
- [warelab/release-notes](https://github.com/warelab/release-notes) - 
- [ZhihaoXie/awesome-bioinformatics-tools](https://github.com/ZhihaoXie/awesome-bioinformatics-tools) - A curated list of awesome Bioinformatics software, tools and resources
- [gis-xh/my-note](https://github.com/gis-xh/my-note) - è¿™æ˜¯ä¸€ä¸ªä½¿ç”¨ markdown çš„è®°å½•å¼€å‘å­¦ä¹ çš„ç¬”è®°ä»“åº“ã€‚
- [jiasun8690/Whole-transcriptome_RNA](https://github.com/jiasun8690/Whole-transcriptome_RNA) - 
- [cxli233/ggpathway](https://github.com/cxli233/ggpathway) - A tutorial for pathway visualization using tidyverse, igraph, and ggraph
- [PacificBiosciences/pbtk](https://github.com/PacificBiosciences/pbtk) - PacBio BAM toolkit
- [CAUJinlab/SV-caller](https://github.com/CAUJinlab/SV-caller) - Structural variation caller collection
- [harvardinformatics/learning-bioinformatics-at-home](https://github.com/harvardinformatics/learning-bioinformatics-at-home) - resources for learning bioinformatics
- [GuShuLongHun/sheng-xin-xiao-bai-yu](https://github.com/GuShuLongHun/sheng-xin-xiao-bai-yu) - å†å²æ–‡æ¡£å¤‡ä»½
- [fudiyi/GWAS-QTL-realted](https://github.com/fudiyi/GWAS-QTL-realted) - QTL/GWASåˆ†æç›¸å…³è„šæœ¬
- [onceupon/Bash-Oneliner](https://github.com/onceupon/Bash-Oneliner) - A collection of handy Bash One-Liners and terminal tricks for data processing and Linux system maintenance.
- [tianheg/stars](https://github.com/tianheg/stars) - All my stars repositories
- [Yedomon/Sequencing101](https://github.com/Yedomon/Sequencing101) - 
- [Sydney-Informatics-Hub/Bioinformatics](https://github.com/Sydney-Informatics-Hub/Bioinformatics) - A suite of bioinformatics data processing and analysis pipelines, software, and training resources for common DNA and RNA sequence analysis methods.
- [timedreamer/useful_note](https://github.com/timedreamer/useful_note) - Some useful note to myself (hopefully to others too) that I keep update.
- [xmuyulab/YuLab_Tutorial](https://github.com/xmuyulab/YuLab_Tutorial) - YuLab Linuxä¸ç”Ÿç‰©ä¿¡æ¯åŸºç¡€åŸ¹è®­
- [Ming-Lian/NGS-analysis](https://github.com/Ming-Lian/NGS-analysis) - äºŒä»£æµ‹åºæ•°æ®åˆ†æ
- [ChenShun-git/scRNA-analysis](https://github.com/ChenShun-git/scRNA-analysis) - 
- [zpeng1989/Catalogue_by_zpeng1989](https://github.com/zpeng1989/Catalogue_by_zpeng1989) - è‡ªå·±é¡¹ç›®æ•´ç†çš„ç›®å½•æ–‡ä»¶ï¼Œæ–¹ä¾¿æŸ¥æ‰¾ï¼ŒæŒç»­æ›´æ–°ä¸­
- [xin-huang/PopGenTools](https://github.com/xin-huang/PopGenTools) - A collection for available tools in population genetics
- [stemcellcommons/scc-galaxy-tools](https://github.com/stemcellcommons/scc-galaxy-tools) - Custom Galaxy tools and workflows
- [cxli233/SimpleTidy_GeneCoEx](https://github.com/cxli233/SimpleTidy_GeneCoEx) - A simple gene co-expression analyses workflow powered by tidyverse and graph analyses
- [cxli233/ggtraces](https://github.com/cxli233/ggtraces) - A tidyverse and grammar of graphics powered line traces visualizer
- [wassermanlab/gene2wordclouds](https://github.com/wassermanlab/gene2wordclouds) - 
- [lizhao007/DeepBSA](https://github.com/lizhao007/DeepBSA) - DeepBSA is a novel bulked segregant analysis (BSA) software for the dissection of complex traits.
- [XiaoleiLiuBio/G2P](https://github.com/XiaoleiLiuBio/G2P) - G2P: A Genome-Wide-Association-Study Simulation Tool for Genotype Simulation, Phenotype Simulation, and Power Evaluation
- [sorrycc/awesome-javascript](https://github.com/sorrycc/awesome-javascript) - ğŸ¢ A collection of awesome browser-side  JavaScript libraries, resources and shiny things.
- [MGI-tech-bioinformatics/stLFR_V1.3](https://github.com/MGI-tech-bioinformatics/stLFR_V1.3) - Tools of stLFR(Single Tube Long Fragment Reads) Resequencing data analysis.
- [dongwei1220/Single-cell-data-visualization-tools](https://github.com/dongwei1220/Single-cell-data-visualization-tools) - Collection of single-cell data visualization tools.
- [Shicheng-Guo/methylation](https://github.com/Shicheng-Guo/methylation) - 
- [Shicheng-Guo/tumorpurity](https://github.com/Shicheng-Guo/tumorpurity) - How to estimate tumor purity with WES and methylation data
- [Shicheng-Guo/tcgaMethylationSubset](https://github.com/Shicheng-Guo/tcgaMethylationSubset) - A subset of the DNA mehtylation data in TCGA
- [mahyam/DNA-methylation-and-metabolism-R-code](https://github.com/mahyam/DNA-methylation-and-metabolism-R-code) - R script used in the manuscript entitled "An integrated analysis of tumor DNA methylation variation identifies a role for metabolism"
- [zm-git-dev/Hosts](https://github.com/zm-git-dev/Hosts) - :bowtie:æ— éšœç¢ä¸Šç½‘-ä½ ~æ‡‚å¾—ï¼[æ›´æ–°é¢‘ç‡7dï¼Œéœ€è¦è¯· fork]
- [zm-git-dev/PopLDdecay](https://github.com/zm-git-dev/PopLDdecay) - PopLDdecay: A new simple and efficient software for Linkage Disequilibrium Decay analysis based Variant Call Format
- [zm-git-dev/hzcdc](https://github.com/zm-git-dev/hzcdc) - 
- [zm-git-dev/forum](https://github.com/zm-git-dev/forum) - é‡‡ç”¨å¼€æºè®ºå›ç³»ç»ŸFlarumå»ºè®¾GATKä¸­æ–‡ç¤¾åŒºï¼Œè¯¥è®ºå›æ‰€æœ‰å¸–å­å°†ä»¥markdownä»£ç å½¢å¼å‘å¸–ï¼Œæ‰€æœ‰çš„markdownæºä»£ç éƒ½ä¼šåŒæ­¥å­˜æ”¾åœ¨æ­¤githubçš„organizationçš„è¿™ä¸ªprojectä¸‹é¢ã€‚GATKä¸­æ–‡ç¤¾åŒºåœ°å€æ˜¯ï¼šhttps://www.gatk.com.cn/ å¯¹åº”çš„broadçš„GATKä¸»é¡µæ˜¯ï¼›https://software.broadinstitute.org/gatk æœ¬ç«™
- [zm-git-dev/Annotation_GO.KEGG.COG](https://github.com/zm-git-dev/Annotation_GO.KEGG.COG) - æ•´åˆå‹çš„å¯¹äºç”Ÿç‰©å­¦æ³¨é‡Šçš„ä¿¡æ¯/GOå’ŒKEGGå…¨è°±æ³¨é‡Šå’Œå¯Œé›†
- [zm-git-dev/RNA-Seq-1](https://github.com/zm-git-dev/RNA-Seq-1) - RNASeq analysis
- [zm-git-dev/BamCoverage](https://github.com/zm-git-dev/BamCoverage) - BamCoverage: A efficient software tools to calculate every site depth of genome based bam files
- [zm-git-dev/Life](https://github.com/zm-git-dev/Life) - Life
- [zm-git-dev/Awesome-Bioinformatics](https://github.com/zm-git-dev/Awesome-Bioinformatics) - A curated list of awesome Bioinformatics libraries and software.
- [zm-git-dev/Bio](https://github.com/zm-git-dev/Bio) - 
- [zm-git-dev/Memo](https://github.com/zm-git-dev/Memo) - å­¦ä¹ ç¬”è®°ï¼šæˆ‘çš„ç¬¬ä¸€ä¸ªgithubä»“åº“
- [zm-git-dev/oneliners](https://github.com/zm-git-dev/oneliners) - Useful bash one-liners for bioinformatics.
- [zm-git-dev/Data-mining](https://github.com/zm-git-dev/Data-mining) - æ•°æ®æŒ–æ˜+çˆ¬è™«
- [zm-git-dev/align2rawsignal](https://github.com/zm-git-dev/align2rawsignal) - A.K.A. WIGGLER: Creates genome-wide raw or normalized signal tracks from aligned sequencing reads (BAM/tagAlign)
- [zm-git-dev/biotools](https://github.com/zm-git-dev/biotools) - A list of useful bioinformatics resources
- [zm-git-dev/Cookbook-for-R-Chinese](https://github.com/zm-git-dev/Cookbook-for-R-Chinese) - :books:Cookbook for R  ä¸­æ–‡ç¿»è¯‘
- [zm-git-dev/G2P](https://github.com/zm-git-dev/G2P) - G2P: A Genome-Wide-Association-Study Simulation Tool for Genotype Simulation, Phenotype Simulation, and Power Evaluation
- [zm-git-dev/NGS-analysis](https://github.com/zm-git-dev/NGS-analysis) - äºŒä»£æµ‹åºæ•°æ®åˆ†æ
- [zm-git-dev/deeplearning-biology](https://github.com/zm-git-dev/deeplearning-biology) - A list of deep learning implementations in biology
- [zm-git-dev/ngs_pipeline_by_snakemake](https://github.com/zm-git-dev/ngs_pipeline_by_snakemake) - writing ngs_pipeline scripts by snakemake
- [zm-git-dev/Biotools_notes](https://github.com/zm-git-dev/Biotools_notes) - 
- [zm-git-dev/XSLab-docs](https://github.com/zm-git-dev/XSLab-docs) - ä¸Šç§‘å¤§åˆ˜é›ªæ¾è¯¾é¢˜ç»„åŸºå› ç»„å­¦åˆ†æä¼ æ‰¿ä¸åˆ†äº«çŸ¥è¯†åº“
- [zm-git-dev/Program_scripts](https://github.com/zm-git-dev/Program_scripts) - 
- [zm-git-dev/bioinformatics-one-liners](https://github.com/zm-git-dev/bioinformatics-one-liners) - Bioinformatics one liners from Ming Tang
- [zm-git-dev/Ostracoda_genomes](https://github.com/zm-git-dev/Ostracoda_genomes) - Code for "First annotated draft genomes of non-marine ostracods (Ostracoda, Crustacea) with different reproductive modes"  manuscript.
- [zm-git-dev/Genetic-Map](https://github.com/zm-git-dev/Genetic-Map) - High-Density Genetic Map Construction and Identification of QTLs Controlling Leaf Abscission Trait in Poncirus trifoliata
- [zm-git-dev/Genetic](https://github.com/zm-git-dev/Genetic) - High-Density Genetic Map Construction and Identification of QTLs Controlling Leaf Abscission Trait in Poncirus trifoliata
- [zm-git-dev/PanSN-spec](https://github.com/zm-git-dev/PanSN-spec) - Pangenome Sequence Naming: a backwards-compatible hack to simplify the identification of samples and haplotypes in pangenomes
- [zm-git-dev/gene-family-identify](https://github.com/zm-git-dev/gene-family-identify) - blastp+hmmsearch+CDD
- [zm-git-dev/NGS-analysis-old](https://github.com/zm-git-dev/NGS-analysis-old) - 
- [zm-git-dev/Bioinformatics-Servers](https://github.com/zm-git-dev/Bioinformatics-Servers) - æ”¶é›†ç”Ÿç‰©ä¸“ä¸šç§‘ç ”æ•°æ®åˆ†æä¸­ä½¿ç”¨çš„åœ¨çº¿ç”Ÿç‰©ä¿¡æ¯å­¦åˆ†ææœåŠ¡ï¼ŒæŒ‰åˆ†å­ç±»å‹(DNAã€RNAã€Protein)å’Œåˆ†æé¢†åŸŸè¿›è¡Œæ•´ç†å’Œåˆ†ç±»ï¼
- [zm-git-dev/pangenome](https://github.com/zm-git-dev/pangenome) - 
- [zm-git-dev/Bash-Oneliner](https://github.com/zm-git-dev/Bash-Oneliner) - A collection of handy Bash One-Liners and terminal tricks for data processing and Linux system maintenance.
- [zm-git-dev/awesome](https://github.com/zm-git-dev/awesome) - ğŸ˜ Awesome lists about all kinds of interesting topics
- [zm-git-dev/sv-cnv-studies](https://github.com/zm-git-dev/sv-cnv-studies) - :books: Relevant papers for CNV and SV approaches
- [lyao222lll/sheng-xin-xiao-bai-yu-2022](https://github.com/lyao222lll/sheng-xin-xiao-bai-yu-2022) - å†å²æ–‡æ¡£å¤‡ä»½ï¼ˆ2022å¹´ï¼‰
- [robynbutler6/MSc-Thesis](https://github.com/robynbutler6/MSc-Thesis) - Code written in R Studio for the analysis presented in my thesis submitted for MSc Microbiome in Health and Disease on genetic variation in group A streptococcus
- [spalalidelen2/GWAS_Rice](https://github.com/spalalidelen2/GWAS_Rice) - 
- [TypeFox/R-Examples](https://github.com/TypeFox/R-Examples) - 
- [zm-git-dev/favoriteRpackages](https://github.com/zm-git-dev/favoriteRpackages) - My favorite R packages
- [WGLab/biocluster](https://github.com/WGLab/biocluster) - Tutorial on building a computing cluster for bioinformatics
- [crazyhottommy/bioinformatics-one-liners](https://github.com/crazyhottommy/bioinformatics-one-liners) - Bioinformatics one liners from Ming Tang
- [sukecosine/RNA-seq](https://github.com/sukecosine/RNA-seq) - from 0-1
- [xuzhougeng/Genomic_tools](https://github.com/xuzhougeng/Genomic_tools) - Genomic related tools
- [FavioVazquez/ds-cheatsheets](https://github.com/FavioVazquez/ds-cheatsheets) - List of Data Science Cheatsheets to rule the world
- [Yixiaohan/codeparkshare](https://github.com/Yixiaohan/codeparkshare) - Pythonåˆå­¦è€…ï¼ˆé›¶åŸºç¡€å­¦ä¹ Pythonã€Pythonå…¥é—¨ï¼‰ä¹¦ç±ã€è§†é¢‘ã€èµ„æ–™ã€ç¤¾åŒºæ¨è
- [ZoeSj/ZD_Data_Structure](https://github.com/ZoeSj/ZD_Data_Structure) - æµ™å¤§æ•°æ®ç»“æ„æ•™ç¨‹
- [z3tt/ggplot-courses](https://github.com/z3tt/ggplot-courses) - ğŸ‘¨â€ğŸ«  ggplot2 Teaching Material
- [gao-lab/Guideline-for-Computational-Biology-and-Bioinformatics](https://github.com/gao-lab/Guideline-for-Computational-Biology-and-Bioinformatics) - 
- [ossu/bioinformatics](https://github.com/ossu/bioinformatics) - :microscope: Path to a free self-taught education in Bioinformatics!
- [bmansfeld/Yang2013data](https://github.com/bmansfeld/Yang2013data) - Data for use with QTLseqr vignette
- [DworkinLab/EvolveResequencePipeline](https://github.com/DworkinLab/EvolveResequencePipeline) - This is a repository that provides an overview, scripts, and helpful hints for doing our evolve and resequence analysis
- [afeiyuanda/help-doc](https://github.com/afeiyuanda/help-doc) - 
- [jlevy/the-art-of-command-line](https://github.com/jlevy/the-art-of-command-line) - Master the command line, in one page
- [piglei/one-python-craftsman](https://github.com/piglei/one-python-craftsman) - æ¥è‡ªä¸€ä½ Pythonista çš„ç¼–ç¨‹ç»éªŒåˆ†äº«ï¼Œå†…å®¹æ¶µç›–ç¼–ç æŠ€å·§ã€æœ€ä½³å®è·µä¸æ€ç»´æ¨¡å¼ç­‰æ–¹é¢ã€‚
- [benben-miao/NoteBook](https://github.com/benben-miao/NoteBook) - NoteBook
- [jameslz/oneliners](https://github.com/jameslz/oneliners) - Useful bash one-liners for bioinformatics.
- [stephenturner/oneliners](https://github.com/stephenturner/oneliners) - Useful bash one-liners for bioinformatics.
- [danielecook/Awesome-Bioinformatics](https://github.com/danielecook/Awesome-Bioinformatics) - A curated list of awesome Bioinformatics libraries and software.

## PHP 

- [zm-git-dev/tripal_synview](https://github.com/zm-git-dev/tripal_synview) - synteny viewer
- [zm-git-dev/PubMedPDF](https://github.com/zm-git-dev/PubMedPDF) - 
- [zm-git-dev/tripal](https://github.com/zm-git-dev/tripal) - The Tripal package is  a suite of Drupal modules for creating biological (genomic, genetic, breeding) websites. Visit the Tripal homepage at http://tripal.info for documentation, support, and other in

## PLpgSQL 

- [solgenomics/sgn](https://github.com/solgenomics/sgn) - The code behind the Sol Genomics Network, Cassavabase and other Breedbase websites

## Pascal 

- [pyscripter/SynEdit](https://github.com/pyscripter/SynEdit) - SynEdit is a syntax highlighting edit control, not based on the Windows common controls.

## Pep8 

- [zhangrengang/TEsorter](https://github.com/zhangrengang/TEsorter) - TEsorter: an accurate and fast method to classify LTR-retrotransposons in plant genomes

## Perl 

- [lixiang117423/PlantTribes](https://github.com/lixiang117423/PlantTribes) - PlantTribes is a collection of automated gene family analysis pipelines for comparative plant genomics
- [cyverse/docker-builds](https://github.com/cyverse/docker-builds) - A place to put docker build files
- [tgfam-finder/scripts_centos](https://github.com/tgfam-finder/scripts_centos) - TGFam-Finder scripts (CentOS)
- [chensole/bioinformatics](https://github.com/chensole/bioinformatics) - 
- [sunjiamin0824/CRISPR-Local](https://github.com/sunjiamin0824/CRISPR-Local) - A local tool for high-throughput CRISPR single-guide RNA (sgRNA) design in plants.
- [jokergoo/ngspipeline](https://github.com/jokergoo/ngspipeline) - NGS pipeline
- [zhutao1009/Genome-annotation-pipeline](https://github.com/zhutao1009/Genome-annotation-pipeline) - Genome annotation
- [zy-optimistic/GAEP](https://github.com/zy-optimistic/GAEP) - A genome assembly assessment pipeline.
- [molevol-ub/bitacora](https://github.com/molevol-ub/bitacora) - BITACORA: A Bioinformatics tool for gene family annotation
- [jgrimplet/Grapevine-version-transfer-annotations](https://github.com/jgrimplet/Grapevine-version-transfer-annotations) - transfert annotation position between grapevine genome versions
- [jameslz/ncbi-kit](https://github.com/jameslz/ncbi-kit) - Download NCBI assembly genomes with ftp. rsync, ascp
- [Zhuxitong/ppsPCP](https://github.com/Zhuxitong/ppsPCP) - A Plant Presence/absence Variants Scanner and Pan-genome Construction Pipeline
- [Ensembl/ensembl-pipeline](https://github.com/Ensembl/ensembl-pipeline) - *DEPRECATED* Job management for the Ensembl Genebuild pipeline
- [jiaxianqing/Pipelines](https://github.com/jiaxianqing/Pipelines) - Pipelines used in bioinformatic analysis
- [PASApipeline/PASApipeline](https://github.com/PASApipeline/PASApipeline) - PASA software
- [TatianneNegri/RNAplonc](https://github.com/TatianneNegri/RNAplonc) - A tool for identification of plant long non-coding RNAs.
- [382998862/research_pipline](https://github.com/382998862/research_pipline) - 
- [meiyang12/Genome-annotation-pipeline](https://github.com/meiyang12/Genome-annotation-pipeline) - Genome annotation pipeline
- [ISUgenomics/common_scripts](https://github.com/ISUgenomics/common_scripts) - my bin directory
- [stat-lab/EvalSVcallers](https://github.com/stat-lab/EvalSVcallers) - Evaluate the performances (precision and recall) of structural variation (SV) callers
- [hanlinqian/LncRNA_Finder2.0](https://github.com/hanlinqian/LncRNA_Finder2.0) - An upgraded version of lncRNA_Finder for identifying lncRNAs
- [nstenz/plot-ks](https://github.com/nstenz/plot-ks) - 
- [zavolanlab/Dockerfiles](https://github.com/zavolanlab/Dockerfiles) - Dockerfile repository of the Zavolan Lab
- [liangjiaoxue/ASEtrans](https://github.com/liangjiaoxue/ASEtrans) - Codes for allele specific expression
- [bradfordcondon/SNP-Sliding-Windows](https://github.com/bradfordcondon/SNP-Sliding-Windows) - 
- [eead-csic-compbio/primers4clades](https://github.com/eead-csic-compbio/primers4clades) - Automated pipeline for the design of PCR primers for cross-species amplification of novel sequences
- [eead-csic-compbio/EEADannot](https://github.com/eead-csic-compbio/EEADannot) - Manual curation of selected DNA motifs and cis regulatory sites, mostly from plants
- [eead-csic-compbio/get_homologues](https://github.com/eead-csic-compbio/get_homologues) - GET_HOMOLOGUES: a versatile software package for pan-genome analysis
- [jameslz/flye-kit](https://github.com/jameslz/flye-kit) - long reads assembly with Flye and Pilon
- [ckcollings1984/NGSplotPipeline](https://github.com/ckcollings1984/NGSplotPipeline) - NGSplotPipeline is a perl wrapper script that enhances the use of the popular bioinformatics software, NGSplot.
- [yinjun111/sbptools](https://github.com/yinjun111/sbptools) - 
- [yjx1217/RecombineX](https://github.com/yjx1217/RecombineX) - A computational framework for tetrad-based genotyping & recombination landscape analysis
- [CAG-CNV/ParseCNV2](https://github.com/CAG-CNV/ParseCNV2) - Parse Copy Number Variation from Array and Sequencing
- [biomarble/onekeyBinMap](https://github.com/biomarble/onekeyBinMap) - 
- [lilei1/Utilites](https://github.com/lilei1/Utilites) - 
- [starskyzheng/JGI_downloader](https://github.com/starskyzheng/JGI_downloader) - Download files from JGI
- [starskyzheng/panpop](https://github.com/starskyzheng/panpop) - Application of pan-genome for population
- [zm-git-dev/ASCPsra](https://github.com/zm-git-dev/ASCPsra) - å°†é€Ÿé“‚ä¸‹è½½å·¥å…·è¿›è¡Œå°è£…ï¼Œä»¥ä¾¿é«˜æ•ˆæ–¹ä¾¿åœ°æ‰¹é‡ä¸‹è½½SRAæµ‹åºæ•°æ®ã€‚
- [ntm/grexome-TIMC-Secondary](https://github.com/ntm/grexome-TIMC-Secondary) - exome pipeline from TIMC - secondary analyses (GVCF to analysis-ready TSVs)
- [NBISweden/AGAT](https://github.com/NBISweden/AGAT) - Another Gtf/Gff Analysis Toolkit
- [harrisonlab/nod](https://github.com/harrisonlab/nod) - Plant genome assembly
- [xhhuanglab/QTN_scripts](https://github.com/xhhuanglab/QTN_scripts) - 
- [zj-wien/Paleognaths](https://github.com/zj-wien/Paleognaths) - Here we demonstrates the analyses of whole-genome non-coding tree, and identification of sex-linked sequences
- [tangerzhang/synDrawer](https://github.com/tangerzhang/synDrawer) - add scripts
- [wonaya/ParaAT](https://github.com/wonaya/ParaAT) - 
- [zhuochenbioinfo/PopGen](https://github.com/zhuochenbioinfo/PopGen) - Small scripts for population genetics analysis
- [jkimlab/RBRC](https://github.com/jkimlab/RBRC) - 
- [jkimlab/IMAP](https://github.com/jkimlab/IMAP) - 
- [Axolotl233/Simple_Script](https://github.com/Axolotl233/Simple_Script) - pipeline and tools for personal use
- [chenqumi/scripts](https://github.com/chenqumi/scripts) - common used scripts
- [johnlees/bioinformatics](https://github.com/johnlees/bioinformatics) - Utilities written for bioinformatics
- [yjx1217/simuG](https://github.com/yjx1217/simuG) - simuG: a general-purpose genome simulator
- [ShaoLB/Population-Analysis](https://github.com/ShaoLB/Population-Analysis) - Population Analysis pipeline
- [oushujun/LTR_FINDER_parallel](https://github.com/oushujun/LTR_FINDER_parallel) - A parallel wrapper for LTR_FINDER
- [Shicheng-Guo/methtype](https://github.com/Shicheng-Guo/methtype) - methylation haplotype based analysis
- [Shicheng-Guo/Bismark](https://github.com/Shicheng-Guo/Bismark) - A tool to map bisulfite converted sequence reads and determine cytosine methylation states
- [mwvaughn/fastlmm-plot](https://github.com/mwvaughn/fastlmm-plot) - Create dynamic plots from FastLMM output
- [tseemann/snippy](https://github.com/tseemann/snippy) - :scissors: :zap: Rapid haploid variant calling and core genome alignment
- [zm-git-dev/multiDEGRunnerResultAnalyzer](https://github.com/zm-git-dev/multiDEGRunnerResultAnalyzer) - This is a perl script to run GSEA, HyperFUNC, GOSlimmer and RamiGO, on the differential expression results specified in geneDEResultPathFile.
- [zm-git-dev/myrna](https://github.com/zm-git-dev/myrna) - Cloud-scale differential expression for RNA-seq
- [zm-git-dev/CAPS-finder](https://github.com/zm-git-dev/CAPS-finder) - Finds potential sites for CAPS markers.
- [zm-git-dev/RNAseq-5](https://github.com/zm-git-dev/RNAseq-5) - Pipeline for RNAseq using Tophat and Cufflinks protocols
- [zm-git-dev/ngspipeline](https://github.com/zm-git-dev/ngspipeline) - NGS pipeline
- [zm-git-dev/IGS-Transcriptomics](https://github.com/zm-git-dev/IGS-Transcriptomics) - Transcriptomics Analysis pipeline developed at the Institute for Genome Sciences
- [zm-git-dev/perlscripts](https://github.com/zm-git-dev/perlscripts) - 
- [zm-git-dev/Panel_docker](https://github.com/zm-git-dev/Panel_docker) - creat a panel analysis pipeline via docker
- [zm-git-dev/trinityrnaseq](https://github.com/zm-git-dev/trinityrnaseq) - Trinity RNA-Seq de novo transcriptome assembly
- [zm-git-dev/MutMap](https://github.com/zm-git-dev/MutMap) - Scripts to faciliate the MutMap protocol.
- [zm-git-dev/DMAP](https://github.com/zm-git-dev/DMAP) - Davids Map Alignment Program for comparing genetic and physical maps
- [zm-git-dev/Find_mutual_BestHit](https://github.com/zm-git-dev/Find_mutual_BestHit) - Identify reciprocal best BLASTp hits
- [zm-git-dev/tissueSpecificExpression](https://github.com/zm-git-dev/tissueSpecificExpression) - Calculate tissue specific gene expression in GTEx tissue samples.
- [zm-git-dev/pm_pepline](https://github.com/zm-git-dev/pm_pepline) - 
- [zm-git-dev/Reseq-scripts](https://github.com/zm-git-dev/Reseq-scripts) - 
- [zm-git-dev/rbh](https://github.com/zm-git-dev/rbh) - Finds reciprocal-best-hit matches between two al-by-all BLAST searches
- [zm-git-dev/PIC_calculation](https://github.com/zm-git-dev/PIC_calculation) - Calculation of SNP No. , InDel No. and PIC (potentially informative characters) values between two sequences after alignment.
- [zm-git-dev/ASEtrans](https://github.com/zm-git-dev/ASEtrans) - Codes for allele specific expression
- [zm-git-dev/test_perl](https://github.com/zm-git-dev/test_perl) - exercise, learning perl
- [zm-git-dev/mybin](https://github.com/zm-git-dev/mybin) - my scripts
- [zm-git-dev/bio_scripts](https://github.com/zm-git-dev/bio_scripts) - Practical, reusable scripts for bioinformatics
- [zm-git-dev/EUPAN](https://github.com/zm-git-dev/EUPAN) - EUcaryotic PAN-genome analysis toolkit
- [zm-git-dev/scripts](https://github.com/zm-git-dev/scripts) - common used scripts
- [zm-git-dev/utilities](https://github.com/zm-git-dev/utilities) - A collection of mostly Perl scripts for Bioinformatics
- [zm-git-dev/bioinformatics](https://github.com/zm-git-dev/bioinformatics) - tools for biolinux
- [zm-git-dev/dolphin-tools](https://github.com/zm-git-dev/dolphin-tools) - 
- [zm-git-dev/my_scripts](https://github.com/zm-git-dev/my_scripts) - 
- [zm-git-dev/scripts-3](https://github.com/zm-git-dev/scripts-3) - 
- [zm-git-dev/reseq_cloud_pipeline](https://github.com/zm-git-dev/reseq_cloud_pipeline) - resequencing Cloud and Cluster pipeline
- [zm-git-dev/FAFU-cgb](https://github.com/zm-git-dev/FAFU-cgb) - 
- [zm-git-dev/apa_bin](https://github.com/zm-git-dev/apa_bin) - Contents of the /home/apa/local/bin/ tree
- [zm-git-dev/2ndGS_Pipelines](https://github.com/zm-git-dev/2ndGS_Pipelines) - Perl 5 and R source codes for next generation (the second generation) sequencing data analysis by integrating lots of open-source softwares.
- [zm-git-dev/RNASeq-6](https://github.com/zm-git-dev/RNASeq-6) - RNASeq pipeline
- [zm-git-dev/pipeline-miRNA](https://github.com/zm-git-dev/pipeline-miRNA) - miRNA pipeline
- [zm-git-dev/RNA-seq-analysis-1](https://github.com/zm-git-dev/RNA-seq-analysis-1) - 
- [zm-git-dev/vcf2maf](https://github.com/zm-git-dev/vcf2maf) - Convert a VCF into a MAF, where each variant is annotated to only one of all possible gene isoforms
- [zm-git-dev/common_scripts](https://github.com/zm-git-dev/common_scripts) - my bin directory
- [zm-git-dev/bin](https://github.com/zm-git-dev/bin) - scripts
- [zm-git-dev/Orthologous-identification](https://github.com/zm-git-dev/Orthologous-identification) - This script is used for orthologous genes by reciprocally best-hit approach
- [zm-git-dev/ChIP_seq](https://github.com/zm-git-dev/ChIP_seq) - 
- [zm-git-dev/ref_RNAseq](https://github.com/zm-git-dev/ref_RNAseq) - 
- [zm-git-dev/protein-GO-term-annotation](https://github.com/zm-git-dev/protein-GO-term-annotation) - 
- [zm-git-dev/ngomicswf](https://github.com/zm-git-dev/ngomicswf) - A workflow tool to automatically run a bioinformatic pipeline
- [zm-git-dev/bioinformatics_codes](https://github.com/zm-git-dev/bioinformatics_codes) - My PERL and other UNIX based codes developed while analysing NGS, microarray other data.
- [zm-git-dev/Anno-pipeline](https://github.com/zm-git-dev/Anno-pipeline) - RNA annotation pipeline
- [zm-git-dev/NGS_data_processing](https://github.com/zm-git-dev/NGS_data_processing) - Tool set for processing fasta/fastq/table formated data. Usually they are perl scripts.
- [zm-git-dev/orthomcl-pipeline](https://github.com/zm-git-dev/orthomcl-pipeline) - Automates running of OrthoMCL software from http://orthomcl.org/common/downloads/software/v2.0/
- [zm-git-dev/research_pipline](https://github.com/zm-git-dev/research_pipline) - 
- [zm-git-dev/any2fasta](https://github.com/zm-git-dev/any2fasta) - Convert various sequence formats to FASTA
- [zm-git-dev/LTR_FINDER_parallel](https://github.com/zm-git-dev/LTR_FINDER_parallel) - A parallel wrapper for LTR_FINDER
- [zm-git-dev/CHURP](https://github.com/zm-git-dev/CHURP) - 
- [zm-git-dev/GWAS_script](https://github.com/zm-git-dev/GWAS_script) - Working on cotton resequencing
- [zm-git-dev/Grass-KaKs](https://github.com/zm-git-dev/Grass-KaKs) - KaKs calculation per gene in multiple grass species
- [zm-git-dev/Mo17_genome_assembly](https://github.com/zm-git-dev/Mo17_genome_assembly) - 
- [zm-git-dev/BioScripts](https://github.com/zm-git-dev/BioScripts) - Scripts for bioinformatics data processing and analysis
- [zm-git-dev/HERA](https://github.com/zm-git-dev/HERA) - 
- [zm-git-dev/Scripts_for_GB](https://github.com/zm-git-dev/Scripts_for_GB) - The code used in the manuscript titled "Gene duplication and evolution in recurring polyploidization-diploidization cycles in plants".
- [zm-git-dev/ASE-pipeline](https://github.com/zm-git-dev/ASE-pipeline) - A pipeline for Detecting Allele Specific Expression
- [zm-git-dev/Pertussis_n257](https://github.com/zm-git-dev/Pertussis_n257) - Scripts used for genome rearrangement analyses in Weigand et al. 2017
- [zm-git-dev/Roary](https://github.com/zm-git-dev/Roary) - Rapid large-scale prokaryote pan genome analysis
- [zm-git-dev/OutcrossSeq](https://github.com/zm-git-dev/OutcrossSeq) - 
- [zm-git-dev/Varathon](https://github.com/zm-git-dev/Varathon) - A scalable variant calling and benchmarking framework supporting both short and long reads.
- [zm-git-dev/radmap](https://github.com/zm-git-dev/radmap) - Analyse RAD-tags and generate the Genetic Map with GWAS
- [zm-git-dev/MotifFinder](https://github.com/zm-git-dev/MotifFinder) - word-based algorithm for regulatory motif identification
- [zm-git-dev/dockers](https://github.com/zm-git-dev/dockers) - Docker files
- [zm-git-dev/ppsPCP](https://github.com/zm-git-dev/ppsPCP) - A Plant Presence/absence Variants Scanner and Pan-genome Construction Pipeline
- [zm-git-dev/wgs](https://github.com/zm-git-dev/wgs) - A mutation detection and annotation pipeline used for WGS or WES, including SNP/InDel/CNV/Fusion/SV detection.
- [zm-git-dev/lettuce2020](https://github.com/zm-git-dev/lettuce2020) - Codes for the lettuce resequencing project in 2020
- [zm-git-dev/ViewBS](https://github.com/zm-git-dev/ViewBS) - ViewBS - a powerful toolkit for visualization of high-throughput bisulfite sequencing data
- [zm-git-dev/Work_flow_of_population_genetics](https://github.com/zm-git-dev/Work_flow_of_population_genetics) - æ•´ç†å¸¸ç”¨çš„ç¾¤ä½“é—ä¼ å­¦åˆ†ææµç¨‹å’Œè„šæœ¬
- [zm-git-dev/Pipelines](https://github.com/zm-git-dev/Pipelines) - Pipelines used in bioinformatic analysis
- [zm-git-dev/GenomeSyn](https://github.com/zm-git-dev/GenomeSyn) - We developed GenomeSyn as a new tool for constructing and visualizing genome synteny, its novel design and implementation can serve as a necessary complement to the study of genomic synteny and struct
- [zm-git-dev/prokka](https://github.com/zm-git-dev/prokka) - :zap: :aquarius: Rapid prokaryotic genome annotation
- [zm-git-dev/ShaoLB](https://github.com/zm-git-dev/ShaoLB) - Population Analysis
- [zm-git-dev/JGI_downloader](https://github.com/zm-git-dev/JGI_downloader) - Download files from JGI
- [zm-git-dev/pandoc-report](https://github.com/zm-git-dev/pandoc-report) - 
- [zm-git-dev/spark](https://github.com/zm-git-dev/spark) - SPARK and genetics analysis of ASD
- [zm-git-dev/pRNASeqTools](https://github.com/zm-git-dev/pRNASeqTools) - plant NGS analysis
- [zm-git-dev/bio-nextflow](https://github.com/zm-git-dev/bio-nextflow) - nextflow pipeline
- [zm-git-dev/RicePangenomeEvolution](https://github.com/zm-git-dev/RicePangenomeEvolution) - scripts used in Rice Pangenome Projects (Evolution).
- [zm-git-dev/panpop](https://github.com/zm-git-dev/panpop) - Application of graph-pan-genome to population based on resequnces
- [jasongit0311/maize_for_Li](https://github.com/jasongit0311/maize_for_Li) - This is the script repository for Mr. Li's article
- [nousiaso/batch_4DTV_calculation](https://github.com/nousiaso/batch_4DTV_calculation) - Calculate
- [abxka/tissueSpecificExpression](https://github.com/abxka/tissueSpecificExpression) - Calculate tissue specific gene expression in GTEx tissue samples.
- [ntm/grexome-TIMC-Primary](https://github.com/ntm/grexome-TIMC-Primary) - exome pipeline from TIMC - primary analyses (FASTQs to GVCF)
- [KCGC/Scripts](https://github.com/KCGC/Scripts) - A location for all our general scripts
- [CAFS-bioinformatics/LR_Gapcloser](https://github.com/CAFS-bioinformatics/LR_Gapcloser) - use long sequenced reads to close gaps in assemblies
- [ctjsw/bin](https://github.com/ctjsw/bin) - scripts
- [starskyzheng/BioScripts](https://github.com/starskyzheng/BioScripts) - Scripts for bioinformatics data processing and analysis
- [Yamei-Ding/Juglandaceae](https://github.com/Yamei-Ding/Juglandaceae) - 
- [NBISweden/GAAS](https://github.com/NBISweden/GAAS) - Genome Assembly and Annotation Service code
- [huangl07/pandoc-report](https://github.com/huangl07/pandoc-report) - 
- [caulai/Mo17_genome_assembly](https://github.com/caulai/Mo17_genome_assembly) - 
- [dontkme/EAHelitron](https://github.com/dontkme/EAHelitron) - Easy to Annotate Helitrons Unix-like command line.
- [zhongmicai/plantTF_identifier](https://github.com/zhongmicai/plantTF_identifier) - 
- [quanrd/CandiHap](https://github.com/quanrd/CandiHap) - Genome-wide association study (GWAS) is widely utilized to identify genes involved in plants, animals and human complex traits. The identified SNP is not necessarily the causal variant, but it is rath
- [zhan2740/rust_scores_conversion](https://github.com/zhan2740/rust_scores_conversion) - Scripts to convert seedling and adult plant stripe, stem and leaf rust scores into numeric scales suitable for genome wide association studies (GWAS)
- [Zhanmengtao/GenoToolBox](https://github.com/Zhanmengtao/GenoToolBox) - A box of genomics tools
- [Zhanmengtao/bin](https://github.com/Zhanmengtao/bin) - scripts
- [xinwang-bio/bioinformatics](https://github.com/xinwang-bio/bioinformatics) - script
- [hyphaltip/mTEA](https://github.com/hyphaltip/mTEA) - Mosquito TE Annotation
- [CJ-Chen/TBtools-Manual](https://github.com/CJ-Chen/TBtools-Manual) - 
- [pseudogene/radmap](https://github.com/pseudogene/radmap) - Analyse RAD-tags and generate the Genetic Map with GWAS
- [pdimens/LepWrap](https://github.com/pdimens/LepWrap) - A pipeline to use Lep-Map3 to create linkage maps and LepAnchor for anchoring and orienting genome assemblies with said linkage maps.
- [Wadauk/scihub_ck](https://github.com/Wadauk/scihub_ck) - A tiny tool for checking the working domain of sci-hub
- [yjx1217/Varathon](https://github.com/yjx1217/Varathon) - A scalable variant calling and benchmarking framework supporting both short and long reads.
- [legumeinfo/scripts-qtlloader](https://github.com/legumeinfo/scripts-qtlloader) - Scripts for loading QTL data
- [xiekunwhy/bsa](https://github.com/xiekunwhy/bsa) - Bulked-Segregant Analysis using vcf file with or without parents
- [YanWangTF/Find_mutual_BestHit](https://github.com/YanWangTF/Find_mutual_BestHit) - Identify reciprocal best BLASTp hits
- [qiao-xin/Scripts_for_GB](https://github.com/qiao-xin/Scripts_for_GB) - The code used in the manuscript titled "Gene duplication and evolution in recurring polyploidization-diploidization cycles in plants".
- [jlew826/Bio7200](https://github.com/jlew826/Bio7200) - Programming for Bioinformatics
- [bakerwm/RNASeq-dataanalysis](https://github.com/bakerwm/RNASeq-dataanalysis) - Scripts for RNA Seq analysis.
- [tannishtha/BIOL-8803](https://github.com/tannishtha/BIOL-8803) - Scripts written for Programming for Bioinformatics class
- [santiagosnchez/rbh](https://github.com/santiagosnchez/rbh) - Finds reciprocal-best-hit matches between two all-by-all BLAST searches
- [lnavarroe/BioScripts](https://github.com/lnavarroe/BioScripts) - Some useful Perl scripts for bioinformatic applications
- [jono-powell/wheat_homoeolog_specificity_scripts](https://github.com/jono-powell/wheat_homoeolog_specificity_scripts) - 
- [binlu1981/bio_scripts](https://github.com/binlu1981/bio_scripts) - Practical, reusable scripts for bioinformatics
- [harrisonlab/pseudomonas](https://github.com/harrisonlab/pseudomonas) - 
- [ab-src/BedDepthCsvGenerator](https://github.com/ab-src/BedDepthCsvGenerator) - Wrapper script that generates a CSV file of read depths based on BAM files and a BED file
- [zsanli/bsa4yeast](https://github.com/zsanli/bsa4yeast) - 
- [Sunhh/NGS_data_processing](https://github.com/Sunhh/NGS_data_processing) - Tool set for processing fasta/fastq/table formated data. Usually they are perl scripts.
- [czheluo/Anno-pipeline](https://github.com/czheluo/Anno-pipeline) - RNA annotation pipeline
- [naturalis/brassica-snps](https://github.com/naturalis/brassica-snps) - Ontology-mediated refinement of bulk segregant analyses
- [shangshanzhizhe/Work_flow_of_population_genetics](https://github.com/shangshanzhizhe/Work_flow_of_population_genetics) - æ•´ç†å¸¸ç”¨çš„ç¾¤ä½“é—ä¼ å­¦åˆ†ææµç¨‹å’Œè„šæœ¬
- [dontkme/BSAscmd](https://github.com/dontkme/BSAscmd) - NGS BSA simple commands.
- [billzt/PrimerServer](https://github.com/billzt/PrimerServer) - Obsolete!! Begin to turn to PrimerServer2
- [jianzuoyi/augustus](https://github.com/jianzuoyi/augustus) - 
- [Pussinboots123/Orthologous-identification](https://github.com/Pussinboots123/Orthologous-identification) - This script is used for orthologous genes by reciprocally best-hit approach
- [bakerwm/test_perl](https://github.com/bakerwm/test_perl) - exercise, learning perl
- [guyleonard/get_jgi_genomes](https://github.com/guyleonard/get_jgi_genomes) - A quick and easy way to download the genomes/predicted proteins of taxa available in JGI's Genome Portal.
- [dePamphilis/PlantTribes](https://github.com/dePamphilis/PlantTribes) - PlantTribes is a collection of automated gene family analysis pipelines for comparative plant genomics
- [daweih/plantortho](https://github.com/daweih/plantortho) - Ortholog inference in plants.
- [YdlNAU/pipeline-for-cufflinks](https://github.com/YdlNAU/pipeline-for-cufflinks) - This is a pileline that you use form BAM to get FPKM for lots of samples!
- [WenchaoLin/Sam2Xpkm](https://github.com/WenchaoLin/Sam2Xpkm) - Calculate RPKM and FPKM from SAM file
- [shengqh/TIGER](https://github.com/shengqh/TIGER) - Tools for Intensive Genome alignment of Extracelllar smallRNA (TIGER) analysis pipeline
- [decodebiology/rpkm_rnaseq_count](https://github.com/decodebiology/rpkm_rnaseq_count) - RPKM for RNASeq counts
- [matzLab/TranscriptomeAnnotation](https://github.com/matzLab/TranscriptomeAnnotation) - Pipeline and scripts for annotating transcriptomes
- [esunlu/go_cluster_analysis](https://github.com/esunlu/go_cluster_analysis) - This tools were used for analysis of go annotation files created for different species
- [apetkau/orthomcl-pipeline](https://github.com/apetkau/orthomcl-pipeline) - Automates running of OrthoMCL software from http://orthomcl.org/common/downloads/software/v2.0/
- [fanyucai1/denovo_RNAseq](https://github.com/fanyucai1/denovo_RNAseq) - 
- [caballero/SeqComplex](https://github.com/caballero/SeqComplex) - Algorithms to compute DNA complexity
- [jhh130910/protein-GO-term-annotation](https://github.com/jhh130910/protein-GO-term-annotation) - 
- [ewmorr/GO_annotation_tools](https://github.com/ewmorr/GO_annotation_tools) - Tools for genome annotation against GO

## PostScript 

- [zm-git-dev/snp_plotter](https://github.com/zm-git-dev/snp_plotter) - snp.plotter is an R package that creates publishable-quality plots of p-values using single SNP and/or haplotype data.
- [avinashkarn/snp_plotter](https://github.com/avinashkarn/snp_plotter) - snp.plotter is an R package that creates publishable-quality plots of p-values using single SNP and/or haplotype data.
- [834810071/NEUBachelorThesis2019](https://github.com/834810071/NEUBachelorThesis2019) - ä¸œåŒ—å¤§å­¦2019å±Š[2015çº§]æœ¬ç§‘æ¯•è®¾Latexæ¨¡ç‰ˆ
- [saketkc/moca](https://github.com/saketkc/moca) - :m: Tool for motif conservation analysis

## Python 

- [mgymrek/estrs](https://github.com/mgymrek/estrs) - All code and data for the study "Abundant contribution of short tandem repeats to gene expression variation in humans"
- [fei0810/ChatPaper](https://github.com/fei0810/ChatPaper) - Use ChatGPT to summarize the arXiv papers. å…¨æµç¨‹åŠ é€Ÿç§‘ç ”ï¼Œåˆ©ç”¨chatgptè¿›è¡Œè®ºæ–‡æ€»ç»“+æ¶¦è‰²+å®¡ç¨¿+å®¡ç¨¿å›å¤
- [bpucker/MybMonday](https://github.com/bpucker/MybMonday) - MYB transcription factors are one of the largest gene family in plants and control many processes. This repository provides additional background to the #MYB_Monday tweets
- [xuzhougeng/HCR_designer](https://github.com/xuzhougeng/HCR_designer) - HCR prober designer
- [bliebeskind/AncestralGeneContent](https://github.com/bliebeskind/AncestralGeneContent) - Class and scripts for recontructing ancestral genome content using EvolMap and Notung
- [dabeaz-course/python-mastery](https://github.com/dabeaz-course/python-mastery) - Advanced Python Mastery (course by @dabeaz)
- [zhutao1009/dnds](https://github.com/zhutao1009/dnds) - Calculate the dn/ds (also pn and ps) value of cds
- [weberlab-hhu/Helixer](https://github.com/weberlab-hhu/Helixer) - Using Deep Learning to predict gene annotations
- [Hua-CM/HuaSmallTools](https://github.com/Hua-CM/HuaSmallTools) - Some bioinformic domestic scripts.
- [zhouyflab/AnnotationPipeline-EVM_based-DClab](https://github.com/zhouyflab/AnnotationPipeline-EVM_based-DClab) - CantÃ¹ Lab @ UC Davis - Annotation pipeline - EVM based
- [patflick/tsppi](https://github.com/patflick/tsppi) - Analysis of tissue-specific protein-protein interaction networks
- [reorx/awesome-chatgpt-api](https://github.com/reorx/awesome-chatgpt-api) - Curated list of apps and tools that not only use the new ChatGPT API, but also allow users to configure their own API keys, enabling free and on-demand usage of their own quota.
- [LiLittleCat/awesome-free-chatgpt](https://github.com/LiLittleCat/awesome-free-chatgpt) - ğŸ†“å…è´¹çš„ ChatGPT é•œåƒç½‘ç«™åˆ—è¡¨ï¼ŒæŒç»­æ›´æ–°ã€‚List of free ChatGPT mirror sites, continuously updated.
- [DongnaMa/python-for-bio](https://github.com/DongnaMa/python-for-bio) - I love python
- [Shixiaoya/T2T_Assembly](https://github.com/Shixiaoya/T2T_Assembly) - Methods for assembling high-quality genomes
- [GaiZhenbiao/ChuanhuChatGPT](https://github.com/GaiZhenbiao/ChuanhuChatGPT) - GUI for ChatGPT API and many LLMs. Supports agents, file-based QA, GPT finetuning and query with web search. All with a neat UI.
- [binary-husky/gpt_academic](https://github.com/binary-husky/gpt_academic) - ä¸ºChatGPT/GLMæä¾›å®ç”¨åŒ–äº¤äº’ç•Œé¢ï¼Œç‰¹åˆ«ä¼˜åŒ–è®ºæ–‡é˜…è¯»/æ¶¦è‰²/å†™ä½œä½“éªŒï¼Œæ¨¡å—åŒ–è®¾è®¡ï¼Œæ”¯æŒè‡ªå®šä¹‰å¿«æ·æŒ‰é’®&å‡½æ•°æ’ä»¶ï¼Œæ”¯æŒPythonå’ŒC++ç­‰é¡¹ç›®å‰–æ&è‡ªè¯‘è§£åŠŸèƒ½ï¼ŒPDF/LaTexè®ºæ–‡ç¿»è¯‘&æ€»ç»“åŠŸèƒ½ï¼Œæ”¯æŒå¹¶è¡Œé—®è¯¢å¤šç§LLMæ¨¡å‹ï¼Œæ”¯æŒchatglm2ç­‰æœ¬åœ°æ¨¡å‹ã€‚å…¼å®¹æ–‡å¿ƒä¸€è¨€, moss, llama2, rwkv, claude2, é€šä¹‰åƒé—®, ä¹¦ç”Ÿ, è®¯é£æ˜Ÿç«ç­‰ã€‚
- [morriso1/tspecscores](https://github.com/morriso1/tspecscores) - 
- [vanheeringen-lab/genomepy](https://github.com/vanheeringen-lab/genomepy) - genes and genomes at your fingertips
- [santiagosnchez/vcf2fasta](https://github.com/santiagosnchez/vcf2fasta) - Converts a VCF file to a FASTA alignment provided a reference genome and a GFF file
- [zhayujie/chatgpt-on-wechat](https://github.com/zhayujie/chatgpt-on-wechat) - Wechat robot based on ChatGPT,  which using OpenAI api and itchat library. ä½¿ç”¨ChatGPTæ­å»ºå¾®ä¿¡èŠå¤©æœºå™¨äººï¼ŒåŸºäº GPT3.5/GPT4.0/Claude/æ–‡å¿ƒä¸€è¨€/è®¯é£æ˜Ÿç«/LinkAIï¼Œæ”¯æŒä¸ªäººå¾®ä¿¡ã€å…¬ä¼—å·ã€ä¼ä¸šå¾®ä¿¡éƒ¨ç½²ï¼Œèƒ½å¤„ç†æ–‡æœ¬ã€è¯­éŸ³å’Œå›¾ç‰‡ï¼Œè®¿é—®æ“ä½œç³»ç»Ÿå’Œäº’è”ç½‘ï¼Œæ”¯æŒåŸºäºçŸ¥è¯†åº“å®šåˆ¶ä¸“å±æœºå™¨äººã€‚
- [YaoXinZhi/PMID-Crawling](https://github.com/YaoXinZhi/PMID-Crawling) - æ ¹æ®titileä¿¡æ¯ çˆ¬å–PMID
- [YaoXinZhi/BioNLP-Toolkit](https://github.com/YaoXinZhi/BioNLP-Toolkit) - Commonly used BioNLP toolkit for myself.
- [DingWB/PyComplexHeatmap](https://github.com/DingWB/PyComplexHeatmap) - PyComplexHeatmap: A Python package to plot complex heatmap (clustermap)
- [kaixindelele/ChatPaper](https://github.com/kaixindelele/ChatPaper) - Use ChatGPT to summarize the arXiv papers. å…¨æµç¨‹åŠ é€Ÿç§‘ç ”ï¼Œåˆ©ç”¨chatgptè¿›è¡Œè®ºæ–‡å…¨æ–‡æ€»ç»“+ä¸“ä¸šç¿»è¯‘+æ¶¦è‰²+å®¡ç¨¿+å®¡ç¨¿å›å¤
- [DAWells/scrapeAF](https://github.com/DAWells/scrapeAF) - Aggregate HLA allele frequencies data from allelefrequencies.net at large multi population scale
- [glarue/climail](https://github.com/glarue/climail) - Send an email from the command-line
- [glarue/fasterq_dump](https://github.com/glarue/fasterq_dump) - A (faster) wrapper for NCBI's fastq-dump with some convenience functions
- [815382636/crawler_for_article](https://github.com/815382636/crawler_for_article) - åœ¨www.connectedpapers.comçˆ¬å–æ–‡çŒ®åŠæ‘˜è¦
- [Belval/pdf2image](https://github.com/Belval/pdf2image) - A python module that wraps the pdftoppm utility to convert PDF to PIL Image object
- [jrderuiter/snakemake-rnaseq](https://github.com/jrderuiter/snakemake-rnaseq) - RNA-seq workflow for Snakemake based on STAR and featureCounts.
- [ZhihaoXie/kegg_crawler](https://github.com/ZhihaoXie/kegg_crawler) - kegg crawler to get some informations of bioinformatics
- [youngDouble/GO_KEGG_annotation](https://github.com/youngDouble/GO_KEGG_annotation) - GO KEGG annotation
- [ZhihaoXie/bioToolkit](https://github.com/ZhihaoXie/bioToolkit) - Practical, reusable scripts for bioinformatics
- [ZhihaoXie/GO_Annotation_Plot](https://github.com/ZhihaoXie/GO_Annotation_Plot) - GO level 2 summary and barplot
- [ljdursi/poapy](https://github.com/ljdursi/poapy) - A simple Partial Order Aligner based on Lee, Grasso and Sharlow (2002), for education/demonstration purposes
- [timlu1024/v2rayN-linux](https://github.com/timlu1024/v2rayN-linux) - Fetch v2rayN subscription and test node availability on Linux
- [wjian8/psvcp_v1.01](https://github.com/wjian8/psvcp_v1.01) - Pan-genome Construction and Population Structure Variation Calling pipeline
- [pachterlab/gget](https://github.com/pachterlab/gget) - ğŸ§¬ gget enables efficient querying of genomic reference databases
- [hzaurzli/Calculate_DNA_percentage](https://github.com/hzaurzli/Calculate_DNA_percentage) - 
- [stonepy/icystone](https://github.com/stonepy/icystone) - 
- [yonglpy/data-analysis](https://github.com/yonglpy/data-analysis) - Record learning data analysis process
- [sc-zhang/popCNV](https://github.com/sc-zhang/popCNV) - popCNV is a tool for calculating copy number of genes and get genes selected by humans
- [aresti/primalscheme](https://github.com/aresti/primalscheme) - a tool for designing primer panels for multiplex PCR.
- [AnimalGenomicsETH/superpangenome_construction](https://github.com/AnimalGenomicsETH/superpangenome_construction) - 
- [PriyaLakr/Manhattan-plot](https://github.com/PriyaLakr/Manhattan-plot) - Script for designing a manhattan plot.
- [JonJala/mtag](https://github.com/JonJala/mtag) - Python command line tool for Multi-Trait Analysis of GWAS (MTAG)
- [haiwangyang/bigWigVis](https://github.com/haiwangyang/bigWigVis) - Visualize bigWig files of RNA-seq data (supported by pyGenomeTracks)
- [epigen/genome_tracks](https://github.com/epigen/genome_tracks) - A Snakemake workflow for easy visualization of genome browser tracks of aligned BAM files powered by the wrapper gtracks for the package pyGenomeTracks.
- [merenlab/anvio](https://github.com/merenlab/anvio) - An analysis and visualization platform for 'omics data
- [lzhtan/NSFC-Report](https://github.com/lzhtan/NSFC-Report) - Python3.6å†™çš„æçŸ­çˆ¬è™«ã€‚è¾“å…¥é¡¹ç›®ç¼–å·ï¼Œå¯ä¸‹è½½pngæ ¼å¼çš„ä¸­å›½å›½å®¶è‡ªç„¶ç§‘å­¦åŸºé‡‘å§”å‘å¸ƒçš„è‡ªç„¶ç§‘å­¦åŸºé‡‘ç»“é¡¹æŠ¥å‘Š
- [hanlinqian/IntegrativeNetworkMap](https://github.com/hanlinqian/IntegrativeNetworkMap) - 
- [suqingdong/nsfc](https://github.com/suqingdong/nsfc) - å›½å®¶è‡ªç„¶ç§‘å­¦åŸºé‡‘æŸ¥è¯¢
- [PacificBiosciences/FALCON](https://github.com/PacificBiosciences/FALCON) - FALCON: experimental PacBio diploid assembler -- Out-of-date -- Please use a binary release: https://github.com/PacificBiosciences/FALCON_unzip/wiki/Binaries
- [liulifenyf/TSPTFBS](https://github.com/liulifenyf/TSPTFBS) - trans-species prediction of TFBS in plants
- [Magic-K1d/MySpiders](https://github.com/Magic-K1d/MySpiders) - ä¸ªäººçˆ¬è™«æ•´ç†ï¼ˆä¸åŒæ—¶æœŸç¼–å†™çš„ä»£ç ï¼Œä»£ç é£æ ¼ä¼šæœ‰è¾ƒå¤§å·®å¼‚ï¼‰
- [KenForever1/nsfcCrawler](https://github.com/KenForever1/nsfcCrawler) - 
- [Dcclandbest/spider_nsfc](https://github.com/Dcclandbest/spider_nsfc) - 
- [MayQueen/nsfcReportExtr](https://github.com/MayQueen/nsfcReportExtr) - ä¸‹è½½NSFCé¡¹ç›®ç»“é¢˜æŠ¥å‘Š
- [walter-git/NSFC](https://github.com/walter-git/NSFC) - 
- [youlinglinlin/NSFC_conclusion_downloader](https://github.com/youlinglinlin/NSFC_conclusion_downloader) - A tool to Download PDF format conclusion from http://output.nsfc.gov.cn/
- [Rhilip/NSFC_Searcher](https://github.com/Rhilip/NSFC_Searcher) - A Searcher of Official NSFC ï¼ˆå›½è‡ªç„¶å®˜ç½‘æ£€ç´¢ï¼‰
- [ponnhide/pyCircos](https://github.com/ponnhide/pyCircos) - python Circos
- [uio-bmi/graph_peak_caller](https://github.com/uio-bmi/graph_peak_caller) - ChIP-seq peak caller for reads mapped to a graph-based reference genome
- [fudiyi/Figure](https://github.com/fudiyi/Figure) - ä½œå›¾è„šæœ¬
- [statgen/locuszoom-standalone](https://github.com/statgen/locuszoom-standalone) - Create regional association plots from GWAS or meta-analysis
- [GuyTeichman/RNAlysis](https://github.com/GuyTeichman/RNAlysis) - RNA sequencing analysis software
- [gbouras13/pharokka](https://github.com/gbouras13/pharokka) - fast phage annotation program
- [p0mel0/GetStarsCode](https://github.com/p0mel0/GetStarsCode) - æ‰¹é‡ä¸‹è½½githubç”¨æˆ·starsé¡¹ç›®çš„æºç 
- [Atvar2/RNAport](https://github.com/Atvar2/RNAport) - This pipeline provide the port for running RNA ref and RNA denovo pipeline
- [Atvar2/plotHicGenome](https://github.com/Atvar2/plotHicGenome) - The tools are used for displaying hic signal of genome
- [EichlerLab/pav](https://github.com/EichlerLab/pav) - Phased assembly variant caller
- [owkin/PyDESeq2](https://github.com/owkin/PyDESeq2) - A Python implementation of the DESeq2 pipeline for bulk RNA-seq DEA.
- [harvardinformatics/snpArcher](https://github.com/harvardinformatics/snpArcher) - Snakemake workflow for highly parallel variant calling designed for ease-of-use in non-model organisms.
- [BIMSBbioinfo/pigx_bsseq](https://github.com/BIMSBbioinfo/pigx_bsseq) - bisulfite sequencing pipeline from fastq to methylation reports
- [GenomicsNX/pigx_chipseq](https://github.com/GenomicsNX/pigx_chipseq) - Pipeline for Analysis of ChIP-Seq data
- [zavolanlab/zarp](https://github.com/zavolanlab/zarp) - Zavolan-Lab Automated RNA-Seq Pipeline
- [yiminghu/post-GWAS-pipeline](https://github.com/yiminghu/post-GWAS-pipeline) - A comprehensive pipeline for post-GWAS analysis leveraging diverse types of omics data
- [sanderslhc/Allele-Specific-Expression](https://github.com/sanderslhc/Allele-Specific-Expression) - 
- [YuSugihara/MutMap](https://github.com/YuSugihara/MutMap) - MutMap pipeline to identify causative mutations responsible for a phenotype
- [bmvdgeijn/WASP](https://github.com/bmvdgeijn/WASP) - WASP: allele-specific pipeline for unbiased read mapping and molecular QTL discovery
- [ryought/mummer-idotplot](https://github.com/ryought/mummer-idotplot) - Generate interactive dotplot from mummer4 output using plotly
- [reneshbedre/bioinfokit](https://github.com/reneshbedre/bioinfokit) - Bioinformatics data analysis and visualization toolkit
- [broadinstitute/gatk-sv](https://github.com/broadinstitute/gatk-sv) - A structural variation pipeline for short-read sequencing
- [rougier/scientific-visualization-book](https://github.com/rougier/scientific-visualization-book) - An open access book on scientific visualization using python and matplotlib
- [PhelaPoscam/scihubDownl](https://github.com/PhelaPoscam/scihubDownl) - 
- [mjlindberg/Misc-Tools](https://github.com/mjlindberg/Misc-Tools) - Place to store assorted legacy convenience scripts.
- [naivenaive/Sci-Hub-Downloader-by-PMID](https://github.com/naivenaive/Sci-Hub-Downloader-by-PMID) - Sci_Hub_Downloader
- [gymreklab/TRTools](https://github.com/gymreklab/TRTools) - Toolkit for genome-wide analysis of tandem repeats
- [lu-wang-github/python_for_sci-hub](https://github.com/lu-wang-github/python_for_sci-hub) - sci-hub desktop, åˆ©ç”¨doiå·è‡ªåŠ¨ä¸‹è½½æ–‡çŒ®ï¼ˆä»¥æ—¶é—´+é¢˜ç›®+ä½œè€…å‘½åï¼‰
- [Gusicun/paper_download](https://github.com/Gusicun/paper_download) - a crawler for downloading papers from sci-hub,check DOI from Baidu schoolar and Google schoolar
- [wolfsonliu/woodpecker](https://github.com/wolfsonliu/woodpecker) - batch download papers by doi from Sci-Hub.
- [shenwanxiang/AutoQueryPubMed](https://github.com/shenwanxiang/AutoQueryPubMed) - 
- [JanaSperschneider/GenomeAssemblyTools](https://github.com/JanaSperschneider/GenomeAssemblyTools) - A collection of scripts I use in genome assembly
- [znstrider/plottable](https://github.com/znstrider/plottable) - most pretty & lovely tables with matplotlib
- [IliaZenkov/async-pubmed-scraper](https://github.com/IliaZenkov/async-pubmed-scraper) - PubMed scraper for async search on a list of keywords and concurrent extraction of all found URLs, returning a DataFrame/CSV containing all article data (title, abstract, authors, affiliations, etc)
- [icervenka/article_highlights_scraper](https://github.com/icervenka/article_highlights_scraper) - Create summaries of recently published articles in Cell, Nature and Science journals
- [simonvh/pita](https://github.com/simonvh/pita) - Pipeline to improve transcript annotation based on RNA-seq and ChIP-seq data.
- [arvestad/alv](https://github.com/arvestad/alv) - A console-based alignment viewer
- [theislab/cellrank](https://github.com/theislab/cellrank) - CellRank: dynamics from multi-view single-cell data. See the docs:
- [parashardhapola/scarf](https://github.com/parashardhapola/scarf) - Toolkit for highly memory efficient analysis of single-cell RNA-Seq, scATAC-Seq and CITE-Seq data. Analyze atlas scale datasets with millions of cells on laptop.
- [vanheeringen-lab/gimmemotifs](https://github.com/vanheeringen-lab/gimmemotifs) - Suite of motif tools, including a motif prediction pipeline for ChIP-seq experiments.  See full GimmeMotifs documentation for detailed installation instructions and usage examples.
- [gem-pasteur/PanACoTA](https://github.com/gem-pasteur/PanACoTA) - PANgenome with Annotations, COre identification, Tree and corresponding Alignments
- [almlab/angst](https://github.com/almlab/angst) - Analyzer of Gene and Species Trees
- [rgerum/pylustrator](https://github.com/rgerum/pylustrator) - Visualisations of data are at the core of every publication of scientific research results. They have to be as clear as possible to facilitate the communication of research. As data can have different
- [nspies/genomeview](https://github.com/nspies/genomeview) - An extensible python-based genomics visualization engine
- [moshi4/pyGenomeViz](https://github.com/moshi4/pyGenomeViz) - A genome visualization python package for comparative genomics
- [moshi4/plot_genome_align_fig](https://github.com/moshi4/plot_genome_align_fig) - Plot genome alignment figure using progressiveMauve & genoPlotR
- [schneebergerlab/plotsr](https://github.com/schneebergerlab/plotsr) - Tool to plot synteny and structural rearrangements between genomes
- [gamcil/synthaser](https://github.com/gamcil/synthaser) - Parse NCBI CD-search results to find and visualise the domain architecture of secondary metabolite synthases
- [gamcil/clinker](https://github.com/gamcil/clinker) - Gene cluster comparison figure generator
- [ZhaoXM-Lab/metaMIC](https://github.com/ZhaoXM-Lab/metaMIC) - Assessing and Improving the quality of metagenomic assemblies
- [google/deepconsensus](https://github.com/google/deepconsensus) - DeepConsensus uses gap-aware sequence transformers to correct errors in Pacific Biosciences (PacBio) Circular Consensus Sequencing (CCS) data.
- [google/deepvariant](https://github.com/google/deepvariant) - DeepVariant is an analysis pipeline that uses a deep neural network to call genetic variants from next-generation DNA sequencing data.
- [comprna/SUPPA](https://github.com/comprna/SUPPA) - SUPPA: Fast quantification of splicing and differential splicing
- [wckdouglas/ngs_qc_plot](https://github.com/wckdouglas/ngs_qc_plot) - Scripts for QC in NGS analyses
- [nadiabaig/SNP_NGS_data_plotting](https://github.com/nadiabaig/SNP_NGS_data_plotting) - Python and R based scripts to plot your VCF, SNP and NGS data
- [harbourlab/SparK](https://github.com/harbourlab/SparK) - Publication quality NGS track plotting
- [pwwang/vcfstats](https://github.com/pwwang/vcfstats) - Powerful statistics for VCF files
- [DieTANG/PiGBS_Pipeline](https://github.com/DieTANG/PiGBS_Pipeline) - Parent-independent genotyping of diploid potato
- [Wangchangsh/SNPbinner](https://github.com/Wangchangsh/SNPbinner) - SNPbinner is a utility for the generation of genotype crossover points and binmaps based on SNP data across recombinant inbred lines.
- [IHEC/integrative_analysis_chip](https://github.com/IHEC/integrative_analysis_chip) - ChIP-Seq pipeline for integrative analysis
- [pwwang/immunopipe](https://github.com/pwwang/immunopipe) - Integrative analysis for scTCR- and scRNA-seq data
- [Chenglin20170390/vcfstats](https://github.com/Chenglin20170390/vcfstats) - Powerful statistics for VCF files
- [Chenglin20170390/plotsr](https://github.com/Chenglin20170390/plotsr) - Tool to plot synteny and structural rearrangements between genomes
- [stajichlab/AAFTF](https://github.com/stajichlab/AAFTF) - Automatic Assembly For The Fungi
- [ryanlayer/samplot](https://github.com/ryanlayer/samplot) - Plot structural variant signals from many BAMs and CRAMs
- [jradrion/TEFLoN](https://github.com/jradrion/TEFLoN) - TEFLoN uses paired-end illumina sequence data to discover and genotype transposable elements present in your samples.
- [fenderglass/Ragout](https://github.com/fenderglass/Ragout) - Chromosome-level scaffolding using multiple references
- [bcgsc/ntLink](https://github.com/bcgsc/ntLink) - Minimizer-based assembly scaffolding and mapping using long reads
- [junjunlab/GetTss](https://github.com/junjunlab/GetTss) - extract gene TSS site form gencode/ensembl/gencode database GTF file and export bed format file.
- [suqingdong/scihub](https://github.com/suqingdong/scihub) - PDF Downloader with SCI-HUB
- [brainbreaks/genome_downloader](https://github.com/brainbreaks/genome_downloader) - 
- [ShujiaHuang/geneview](https://github.com/ShujiaHuang/geneview) - Genomics data visualization in Python by using matplotlib.
- [biomarble/kasp_caps_primer_pipeline](https://github.com/biomarble/kasp_caps_primer_pipeline) - Design gene specific KASP and CAPS/dCAPS primers for any species
- [AbdelmajidLh/SNP_density_plot_from_VCF_python](https://github.com/AbdelmajidLh/SNP_density_plot_from_VCF_python) - # Author: Abdelmajid EL HOU, Data analyst - France # version 1.0 # date: 25-10-2021  # tested on ubuntu with python3.8  # This python program parse a Variant Call Format (VCF) file and then create a p
- [lorenziha/Python_tools](https://github.com/lorenziha/Python_tools) - Repository of python tools
- [neherlab/pan-genome-analysis](https://github.com/neherlab/pan-genome-analysis) - Processing pipeline for pan-genome visulization and exploration
- [wjwei-handsome/ZeaGeneMap](https://github.com/wjwei-handsome/ZeaGeneMap) - A python tool for identifying the mapping of genes between two genomes.
- [ucrbioinfo/PGV](https://github.com/ucrbioinfo/PGV) - Reference-agnostic Representation and Visualization of Pan-genomes
- [MikeTrizna/assembly_stats](https://github.com/MikeTrizna/assembly_stats) - Calculates both scaffold and contig statistics (N50, L50, etc.) from a scaffold FASTA file.
- [EichlerLab/svpop](https://github.com/EichlerLab/svpop) - Variant annotation and merging pipeline
- [mattravenhall/SV-Pop](https://github.com/mattravenhall/SV-Pop) - Upscaling SV detection to a multi-population level.
- [fuergaosi233/gitbook2pdf](https://github.com/fuergaosi233/gitbook2pdf) - Grab the contents of the gitbook document and convert it to pdf
- [phasegenomics/juicebox_scripts](https://github.com/phasegenomics/juicebox_scripts) - A collection of scripts for working with Hi-C data, Juicebox, and other genomic file formats
- [liuyujie0136/Sci-Tech-Notes](https://github.com/liuyujie0136/Sci-Tech-Notes) - 
- [zm-git-dev/bam-filter](https://github.com/zm-git-dev/bam-filter) - A simple tool to calculate metrics from a BAM file and filter references with uneven coverages
- [rrazaghi/pb-mod2bed](https://github.com/rrazaghi/pb-mod2bed) - Convert modified base BAM to BED
- [conchoecia/yabtap](https://github.com/conchoecia/yabtap) - Yet another bulk transcriptome assembly pipeline
- [conchoecia/genome_assembly_pipelines](https://github.com/conchoecia/genome_assembly_pipelines) - A collection of snakemake scripts to streamline genome annotation
- [AnimalGenomicsETH/bovine-graphs](https://github.com/AnimalGenomicsETH/bovine-graphs) - Integrate multiple genome assemblies into a pangenome graph
- [xuzhougeng/wgap](https://github.com/xuzhougeng/wgap) - Whole Genome Annotation Pipeline
- [xuzhougeng/auto_sra_rnaseq_pipeline](https://github.com/xuzhougeng/auto_sra_rnaseq_pipeline) - 
- [diekhans/T2T_cenSat_hub](https://github.com/diekhans/T2T_cenSat_hub) - T2T Centromere hub
- [GooglingTheCancerGenome/sv-callers](https://github.com/GooglingTheCancerGenome/sv-callers) - Snakemake-based workflow for detecting structural variants in genomic data
- [tolkit/plant-organellome-assembly](https://github.com/tolkit/plant-organellome-assembly) - Automation of plant organelle assemblies from raw reads.
- [baozg/assembly-annotation-pipeline](https://github.com/baozg/assembly-annotation-pipeline) - Plant genome assembly and annotation pipeline using snakemake
- [hyunkyungclairekim/scRNA-Seq-pipeline](https://github.com/hyunkyungclairekim/scRNA-Seq-pipeline) - 
- [mattfemia/scrnaseq-pipeline](https://github.com/mattfemia/scrnaseq-pipeline) - Nextflow-orchestrated distributed data pipeline for processing and analyzing raw scRNA-seq files
- [bodegalab/irescue](https://github.com/bodegalab/irescue) - A tool to quantify transposable elements expression in scRNA-seq
- [ZhikunWu/T2T-Assembly](https://github.com/ZhikunWu/T2T-Assembly) - 
- [BaiLab/SNPAAMapper-Python](https://github.com/BaiLab/SNPAAMapper-Python) - Python version of Github
- [CCCKW/mHapTk](https://github.com/CCCKW/mHapTk) - 
- [marbl/verkko](https://github.com/marbl/verkko) - 
- [lvrcek/GNNome-assembly](https://github.com/lvrcek/GNNome-assembly) - Learning to untangle genome assembly with graph neural networks.
- [FrankEP/scGMAI](https://github.com/FrankEP/scGMAI) - scGMAI: a Gaussian mixture model for clustering single-cell RNA-seq data based on deep autoencoder
- [JiaShun-Xiao/BLAST-bioinfor-tool](https://github.com/JiaShun-Xiao/BLAST-bioinfor-tool) - Python implementation of Basic Local Alignment Search Tool (BLAST) , which is the core algorithm in sequence alignment for genomes.
- [pgxcentre/region-plot](https://github.com/pgxcentre/region-plot) - A tool to plot significant regions of GWAS
- [Sh1ne111/manhattan_generator](https://github.com/Sh1ne111/manhattan_generator) - Manhattan plot Generator
- [sc-zhang/AlleleFinder](https://github.com/sc-zhang/AlleleFinder) - AlleleFinder is a tool for identifying allele genes from polyploid genome.
- [sc-zhang/T2Tools](https://github.com/sc-zhang/T2Tools) - T2Tools is a tool contains several tools for T2T assembly, and it is under development.
- [marbl/SALSA](https://github.com/marbl/SALSA) - SALSA: A tool to scaffold long read assemblies with Hi-C data
- [mikaell/chromograph](https://github.com/mikaell/chromograph) - Genetics graphing tool
- [Logancreator/Interesting](https://github.com/Logancreator/Interesting) - Interesting  code
- [sutormin94/E_coli_DRIP-Seq_analysis](https://github.com/sutormin94/E_coli_DRIP-Seq_analysis) - Scripts used for DRIP-Seq data analysis performed for E. coli
- [wwood/kingfisher-download](https://github.com/wwood/kingfisher-download) - Easier download/extract of FASTA/Q read data and metadata from the ENA, NCBI, AWS or GCP.
- [Jianhua-Wang/autosbatch](https://github.com/Jianhua-Wang/autosbatch) - submit hundreds of jobs to slurm automatically
- [kblin/ncbi-acc-download](https://github.com/kblin/ncbi-acc-download) - Download files from NCBI Entrez by accession
- [cgat-developers/cgat-apps](https://github.com/cgat-developers/cgat-apps) - cgat-apps repository
- [NCBI-Hackathons/SPeW](https://github.com/NCBI-Hackathons/SPeW) - Automatic Packaging and Distribution of Bioinformatics Pipelines
- [wonaya/genomic_tools](https://github.com/wonaya/genomic_tools) - various python scripts
- [adamewing/tldr](https://github.com/adamewing/tldr) - Identify and annotate TE-mediated insertions in long-read sequence data
- [genomewalker/derep-genomes](https://github.com/genomewalker/derep-genomes) - A simple genome de-replication tool using fastANI and a bit of graph-based analysis
- [loosolab/TOBIAS](https://github.com/loosolab/TOBIAS) - Transcription factor Occupancy prediction By Investigation of ATAC-seq Signal
- [broadinstitute/tensorqtl](https://github.com/broadinstitute/tensorqtl) - Ultrafast GPU-enabled QTL mapper
- [yuanlizhanshi/NGS_Workflow](https://github.com/yuanlizhanshi/NGS_Workflow) - Some NGS workflow created by snakemake
- [avoukadinova/clustering_homologs](https://github.com/avoukadinova/clustering_homologs) - 
- [daler/metaseq](https://github.com/daler/metaseq) - Framework for integrated analysis and plotting of ChIP/RIP/RNA/*-seq data
- [JackNg88/VARACS](https://github.com/JackNg88/VARACS) - Visualization Analysis of RNAseq,ATACseq,Chipseq and ScRNAseq
- [afrendeiro/toolkit](https://github.com/afrendeiro/toolkit) - A toolkit for NGS analysis with Python
- [deeptools/deepTools](https://github.com/deeptools/deepTools) - Tools to process and analyze deep sequencing data.
- [YaqiangCao/seqFlow](https://github.com/YaqiangCao/seqFlow) - My pipeline and code collections for NGS data analysis, such as RNA-seq, ChIP-seq, MNase-seq, Hi-C, Trac-looping and etc, mainly keep the analysis within Python.
- [maxplanck-ie/snakepipes](https://github.com/maxplanck-ie/snakepipes) - Customizable workflows based on snakemake and python for the analysis of NGS data
- [crazyhottommy/pyflow-ChIPseq](https://github.com/crazyhottommy/pyflow-ChIPseq) - a snakemake pipeline to process ChIP-seq files from GEO or in-house
- [timeu/PyGWAS](https://github.com/timeu/PyGWAS) - 
- [tommycarstensen/tc9](https://github.com/tommycarstensen/tc9) - 
- [paoyangchen-laboratory/methgo](https://github.com/paoyangchen-laboratory/methgo) - MethGo: a comprehensive tool for analyzing whole-genome bisulfite sequencing data
- [showteeth/blog_reprint](https://github.com/showteeth/blog_reprint) - åšå®¢è½¬è½½é¡¹ç›®(å°†åœ¨çº¿çš„åšå®¢è½¬åŒ–ä¸ºmarkdownæ ¼å¼æ–‡ä»¶)ï¼Œæ”¯æŒCSDNã€åšå®¢å›­ã€ç®€ä¹¦ä»¥åŠå°è±¡ç¬”è®°çš„å¯¼å‡ºæ–‡ä»¶
- [decouples/pubmed_download](https://github.com/decouples/pubmed_download) - download article by pmid list
- [nomadicburrito/PMIDtoAbstract](https://github.com/nomadicburrito/PMIDtoAbstract) - Program to grab abstracts and sort out DOIs from NCBI using PMIDs.
- [wangnan9394/apomixis_parallel_evolution](https://github.com/wangnan9394/apomixis_parallel_evolution) - A groups of data and scripts in the analysis
- [wangpanqiao/ilus](https://github.com/wangpanqiao/ilus) - A handy pipeline generator for whole genome re-sequencing (WGS) and whole exom sequencing data (WES) analysis.  ä¸€ä¸ªç®€æ˜“ä¸”å…¨é¢çš„ WGS/WES åˆ†ææµç¨‹ç”Ÿæˆå™¨.
- [gines-almagro/AnalysingProtein-DNAbindingSitesArabidopsisThalianaChip-seqExperiments](https://github.com/gines-almagro/AnalysingProtein-DNAbindingSitesArabidopsisThalianaChip-seqExperiments) - 
- [yangmqglobe/depipe](https://github.com/yangmqglobe/depipe) - è‡ªåŠ¨åŒ–çš„ ChIP-Seq åˆ†ææµç¨‹
- [ncbi/elastic-blast](https://github.com/ncbi/elastic-blast) - ElasticBLAST is a cloud-based tool to perform your BLAST searches faster and make you more effective
- [Sendrowski/BirchesScandinavia](https://github.com/Sendrowski/BirchesScandinavia) - Analysis pipeline associated with master's thesis on the population structure, demographic history and distribution of fitness effects of birches in Scandinavia.
- [pandas-dev/pandas](https://github.com/pandas-dev/pandas) - Flexible and powerful data analysis / manipulation library for Python, providing labeled data structures similar to R data.frame objects, statistical functions, and much more
- [mwaskom/seaborn](https://github.com/mwaskom/seaborn) - Statistical data visualization in Python
- [yarden/rnaseqlib](https://github.com/yarden/rnaseqlib) - RNA-Seq pipeline
- [olgabot/prettyplotlib](https://github.com/olgabot/prettyplotlib) - Painlessly create beautiful matplotlib plots.
- [adamewing/methylartist](https://github.com/adamewing/methylartist) - Tools for plotting methylation data in various ways
- [lpryszcz/bin](https://github.com/lpryszcz/bin) - My bioinfo toolbox
- [ErillLab/gpu_pssm](https://github.com/ErillLab/gpu_pssm) - A GPU-based implementation of sliding window PSSM scoring.
- [papaemmelab/mergeSVvcf](https://github.com/papaemmelab/mergeSVvcf) - Python package and routines for merging SV VCF files
- [chzh9311/structural-triangulation](https://github.com/chzh9311/structural-triangulation) - The official implementation of Structural Triangulation
- [glennhickey/pg-stuff](https://github.com/glennhickey/pg-stuff) - Helper scripts used to build and debug Cactus pangenomes
- [jnothman/UpSetPlot](https://github.com/jnothman/UpSetPlot) - Draw UpSet plots with Matplotlib
- [tsackton/snpArcher](https://github.com/tsackton/snpArcher) - Snakemake workflow for highly parallel variant calling designed for ease-of-use in non-model organisms.
- [peterjc/picobio](https://github.com/peterjc/picobio) - Miscellaneous Bioinformatics scripts etc mostly in Python
- [liusy-jz/MODAS](https://github.com/liusy-jz/MODAS) - MODAS: Multi-Omics Data Association Study toolkit
- [jhyun95/amr_pangenome](https://github.com/jhyun95/amr_pangenome) - Pan-genome construction/analysis tools centered around PATRIC and antimicrobial resistance
- [Joyvalley/GWAS_Flow](https://github.com/Joyvalley/GWAS_Flow) - GPU accelerated GWAS framework based on TensorFlow
- [matplotlib/matplotlib](https://github.com/matplotlib/matplotlib) - matplotlib: plotting with Python
- [ElderMedic/openHZAU](https://github.com/ElderMedic/openHZAU) - åä¸­å†œä¸šå¤§å­¦è¯¾ç¨‹èµ„æ–™æ•´ç†åˆ†äº«
- [SunPengChuan/wgdi](https://github.com/SunPengChuan/wgdi) - WGDI: A user-friendly toolkit for evolutionary analyses of whole-genome duplications and ancestral karyotypes
- [Gabaldonlab/perSVade](https://github.com/Gabaldonlab/perSVade) - perSVade: personalized Structural Variation detection
- [shawnzhangyx/PePr](https://github.com/shawnzhangyx/PePr) - a peak-calling and differential analysis tool for replicated ChIP-Seq data
- [YichaoOU/HemTools](https://github.com/YichaoOU/HemTools) - HemTools: a collection of NGS pipelines and bioinformatic analyses
- [bioinfo-ibms-pumc/MutPrimerDesign](https://github.com/bioinfo-ibms-pumc/MutPrimerDesign) - MutPrimerDesign: Primer design for human mutations
- [yupenghe/methylpy](https://github.com/yupenghe/methylpy) - WGBS/NOMe-seq Data Processing & Differential Methylation Analysis
- [mulinlab/CAUSALdb-finemapping-pip](https://github.com/mulinlab/CAUSALdb-finemapping-pip) - A GWAS fine-mapping pipeline used in CAUSALdb
- [karel-brinda/svg42pdf](https://github.com/karel-brinda/svg42pdf) - 
- [omicsEye/omeClust](https://github.com/omicsEye/omeClust) - multi-resolution clustering of omics data
- [BIMSBbioinfo/pigx_chipseq](https://github.com/BIMSBbioinfo/pigx_chipseq) - Pipeline for Analysis of ChIP-Seq data
- [PGSB-HMGU/plant.annot](https://github.com/PGSB-HMGU/plant.annot) - Annotation pipeline for plant genomes by PGSB
- [shenlab-sinai/NGS-Data-Charmer](https://github.com/shenlab-sinai/NGS-Data-Charmer) - This repository hosts an automated NGS data analysis (RNA-seq and ChIP-seq) pipeline created using Snakemake.
- [shulp2211/chip-seq-pipeline2](https://github.com/shulp2211/chip-seq-pipeline2) - ENCODE ChIP-seq pipeline
- [Shicheng-Guo/cpgtools](https://github.com/Shicheng-Guo/cpgtools) - Python package to analyze DNA methylation data
- [Shicheng-Guo/deepcpg](https://github.com/Shicheng-Guo/deepcpg) - Deep neural networks for predicting CpG methylation
- [Shicheng-Guo/MethylStar](https://github.com/Shicheng-Guo/MethylStar) - An efficient and flexible pipeline for high-throughput analysis of WGBS data
- [Shicheng-Guo/wgbs-pipeline](https://github.com/Shicheng-Guo/wgbs-pipeline) - ENCODE whole-genome bisulfite sequencing (WGBS) pipeline
- [liguowang/cpgtools](https://github.com/liguowang/cpgtools) - Python package to analyze DNA methylation data
- [liguowang/spiker](https://github.com/liguowang/spiker) - Analysis tool for Spike-in ChIP-seq data
- [liguowang/cobind](https://github.com/liguowang/cobind) - collocation analysis of genomics regions
- [liguowang/CrossMap](https://github.com/liguowang/CrossMap) - CrossMap is a python program to lift over genome coordinates from one genome version to another.
- [Parks-Laboratory/fastlmm_pipeline](https://github.com/Parks-Laboratory/fastlmm_pipeline) - 
- [matted-zz/genome_coverage_plotter](https://github.com/matted-zz/genome_coverage_plotter) - Simple script to generate whole-genome coverage plots
- [jsta/jsta](https://github.com/jsta/jsta) - An automated build system for generating a release and blog list for Github profiles
- [philippbayer/orthomclToVenn](https://github.com/philippbayer/orthomclToVenn) - Creates Venn diagrams for OrthoMCL output
- [zm-git-dev/article-downloader](https://github.com/zm-git-dev/article-downloader) - A web scraper for downloading article metadata (title, journal, authors, year, etc.)
- [zm-git-dev/Biopython-cn](https://github.com/zm-git-dev/Biopython-cn) - Biopython Tutorial Chinese Translation
- [zm-git-dev/rtd](https://github.com/zm-git-dev/rtd) - reference-free ddRADseq analysis tools
- [zm-git-dev/RNA-SeqPipeline](https://github.com/zm-git-dev/RNA-SeqPipeline) - 
- [zm-git-dev/RefDGE_sjm](https://github.com/zm-git-dev/RefDGE_sjm) - 
- [zm-git-dev/TR_generate_v3](https://github.com/zm-git-dev/TR_generate_v3) - 
- [zm-git-dev/TanglabCircularRNAPipeline](https://github.com/zm-git-dev/TanglabCircularRNAPipeline) - 
- [zm-git-dev/snakemake-workflows](https://github.com/zm-git-dev/snakemake-workflows) - An experimental repo for common snakemake rules and workflows
- [zm-git-dev/RNA-Seq-Pipeline](https://github.com/zm-git-dev/RNA-Seq-Pipeline) - RNA-Seq Pipeline written in Python
- [zm-git-dev/RNA](https://github.com/zm-git-dev/RNA) - 
- [zm-git-dev/PyVennerable](https://github.com/zm-git-dev/PyVennerable) - A wrapper for the R library Vennerable for creating Venn and Euler diagrams with up to 9 sets
- [zm-git-dev/vcfstats-1](https://github.com/zm-git-dev/vcfstats-1) - Vcf stats and visualization
- [zm-git-dev/RNASeq_old](https://github.com/zm-git-dev/RNASeq_old) - RNASeq analysis pipeline
- [zm-git-dev/altanalyze](https://github.com/zm-git-dev/altanalyze) - Automatically exported from code.google.com/p/altanalyze
- [zm-git-dev/ddns_port_forwarding](https://github.com/zm-git-dev/ddns_port_forwarding) - è®©å†…ç½‘æœºå™¨ä¸é€šè¿‡ DDNS åŠ¨æ€æŒ‡å®šçš„è¿œç¨‹ä¸»æœºè¿›è¡Œ SSH è¿œç¨‹ç«¯å£è½¬å‘ï¼Œä»¥æ­¤å®ç°å†…ç½‘ç©¿é€
- [zm-git-dev/how-to-write-makefile](https://github.com/zm-git-dev/how-to-write-makefile) - è·Ÿæˆ‘ä¸€èµ·å†™Makefileé‡åˆ¶ç‰ˆ
- [zm-git-dev/pyScripts](https://github.com/zm-git-dev/pyScripts) - Contains misc data specific python3 scripts
- [zm-git-dev/RNA_v2](https://github.com/zm-git-dev/RNA_v2) - A revised pipeline for RNA seq pipeline
- [zm-git-dev/RunCmdsMP](https://github.com/zm-git-dev/RunCmdsMP) - Run system commands parallelly
- [zm-git-dev/svv](https://github.com/zm-git-dev/svv) - Stupid Simple Structural Variant View
- [zm-git-dev/scholar_articles](https://github.com/zm-git-dev/scholar_articles) - 
- [zm-git-dev/PieScripts](https://github.com/zm-git-dev/PieScripts) - This is directory with python script that some might find useful
- [zm-git-dev/nbio_mrnaseq](https://github.com/zm-git-dev/nbio_mrnaseq) - mRNAseq pipeline for differential expression analysis
- [zm-git-dev/pipe_v1.1](https://github.com/zm-git-dev/pipe_v1.1) - pipe and other scripts
- [zm-git-dev/rna-seq-qc](https://github.com/zm-git-dev/rna-seq-qc) - Rna-seq pipeline, From FASTQ to differential expression analysis...
- [zm-git-dev/chip-seq_preprocess](https://github.com/zm-git-dev/chip-seq_preprocess) - A preprocessing pipeline for ChIP-seq, including alignment, quality control, and visualization.
- [zm-git-dev/goatools](https://github.com/zm-git-dev/goatools) - Python scripts to find enrichment of GO terms
- [zm-git-dev/fpkm-calculate-from-bam-or-sam-](https://github.com/zm-git-dev/fpkm-calculate-from-bam-or-sam-) - 
- [zm-git-dev/chipseq-utils](https://github.com/zm-git-dev/chipseq-utils) - Some of my random chipseq analysis scripts
- [zm-git-dev/RNAseq-4](https://github.com/zm-git-dev/RNAseq-4) - RNAseq scrips
- [zm-git-dev/GFFtools-GX](https://github.com/zm-git-dev/GFFtools-GX) - gfftools - Galaxy toolshed repository
- [zm-git-dev/PacBio.VR87.RNAseq.Snakemake](https://github.com/zm-git-dev/PacBio.VR87.RNAseq.Snakemake) - 
- [zm-git-dev/RNAseq-2](https://github.com/zm-git-dev/RNAseq-2) - The Gresham Lab RNAseq pipeline
- [zm-git-dev/HiC-Pro](https://github.com/zm-git-dev/HiC-Pro) - 
- [zm-git-dev/bioinfomatics](https://github.com/zm-git-dev/bioinfomatics) - 
- [zm-git-dev/RNA-seq-analysis](https://github.com/zm-git-dev/RNA-seq-analysis) - RNAseq analysis notes from Tommy Tang
- [zm-git-dev/scripts-1](https://github.com/zm-git-dev/scripts-1) - storing my scripts
- [zm-git-dev/AfterQC](https://github.com/zm-git-dev/AfterQC) - Automatic Filtering, Trimming, Error Removing and Quality Control for fastq data
- [zm-git-dev/LepMAP3](https://github.com/zm-git-dev/LepMAP3) - scripts for a Genetics Map software
- [zm-git-dev/ngs_python](https://github.com/zm-git-dev/ngs_python) - Contains functions for the analysis of ngs data
- [zm-git-dev/CHSLABmiRNApipeline](https://github.com/zm-git-dev/CHSLABmiRNApipeline) - Teach using, miRNA-seq analysis pipeline
- [zm-git-dev/smsk_tuxedo2](https://github.com/zm-git-dev/smsk_tuxedo2) - snakemake files of the tuxedo v2 pipeline from Pertea et al 2016
- [zm-git-dev/igv_snapshotter](https://github.com/zm-git-dev/igv_snapshotter) - Automated IGV snapshot without IGV GUI session
- [zm-git-dev/smallRNApipeline](https://github.com/zm-git-dev/smallRNApipeline) - 
- [zm-git-dev/snakemake-rnaseq-counts](https://github.com/zm-git-dev/snakemake-rnaseq-counts) - snakemake workflow for hisat2 and htseq-count counts generation for use with deseq2 etc
- [zm-git-dev/parallel-fastq-dump](https://github.com/zm-git-dev/parallel-fastq-dump) - parallel fastq-dump wrapper
- [zm-git-dev/circosPrep](https://github.com/zm-git-dev/circosPrep) - scripts to prepare input data files for Circos
- [zm-git-dev/rna-seq-3](https://github.com/zm-git-dev/rna-seq-3) - scripts for rna-seq flow
- [zm-git-dev/omsTools](https://github.com/zm-git-dev/omsTools) - a collection of scripts for NGS (mainly for RNAseq) analysis
- [zm-git-dev/VCF-to-Joinmap](https://github.com/zm-git-dev/VCF-to-Joinmap) - export JoinMap input format (loc) from multiple VCF
- [zm-git-dev/SPA](https://github.com/zm-git-dev/SPA) - 
- [zm-git-dev/new_rna_seq](https://github.com/zm-git-dev/new_rna_seq) - An RNA-seq pipeline for basic differential expression analysis and report generation
- [zm-git-dev/VcfFilterPy](https://github.com/zm-git-dev/VcfFilterPy) - VCF filtering tool
- [zm-git-dev/Pro-seq-Cassava](https://github.com/zm-git-dev/Pro-seq-Cassava) - Pro-seq Cassava
- [zm-git-dev/BSAplot](https://github.com/zm-git-dev/BSAplot) - 
- [zm-git-dev/germline_snakepipe](https://github.com/zm-git-dev/germline_snakepipe) - 
- [zm-git-dev/RNA-Seq-2](https://github.com/zm-git-dev/RNA-Seq-2) - 
- [zm-git-dev/ASEr](https://github.com/zm-git-dev/ASEr) - Get ASE counts from BAMs or raw fastq data -- repackage of pipeline by Carlo Artieri
- [zm-git-dev/STRIDE](https://github.com/zm-git-dev/STRIDE) - STRIDE: Species Tree Root Inference from Gene Duplication Events
- [zm-git-dev/RNA-seq_QC_analysis](https://github.com/zm-git-dev/RNA-seq_QC_analysis) - 
- [zm-git-dev/Snakemake_RNA_Seq](https://github.com/zm-git-dev/Snakemake_RNA_Seq) - Snakemake pipeline for the analysis of RNA_Seq data
- [zm-git-dev/getprimer](https://github.com/zm-git-dev/getprimer) - Python script to design genome specific or homeolog specific primers using Primer3
- [zm-git-dev/Snakemake_salmon](https://github.com/zm-git-dev/Snakemake_salmon) - test snakemake pipeline for RNA_seq data
- [zm-git-dev/rna-seq_pipelines](https://github.com/zm-git-dev/rna-seq_pipelines) - Simple RNA-seq pipelines
- [zm-git-dev/NAL_RNA_seq_annotation_pipeline](https://github.com/zm-git-dev/NAL_RNA_seq_annotation_pipeline) - 
- [zm-git-dev/RNApipeline](https://github.com/zm-git-dev/RNApipeline) - RNAæµç¨‹å¼€å‘
- [zm-git-dev/Create_PDF_report](https://github.com/zm-git-dev/Create_PDF_report) - this is a python script can create latex pdf report for RNAref,RNAdenovo
- [zm-git-dev/GoogleScholarScraper](https://github.com/zm-git-dev/GoogleScholarScraper) - Uses Python scholarly package to scrape relevant articles with given search terms, and then filters by title
- [zm-git-dev/flowcraft](https://github.com/zm-git-dev/flowcraft) - A Nextflow pipeline assembler for genomics :whale::package:
- [zm-git-dev/Snakemake_ChIP_seq_SE](https://github.com/zm-git-dev/Snakemake_ChIP_seq_SE) - Pipeline for the analysis of Single-end ChIP-seq data
- [zm-git-dev/Snakemake_ATAC_seq](https://github.com/zm-git-dev/Snakemake_ATAC_seq) - Pipeline for the analysis of PE ATAC-seq data
- [zm-git-dev/Snakemake_ChIPseq_PE](https://github.com/zm-git-dev/Snakemake_ChIPseq_PE) - Pipeline for the analysis of PE ChIP-seq data
- [zm-git-dev/piret](https://github.com/zm-git-dev/piret) - Pythonic PiReT
- [zm-git-dev/RNA_Pipeline](https://github.com/zm-git-dev/RNA_Pipeline) - RNA_Pipeline from BornToRun
- [zm-git-dev/Dxy](https://github.com/zm-git-dev/Dxy) - Caculate the genetic distance of Dxy(like Fst) between the two populations form the vcf file .
- [zm-git-dev/Molecular_sequence_analyses_scripts](https://github.com/zm-git-dev/Molecular_sequence_analyses_scripts) - 
- [zm-git-dev/Manuals](https://github.com/zm-git-dev/Manuals) - 
- [zm-git-dev/brassica-napus-bulk-segregant](https://github.com/zm-git-dev/brassica-napus-bulk-segregant) - Scripts used to perform bulk segregant analysis in Brassica napus
- [zm-git-dev/arbor-coding-challenge](https://github.com/zm-git-dev/arbor-coding-challenge) - Repository for the Arbor Biotechnologies internship coding challenge.
- [zm-git-dev/arabidopsis](https://github.com/zm-git-dev/arabidopsis) - arabidopsis transcriptomics
- [zm-git-dev/showTree](https://github.com/zm-git-dev/showTree) - showTree can visualize the phylogeny, protein sequences and protein domains of a gene family in one figure.
- [zm-git-dev/SNPbinner](https://github.com/zm-git-dev/SNPbinner) - SNPbinner is a utility for the generation of genotype crossover points and binmaps based on SNP data across recombinant inbred lines.
- [zm-git-dev/msa2snp](https://github.com/zm-git-dev/msa2snp) - SNP and indel calls from multiple sequence alignment (MSA)
- [zm-git-dev/snakemake_CNVkit-RNAseq](https://github.com/zm-git-dev/snakemake_CNVkit-RNAseq) - 
- [zm-git-dev/vcf2joinmap](https://github.com/zm-git-dev/vcf2joinmap) - 
- [zm-git-dev/ngs-tools](https://github.com/zm-git-dev/ngs-tools) - python program for bioinformatics
- [zm-git-dev/tools](https://github.com/zm-git-dev/tools) - 
- [zm-git-dev/My-scripts](https://github.com/zm-git-dev/My-scripts) - 
- [zm-git-dev/BSRPipline](https://github.com/zm-git-dev/BSRPipline) - Analysis Pipline Of Bulked Segregant RNA-Seq
- [zm-git-dev/FaST-LMM](https://github.com/zm-git-dev/FaST-LMM) - Python version of Factored Spectrally Transformed Linear Mixed Models
- [zm-git-dev/ncbi-genome-download](https://github.com/zm-git-dev/ncbi-genome-download) - Scripts to download genomes from the NCBI FTP servers
- [zm-git-dev/jcvi](https://github.com/zm-git-dev/jcvi) - Python utility libraries on genome assembly, annotation and comparative genomics
- [zm-git-dev/CAFE_fig](https://github.com/zm-git-dev/CAFE_fig) - A tool to extract and visualize the results of CAFE (Computational Analysis of gene Family Evolution)
- [zm-git-dev/TElocal](https://github.com/zm-git-dev/TElocal) - A package for quantifying transposable elements at a locus level for RNAseq datasets.
- [zm-git-dev/teaching_book](https://github.com/zm-git-dev/teaching_book) - 
- [zm-git-dev/deg_pipeline](https://github.com/zm-git-dev/deg_pipeline) - Differential gene expression analysis from RNA-seq data
- [zm-git-dev/PAV](https://github.com/zm-git-dev/PAV) - presence - absence variation based on mapping depth.
- [zm-git-dev/ChIP-seq-data-one-step-workflow](https://github.com/zm-git-dev/ChIP-seq-data-one-step-workflow) - A self-built workflow for processing ChIP-seq data.
- [zm-git-dev/phytophthora_fragariae](https://github.com/zm-git-dev/phytophthora_fragariae) - Commands used in the analysis of P. fragariae genomes
- [zm-git-dev/gtfilter](https://github.com/zm-git-dev/gtfilter) - Filters VCF by genotype.
- [zm-git-dev/WASP](https://github.com/zm-git-dev/WASP) - WASP: allele-specific pipeline for unbiased read mapping and molecular QTL discovery
- [zm-git-dev/SRA_Metadata](https://github.com/zm-git-dev/SRA_Metadata) - Get, parse, and extract information from the SRA metadata files
- [zm-git-dev/pyflow-ChIPseq](https://github.com/zm-git-dev/pyflow-ChIPseq) - a snakemake pipeline to process ChIP-seq files from GEO or in-house
- [zm-git-dev/BasicLearning](https://github.com/zm-git-dev/BasicLearning) - 
- [zm-git-dev/TCAG-WGS-CNV-workflow](https://github.com/zm-git-dev/TCAG-WGS-CNV-workflow) - Scripts involved in our workflow for detecting CNVs from WGS data using read depth-based methods
- [zm-git-dev/DAP-seq_Analysis](https://github.com/zm-git-dev/DAP-seq_Analysis) - 
- [zm-git-dev/py37](https://github.com/zm-git-dev/py37) - My python and R's scripts
- [zm-git-dev/ArticleScraper](https://github.com/zm-git-dev/ArticleScraper) - A web scraper based on scrapy to harvest scientific papers from conferences and journal websites.
- [zm-git-dev/pan_genome_scripts](https://github.com/zm-git-dev/pan_genome_scripts) - Python scripts for constructing pangenomes using the iterative mapping style for the Bioinformatics Methods chapter: Making a Pan genome using the iterative mapping approach
- [zm-git-dev/KevinCrawler](https://github.com/zm-git-dev/KevinCrawler) - Kevinå¸¸ç”¨ç½‘ç«™çš„æ•°æ®çˆ¬è™«
- [zm-git-dev/tspex](https://github.com/zm-git-dev/tspex) - tspex: tissue-specificity calculator
- [zm-git-dev/fastqsplitter](https://github.com/zm-git-dev/fastqsplitter) - Splits fastq files evenly
- [zm-git-dev/metaseq](https://github.com/zm-git-dev/metaseq) - Framework for integrated analysis and plotting of ChIP/RIP/RNA/*-seq data
- [zm-git-dev/SNP_Primer_Pipeline2](https://github.com/zm-git-dev/SNP_Primer_Pipeline2) - Design gene specific KASP and CAPS/dCAPS primers for any species
- [zm-git-dev/TREC_Fair_2020](https://github.com/zm-git-dev/TREC_Fair_2020) - Public Version of the Private TREC Repo
- [zm-git-dev/chromatiblock](https://github.com/zm-git-dev/chromatiblock) - Colinear block visualisation tool
- [zm-git-dev/QTL-seq](https://github.com/zm-git-dev/QTL-seq) - QTL-seq pipeline to identify causative mutations responsible for a phenotype
- [zm-git-dev/GenomeFeature](https://github.com/zm-git-dev/GenomeFeature) - 
- [zm-git-dev/Open_Orthobench](https://github.com/zm-git-dev/Open_Orthobench) - 
- [zm-git-dev/Jalview-Sequence-Alignment-Consensus](https://github.com/zm-git-dev/Jalview-Sequence-Alignment-Consensus) - Orders amino acid positions with respect to multiple sequence alignment consensus data
- [zm-git-dev/IGV-snapshot-automator](https://github.com/zm-git-dev/IGV-snapshot-automator) - Script to automatically create and run IGV snapshot batchscripts
- [zm-git-dev/Genome2OR](https://github.com/zm-git-dev/Genome2OR) - Annotate Olfactory receptor CDS from genome
- [zm-git-dev/vcf2db](https://github.com/zm-git-dev/vcf2db) - create a gemini-compatible database from a VCF
- [zm-git-dev/paperInformation](https://github.com/zm-git-dev/paperInformation) - Literature information collection
- [zm-git-dev/genome_info](https://github.com/zm-git-dev/genome_info) - 
- [zm-git-dev/Spider-of-Baidu-Scholars](https://github.com/zm-git-dev/Spider-of-Baidu-Scholars) - 
- [zm-git-dev/fund_data_from_web](https://github.com/zm-git-dev/fund_data_from_web) - 
- [zm-git-dev/r2g](https://github.com/zm-git-dev/r2g) - A homology-based, computationally lightweight pipeline for discovering genes in the absence of an assembly
- [zm-git-dev/primate-X-palindromes](https://github.com/zm-git-dev/primate-X-palindromes) - Code for sequencing and evolutionary analysis of primate X chromosome palindromes
- [zm-git-dev/ChIP-seq-analysis](https://github.com/zm-git-dev/ChIP-seq-analysis) - ChIP-seq analysis notes from Ming Tang
- [zm-git-dev/VCF-kit](https://github.com/zm-git-dev/VCF-kit) - VCF-kit: Assorted utilities for the variant call format
- [zm-git-dev/ChIPpipes](https://github.com/zm-git-dev/ChIPpipes) - Pipelines for ChIP-seq / CUT&RUN / CUT&Tag etc. analysis
- [zm-git-dev/pangenome_graphs](https://github.com/zm-git-dev/pangenome_graphs) - Constructing pan genome graphs from "protein family" assignments
- [zm-git-dev/Allele-Specific-Expression-1](https://github.com/zm-git-dev/Allele-Specific-Expression-1) - 
- [zm-git-dev/article_highlights_scraper](https://github.com/zm-git-dev/article_highlights_scraper) - Create summaries of recently published articles in Cell, Nature and Science journals
- [zm-git-dev/F2_genotype_processing](https://github.com/zm-git-dev/F2_genotype_processing) - python scripts for filtering and binning ddRAD data from an F2 mapping population
- [zm-git-dev/search-sci](https://github.com/zm-git-dev/search-sci) - serch paper whose name in excel from wos and save url and wos number
- [zm-git-dev/jgi-query](https://github.com/zm-git-dev/jgi-query) - A simple command-line tool to download data from Joint Genome Institute databases
- [zm-git-dev/PGV](https://github.com/zm-git-dev/PGV) - Reference-agnostic Representation and Visualization of Pan-genomes
- [zm-git-dev/Variation_pipeline](https://github.com/zm-git-dev/Variation_pipeline) - 
- [zm-git-dev/Pipeline_Allele-specific-expression-ASE](https://github.com/zm-git-dev/Pipeline_Allele-specific-expression-ASE) - 
- [zm-git-dev/LRRprofiler](https://github.com/zm-git-dev/LRRprofiler) - LRRprofiler identified LRR-containing proteins from plant proteomes and annotate their domains and repeats
- [zm-git-dev/plot_genome_align_fig](https://github.com/zm-git-dev/plot_genome_align_fig) - Plot genome alignment figure using progressiveMauve & genoPlotR
- [zm-git-dev/allele-specific-expression-2](https://github.com/zm-git-dev/allele-specific-expression-2) - Allele Specific Expression (ASE) analysis package
- [zm-git-dev/wgd](https://github.com/zm-git-dev/wgd) - Python package and CLI for whole-genome duplication related analyses
- [zm-git-dev/Notes-1](https://github.com/zm-git-dev/Notes-1) - 
- [zm-git-dev/getarticle](https://github.com/zm-git-dev/getarticle) - getarticle is a package based on SciHub and Google Scholar that can download articles given DOI, website address or keywords.
- [zm-git-dev/panGraphViewer](https://github.com/zm-git-dev/panGraphViewer) - Show pangenome graphs in an easy way
- [zm-git-dev/mgv_data](https://github.com/zm-git-dev/mgv_data) - Generates data files for MGV.
- [zm-git-dev/CoolBox](https://github.com/zm-git-dev/CoolBox) - Jupyter notebook based genomic data visualization toolkit.
- [zm-git-dev/GGisy](https://github.com/zm-git-dev/GGisy) - Genome-Genome circle synteny
- [zm-git-dev/python_convert](https://github.com/zm-git-dev/python_convert) - A collection of various utilities for GWAS summary statistics
- [zm-git-dev/SNP_Primer_Pipeline](https://github.com/zm-git-dev/SNP_Primer_Pipeline) - 
- [zm-git-dev/PrimerServer2](https://github.com/zm-git-dev/PrimerServer2) - PrimerServer2: a high-throughput primer design and specificity-checking platform
- [zm-git-dev/VarTools](https://github.com/zm-git-dev/VarTools) - Tools for whole genome sequence analysis.
- [zm-git-dev/wgbs-pipeline](https://github.com/zm-git-dev/wgbs-pipeline) - ENCODE whole-genome bisulfite sequencing (WGBS) pipeline
- [zm-git-dev/Comparative-Annotation-Toolkit](https://github.com/zm-git-dev/Comparative-Annotation-Toolkit) - 
- [zm-git-dev/reciprologs](https://github.com/zm-git-dev/reciprologs) - Find reciprocal best hits (DIAMOND or BLAST) between two (or more!) sets of sequences.
- [zm-git-dev/wrangling-pangenomes](https://github.com/zm-git-dev/wrangling-pangenomes) - Data Wrangling and Processing for pangenomics
- [zm-git-dev/pyGenomeTracks](https://github.com/zm-git-dev/pyGenomeTracks) - python module to plot beautiful and highly customizable genome browser tracks
- [zm-git-dev/GetStarsCode](https://github.com/zm-git-dev/GetStarsCode) - æ‰¹é‡ä¸‹è½½githubç”¨æˆ·starsé¡¹ç›®çš„æºç 
- [zm-git-dev/Ka-Ks](https://github.com/zm-git-dev/Ka-Ks) - æ‰¹é‡è®¡ç®—Ka/Ksæµç¨‹
- [zm-git-dev/vogue](https://github.com/zm-git-dev/vogue) - Vogue: Capture trending data
- [zm-git-dev/zarp](https://github.com/zm-git-dev/zarp) - Zavolan-Lab Automated RNA-Seq Pipeline
- [zm-git-dev/gtracks](https://github.com/zm-git-dev/gtracks) - plot genome track data (for example from a BigWig file)
- [zm-git-dev/pyvenn](https://github.com/zm-git-dev/pyvenn) - 2 ~ 6 sets venn diagram for python
- [zm-git-dev/features_from_snps](https://github.com/zm-git-dev/features_from_snps) - A python script for designed to find any genomic features (genes, CDS, etc) near SNPs or arbitrary coordinates
- [zm-git-dev/zarp-cli](https://github.com/zm-git-dev/zarp-cli) - User-friendly command-line interface for the ZARP RNA-Seq analysis pipeline
- [zm-git-dev/bin-1](https://github.com/zm-git-dev/bin-1) - My bioinfo toolbox
- [zm-git-dev/phaser](https://github.com/zm-git-dev/phaser) - phasing and Allele Specific Expression from RNA-seq
- [zm-git-dev/RNAseq_pipeline-2](https://github.com/zm-git-dev/RNAseq_pipeline-2) - 
- [zm-git-dev/awesome-python](https://github.com/zm-git-dev/awesome-python) - A curated list of awesome Python frameworks, libraries, software and resources
- [zm-git-dev/gget](https://github.com/zm-git-dev/gget) - gget enables efficient querying of genomic databases, such as Ensembl, UniProt, NCBI, directly into a Python or terminal programming environment. It was designed to support genomic data analysis.
- [zm-git-dev/PanACoTA](https://github.com/zm-git-dev/PanACoTA) - PANgenome with Annotations, COre identification, Tree and corresponding Alignments
- [zm-git-dev/PseudogenePipeline](https://github.com/zm-git-dev/PseudogenePipeline) - 
- [zm-git-dev/fastq-dl](https://github.com/zm-git-dev/fastq-dl) - Download FASTQ files from SRA or ENA repositories.
- [zm-git-dev/NGS-Data-Charmer](https://github.com/zm-git-dev/NGS-Data-Charmer) - This repository hosts an automated NGS data analysis (RNA-seq and ChIP-seq) pipeline created using Snakemake.
- [zm-git-dev/transXpress](https://github.com/zm-git-dev/transXpress) - transXpress: a Snakemake pipeline for rapid de novo transcriptome assembly and annotation
- [zm-git-dev/TGG](https://github.com/zm-git-dev/TGG) - tomato graph pangenome
- [zm-git-dev/sra_tools](https://github.com/zm-git-dev/sra_tools) - Download sequence data from the SRA
- [zm-git-dev/DeepRegFinder](https://github.com/zm-git-dev/DeepRegFinder) - DeepRegFinder is a deep learning based program used to identify DNA regulatory elements using ChIP-seq
- [zm-git-dev/CIAlign](https://github.com/zm-git-dev/CIAlign) - 
- [zm-git-dev/seq2science](https://github.com/zm-git-dev/seq2science) - Automated and customizable preprocessing of Next-Generation Sequencing data, including full (sc)ATAC-seq, ChIP-seq, and (sc)RNA-seq workflows. Works equally easy with public as local data.
- [zm-git-dev/sv-callers](https://github.com/zm-git-dev/sv-callers) - Snakemake-based workflow for detecting structural variants in WGS data
- [zm-git-dev/kingfisher-download](https://github.com/zm-git-dev/kingfisher-download) - Easier download/extract of FASTA/Q read data and metadata from the ENA, NCBI, AWS or GCP.
- [zm-git-dev/ChIP-seq-1](https://github.com/zm-git-dev/ChIP-seq-1) - 
- [zm-git-dev/rnawl](https://github.com/zm-git-dev/rnawl) - 
- [zm-git-dev/SparK-1](https://github.com/zm-git-dev/SparK-1) - Publication quality NGS track plotting
- [zm-git-dev/vcfstats](https://github.com/zm-git-dev/vcfstats) - Powerful statistics for VCF files
- [zm-git-dev/PanChIP](https://github.com/zm-git-dev/PanChIP) - Pan-ChIP-seq Analysis of Peak Sets
- [zm-git-dev/ffq](https://github.com/zm-git-dev/ffq) - A tool to find sequencing data and metadata from public databases.
- [zm-git-dev/BayesASE](https://github.com/zm-git-dev/BayesASE) - Bayesian analysis of allele specific expression
- [zm-git-dev/PyPaperBot](https://github.com/zm-git-dev/PyPaperBot) - PyPaperBot is a Python tool for downloading scientific papers using Google Scholar, Crossref, and SciHub.
- [zm-git-dev/blastMining](https://github.com/zm-git-dev/blastMining) - Mining NCBI BLAST output
- [zm-git-dev/pyGenomeViz](https://github.com/zm-git-dev/pyGenomeViz) - A genome visualization python package for comparative genomics
- [zm-git-dev/xTea](https://github.com/zm-git-dev/xTea) - Comprehensive TE insertion identification with WGS/WES data from multiple sequencing technics
- [zm-git-dev/Useful_scripts](https://github.com/zm-git-dev/Useful_scripts) - Some potentially useful scripts
- [zm-git-dev/tensorqtl](https://github.com/zm-git-dev/tensorqtl) - Ultrafast GPU-based QTL mapper
- [zm-git-dev/snakepipes](https://github.com/zm-git-dev/snakepipes) - Customizable workflows based on snakemake and python for the analysis of NGS data
- [zm-git-dev/samplot](https://github.com/zm-git-dev/samplot) - Plot structural variant signals from many BAMs and CRAMs
- [zm-git-dev/fasta_tools](https://github.com/zm-git-dev/fasta_tools) - Some code to play with fasta files
- [zm-git-dev/JournalSpider](https://github.com/zm-git-dev/JournalSpider) - This repository is a spider for some Journal site.
- [zm-git-dev/CNVpytor](https://github.com/zm-git-dev/CNVpytor) - a python extension of CNVnator -- a tool for CNV analysis from depth-of-coverage by mapped reads
- [zm-git-dev/plotsr](https://github.com/zm-git-dev/plotsr) - Tool to plot synteny and structural rearrangements between genomes
- [zm-git-dev/ChIP-AP](https://github.com/zm-git-dev/ChIP-AP) - An Integrated Analysis Pipeline for Unbiased ChIP seq Analysis
- [zm-git-dev/easylift](https://github.com/zm-git-dev/easylift) - convert genomic position for gwas summary
- [zm-git-dev/google_scholar_downloads](https://github.com/zm-git-dev/google_scholar_downloads) - 
- [zm-git-dev/TEtranscripts](https://github.com/zm-git-dev/TEtranscripts) - A package for including transposable elements in differential enrichment analysis of sequencing datasets.
- [zm-git-dev/wgap](https://github.com/zm-git-dev/wgap) - Whole Genome Annotation Pipeline
- [tjiangHIT/cuteSV](https://github.com/tjiangHIT/cuteSV) - Long read based human genomic structural variation detection with cuteSV
- [parklab/xTea](https://github.com/parklab/xTea) - Comprehensive TE insertion identification with WGS/WES data from multiple sequencing technics
- [sormond/gtfilter](https://github.com/sormond/gtfilter) - Filters VCF by genotype.
- [Jome0169/IntronExonFetcher](https://github.com/Jome0169/IntronExonFetcher) - A python script to get intron and exon files from a fasta file with multiple genomic scaffolds as well as
- [jayceejiao/WASP](https://github.com/jayceejiao/WASP) - WASP: allele-specific pipeline for unbiased read mapping and molecular QTL discovery
- [noanabeshima/github-downloader](https://github.com/noanabeshima/github-downloader) - Script for downloading GitHub.
- [guozetang/GithubCrawler](https://github.com/guozetang/GithubCrawler) - 
- [myersjustinc/octostarfish](https://github.com/myersjustinc/octostarfish) - Download a GitHub user's stars
- [zmh1995105/github_spider](https://github.com/zmh1995105/github_spider) - This project will scrape all of repos on github according to the keywords.
- [pinbo/SNP_Primer_Pipeline2](https://github.com/pinbo/SNP_Primer_Pipeline2) - Design gene specific KASP and CAPS/dCAPS primers for any species
- [apcamargo/tspex](https://github.com/apcamargo/tspex) - tspex: tissue-specificity calculator
- [wchathura/utility_code](https://github.com/wchathura/utility_code) - 
- [bensonlew/rnawl](https://github.com/bensonlew/rnawl) - 
- [deeptools/pyGenomeTracks](https://github.com/deeptools/pyGenomeTracks) - python module to plot beautiful and highly customizable genome browser tracks
- [axbazin/drawbank](https://github.com/axbazin/drawbank) - Draws genbank or refseq genome count through time, indicating most common species  and filtering on specific groups
- [pachterlab/ffq](https://github.com/pachterlab/ffq) - A tool to find sequencing data and metadata from public databases.
- [glarue/jgi-query](https://github.com/glarue/jgi-query) - A simple command-line tool to download data from Joint Genome Institute databases
- [SJTU-CGM/TGSRICEPAN](https://github.com/SJTU-CGM/TGSRICEPAN) - The codes of "the Third Generation Sequencing of 111 rice genomes reveals a huge size of novel genome sequence in the RICE PAN-genome"
- [dmiller903/CompoundHetVIP](https://github.com/dmiller903/CompoundHetVIP) - A pipeline for the identification of Compound Heterozygous Variants
- [shenlab-sinai/chip-seq_preprocess](https://github.com/shenlab-sinai/chip-seq_preprocess) - A preprocessing pipeline for ChIP-seq, including alignment, quality control, and visualization.
- [crazyhottommy/ChIP-seq-analysis](https://github.com/crazyhottommy/ChIP-seq-analysis) - ChIP-seq analysis notes from Ming Tang
- [fastlmm/FaST-LMM](https://github.com/fastlmm/FaST-LMM) - Python version of Factored Spectrally Transformed Linear Mixed Models
- [NuttyLogic/BSBolt](https://github.com/NuttyLogic/BSBolt) - BiSulfite Bolt -  A Bisulfite Sequencing Alignment and Processing Tool
- [YuSugihara/QTL-seq](https://github.com/YuSugihara/QTL-seq) - QTL-seq pipeline to identify causative mutations responsible for a phenotype
- [mgijax/mgv_data](https://github.com/mgijax/mgv_data) - Generates data files for MGV.
- [ganlab/GALA](https://github.com/ganlab/GALA) - Long-reads Gap-free Chromosome-scale Assembler
- [zhangmenghan/PyVisualization](https://github.com/zhangmenghan/PyVisualization) - pythonæ•°æ®å¯è§†åŒ–
- [simonorozcoarias/TIP_finder](https://github.com/simonorozcoarias/TIP_finder) - TIP_finder: Transposable element Insertion Polymorphism Finder for huge genomic datasets
- [bvilhjal/mixmogam](https://github.com/bvilhjal/mixmogam) - Mixed Model Package For Genome-wide association mapping.
- [guosongwang/GwasWin](https://github.com/guosongwang/GwasWin) - Sliding window calculation for GWAS results.
- [ShujiaHuang/qmplot](https://github.com/ShujiaHuang/qmplot) - A Python package for creating high-quality manhattan and Q-Q plots from GWAS results.
- [mchone/fastlmm_and_plink_gwas_pipeline](https://github.com/mchone/fastlmm_and_plink_gwas_pipeline) - 
- [craftyKraken/maize-leaf-cuticle](https://github.com/craftyKraken/maize-leaf-cuticle) - Exploring the genetic basis of leaf cuticular evaporation rate in maize
- [vienomics/fastqtools](https://github.com/vienomics/fastqtools) - fastqtools includes fastqClean , fastqSplit ,fastqStat
- [wyim-pgl/vcf-primers](https://github.com/wyim-pgl/vcf-primers) - Generate sequencing primers from a vcf file
- [gturco/repeat_elements](https://github.com/gturco/repeat_elements) - for finding repeat elements and transposons in genome
- [Peims/compare-two-genomes](https://github.com/Peims/compare-two-genomes) - big sequence blast
- [jamiemcg/CompareSequences](https://github.com/jamiemcg/CompareSequences) - Generates plots to compare properties of protein sequences from different FASTA files
- [neyhartj/bioinformatic-utils](https://github.com/neyhartj/bioinformatic-utils) - Script to perform various bioinformatics operations
- [krishnam/vcf2hmp](https://github.com/krishnam/vcf2hmp) - convert VCF to HMP format
- [ijustwanthaveaname/Genefamily_pipeline](https://github.com/ijustwanthaveaname/Genefamily_pipeline) - A pipeline to identify gene family using mutiple tools and scripts.
- [serbier/PedigreeRectificator](https://github.com/serbier/PedigreeRectificator) - A python script to compare the marker ancestry for a given genotype based in his pedigree
- [edgardomortiz/vcf2phylip](https://github.com/edgardomortiz/vcf2phylip) - Convert SNPs in VCF format to PHYLIP, NEXUS, binary NEXUS, or FASTA alignments for phylogenetic analysis
- [bpucker/MYB_annotator](https://github.com/bpucker/MYB_annotator) - This tool performs an automatic identification, annotation, and analysis of the MYB gene family in plants. It can be applied to new transcriptome of genome assemblies.
- [FINNGEN/finemapping-pipeline](https://github.com/FINNGEN/finemapping-pipeline) - Statistical fine-mapping pipeline in FinnGen
- [frankvogt/vcf2gwas](https://github.com/frankvogt/vcf2gwas) - Python API for comprehensive GWAS analysis using GEMMA
- [Marowalker/retrain_w2v](https://github.com/Marowalker/retrain_w2v) - retrain word2vec embedding with data from PMC and PubMed (soon)
- [bergcolette/F2_genotype_processing](https://github.com/bergcolette/F2_genotype_processing) - python scripts for filtering and binning ddRAD data from an F2 mapping population
- [N-damo/wgsreport](https://github.com/N-damo/wgsreport) - 
- [MonashBioinformaticsPlatform/PieScripts](https://github.com/MonashBioinformaticsPlatform/PieScripts) - This is directory with python script that some might find useful
- [SouthGreenPlatform/galaxy-wrappers](https://github.com/SouthGreenPlatform/galaxy-wrappers) - Galaxy tools from South Green
- [ijustwanthaveaname/py37](https://github.com/ijustwanthaveaname/py37) - My Python and R scripts
- [ratschlab/gromics](https://github.com/ratschlab/gromics) - Collection of tools and utilities for *omics analyses
- [panchyni/PseudogenePipeline](https://github.com/panchyni/PseudogenePipeline) - Scripts and wrapper for runing  the Shiu Lab's pseduogene pipeline.
- [chenym1/genetribe](https://github.com/chenym1/genetribe) - A tool for performing collinearity-incorporating homology inference
- [ibest/hitsapipe](https://github.com/ibest/hitsapipe) - High Throughput Sequence Analysis Pipeline
- [ehsansaleh/msaevaluationproject](https://github.com/ehsansaleh/msaevaluationproject) - Multiple Sequence Alignment Methods Evaluation Project
- [DRL/manuscripts](https://github.com/DRL/manuscripts) - 
- [LarsMaas/QTL](https://github.com/LarsMaas/QTL) - School assignment where a genetic linkage map is to be created.
- [manning-lab/fineMap](https://github.com/manning-lab/fineMap) - Pipelines for fine mapping in genetic association analysis
- [thelastleft-back/taobao-tianmao-auto-buy](https://github.com/thelastleft-back/taobao-tianmao-auto-buy) - æ·˜å®/å¤©çŒ«ç§’æ€æŠ¢è´­
- [glarue/reciprologs](https://github.com/glarue/reciprologs) - Find reciprocal best hits (DIAMOND or BLAST) between two (or more!) sets of sequences.
- [snakemake-workflows/rna-seq-star-deseq2](https://github.com/snakemake-workflows/rna-seq-star-deseq2) - RNA-seq workflow using STAR and DESeq2
- [BrooksLabUCSC/flair](https://github.com/BrooksLabUCSC/flair) - Full-Length Alternative Isoform analysis of RNA
- [tanghaibao/jcvi](https://github.com/tanghaibao/jcvi) - Python library to facilitate genome assembly, annotation, and comparative genomics
- [BTomlinson/OmicsTools](https://github.com/BTomlinson/OmicsTools) - Collection of scripts which make handling large data much more efficient. Focus on transcriptomics.
- [LeeBergstrand/BackBLAST_Reciprocal_BLAST](https://github.com/LeeBergstrand/BackBLAST_Reciprocal_BLAST) - This repository contains a reciprocal BLAST program for filtering down BLAST results to best bidirectional hits. It also contains a toolkit for finding and visualizing BLAST hits for gene clusters wit
- [j-andrews7/Pipelines](https://github.com/j-andrews7/Pipelines) - Because backups and revision history are good things.
- [xuzhougeng/myscripts](https://github.com/xuzhougeng/myscripts) - 
- [harvardinformatics/TranscriptomeAssemblyEvaluation](https://github.com/harvardinformatics/TranscriptomeAssemblyEvaluation) - Workflows and scripts used to perform analyses for de novo transcriptome assembly manuscript
- [davidemms/OrthoFinder](https://github.com/davidemms/OrthoFinder) - Phylogenetic orthology inference for comparative genomics
- [ucsc-mus-strain-cactus/comparativeAnnotator](https://github.com/ucsc-mus-strain-cactus/comparativeAnnotator) - 
- [pinbo/SNP_Primer_Pipeline](https://github.com/pinbo/SNP_Primer_Pipeline) - 
- [henryjuho/parus_indel](https://github.com/henryjuho/parus_indel) - Barton and Zeng (2019) - Pipeline for great tit indel analysis.
- [BabyTong/Bioinfomatics-Tutor](https://github.com/BabyTong/Bioinfomatics-Tutor) - bioinfomaticså­¦ä¹ èµ„æ–™
- [zaytoun/scihub.py](https://github.com/zaytoun/scihub.py) - Python API and command-line tool for Sci-Hub
- [liudab/phytozomedownloader](https://github.com/liudab/phytozomedownloader) - A py3 script which is designed to download whole phytozome database with directory arranged.
- [chenjiandongx/mzitu](https://github.com/chenjiandongx/mzitu) - ğŸ‘§ ç¾å¥³å†™çœŸå¥—å›¾çˆ¬è™«ï¼ˆäºŒï¼‰
- [jorvis/biocode](https://github.com/jorvis/biocode) - Bioinformatics code libraries and scripts
- [juanu/CompMicroGenom](https://github.com/juanu/CompMicroGenom) - 
- [dblhlx/PyBSASeq](https://github.com/dblhlx/PyBSASeq) - A novel algorithm with high detection power for BSA-Seq data analysis - the significant structural variant method
- [dblhlx/PyGStatistic](https://github.com/dblhlx/PyGStatistic) - BSA-Seq data analysis - Python implementation of the G-statistic method
- [czheluo/Env-pipeline](https://github.com/czheluo/Env-pipeline) - Bayesian method for Environmental correlations analysis pipeline
- [wlz0726/tools](https://github.com/wlz0726/tools) - 
- [zm-git-dev/BSA](https://github.com/zm-git-dev/BSA) - 
- [testerSunshine/12306](https://github.com/testerSunshine/12306) - 12306æ™ºèƒ½åˆ·ç¥¨ï¼Œè®¢ç¥¨
- [dblhlx/PySNPIndex](https://github.com/dblhlx/PySNPIndex) - BSA-Seq data analysis - Python implementation of the allele frequency method
- [kblin/ncbi-genome-download](https://github.com/kblin/ncbi-genome-download) - Scripts to download genomes from the NCBI FTP servers
- [cmdoret/Acastellanii_genome_analysis](https://github.com/cmdoret/Acastellanii_genome_analysis) - Comparative analysis of the genomes from two Acanthamoeba castellanii strains
- [marbl/MetaCompass](https://github.com/marbl/MetaCompass) - MetaCompass: Reference-guided Assembly of Metagenomes
- [ningchaozky/fpkm-calculate-from-bam-or-sam-](https://github.com/ningchaozky/fpkm-calculate-from-bam-or-sam-) - 
- [vipints/GFFtools-GX](https://github.com/vipints/GFFtools-GX) - gfftools - Galaxy toolshed repository
- [PeterUlz/NPH](https://github.com/PeterUlz/NPH) - Make use of nucleosome protection hypothesis to infer functional aspects of ctDNA
- [wangpanqiao/jianshutopdf](https://github.com/wangpanqiao/jianshutopdf) - å°†ç®€ä¹¦ä½œè€…æ–‡ç« è½¬ä¸ºpdfåˆè¾‘ï¼Œå¹¶ç”Ÿæˆè¯äº‘
- [raisok/snakemakepipe](https://github.com/raisok/snakemakepipe) - 
- [0820LL/TR_generate_v3](https://github.com/0820LL/TR_generate_v3) - 
- [sfyc23/EverydayWechat](https://github.com/sfyc23/EverydayWechat) - å¾®ä¿¡åŠ©æ‰‹ï¼š1.æ¯æ—¥å®šæ—¶ç»™å¥½å‹ï¼ˆå¥³å‹ï¼‰å‘é€å®šåˆ¶æ¶ˆæ¯ã€‚2.æœºå™¨äººè‡ªåŠ¨å›å¤å¥½å‹ã€‚3.ç¾¤åŠ©æ‰‹åŠŸèƒ½ï¼ˆä¾‹å¦‚ï¼šæŸ¥è¯¢åƒåœ¾åˆ†ç±»ã€å¤©æ°”ã€æ—¥å†ã€ç”µå½±å®æ—¶ç¥¨æˆ¿ã€å¿«é€’ç‰©æµã€PM2.5ç­‰ï¼‰
- [bioShaun/qc_report_v1](https://github.com/bioShaun/qc_report_v1) - generate qc report for FastQC results
- [GangCaoLab/CoolBox](https://github.com/GangCaoLab/CoolBox) - Jupyter notebook based genomic data visualization toolkit.
- [jackfrued/Python-100-Days](https://github.com/jackfrued/Python-100-Days) - Python - 100å¤©ä»æ–°æ‰‹åˆ°å¤§å¸ˆ
- [tctianchi/pyvenn](https://github.com/tctianchi/pyvenn) - 2 ~ 6 sets venn diagram for python
- [yanxueqing621/Perl-for-Bioinformatics](https://github.com/yanxueqing621/Perl-for-Bioinformatics) - An attempt to help anyone interested in using Perl for Bioinformatics
- [kentnf/RSQuantification](https://github.com/kentnf/RSQuantification) - Quantification the RNA-seq reads on gene level and perform differentially expression analysis
- [bensonlew/dev_test](https://github.com/bensonlew/dev_test) - dev_test
- [huboqiang/RNA_v2](https://github.com/huboqiang/RNA_v2) - A revised pipeline for RNA seq pipeline
- [danielseaton/merge_featurecounts](https://github.com/danielseaton/merge_featurecounts) - Merge featurecounts outputs across thousands of files.
- [tanghaibao/goatools](https://github.com/tanghaibao/goatools) - Python library to handle Gene Ontology (GO) terms
- [Odinidoer/GO_annotation](https://github.com/Odinidoer/GO_annotation) - need GO.list and go-bases.obo
- [gizemabali/bioinformatics_common_functions_processes](https://github.com/gizemabali/bioinformatics_common_functions_processes) - Find/list Gene Ontology (GO) annotations (molecular function, biological process and cellular component) and pathways for given proteins
- [enormandeau/go_enrichment](https://github.com/enormandeau/go_enrichment) - Transcripts annotation and GO enrichment Fisher tests
- [CFSAN-Biostatistics/snp-pipeline](https://github.com/CFSAN-Biostatistics/snp-pipeline) - SNP Pipeline is a pipeline for the production of SNP matrices from sequence data used in the phylogenetic analysis of pathogenic organisms sequenced from samples of interest to food safety.
- [vinta/awesome-python](https://github.com/vinta/awesome-python) - A curated list of awesome Python frameworks, libraries, software and resources
- [biolab/orange3-bioinformatics](https://github.com/biolab/orange3-bioinformatics) - ğŸŠğŸ”¬ Bioinformatics add-on for Orange3

## R 

- [ilarsf/gwasTools](https://github.com/ilarsf/gwasTools) - Basic and fast GWAS functions for QQ and Manhattan plots (incl. gene names)
- [mgoeker/table2itol](https://github.com/mgoeker/table2itol) - Generating iToL annotations from Spreadsheet or CSV files
- [const-ae/ggsignif](https://github.com/const-ae/ggsignif) - Easily add significance brackets to your ggplots
- [const-ae/ggupset](https://github.com/const-ae/ggupset) - Combination matrix axis for 'ggplot2' to create 'UpSet' plots
- [Zhouqh504/qPCRsum](https://github.com/Zhouqh504/qPCRsum) - qPCRsum is an efficient tool that enables rapid calculation of relative gene expression levels, statistical testing, and visualization.
- [Tuteja-Lab/TissueEnrich](https://github.com/Tuteja-Lab/TissueEnrich) - R package for tissue-specific gene enrichment
- [songliVT/TissueSpecificGenes](https://github.com/songliVT/TissueSpecificGenes) - 
- [barnett874/barplot_shiny](https://github.com/barnett874/barplot_shiny) - 
- [TickingClock1992/Idiogram-plotting](https://github.com/TickingClock1992/Idiogram-plotting) - Plotting chromosome idiogram
- [xuzhougeng/org.At.tair.db](https://github.com/xuzhougeng/org.At.tair.db) - 
- [mdozmorov/scRNA-seq_notes](https://github.com/mdozmorov/scRNA-seq_notes) - A list of scRNA-seq analysis tools
- [mdozmorov/RNA-seq](https://github.com/mdozmorov/RNA-seq) - RNA-seq analysis scripts
- [BioinfGuru/tispec](https://github.com/BioinfGuru/tispec) - Calculates tissue specificity of genes from RNA-seq data
- [jiaxianqing/Ploting_scripts](https://github.com/jiaxianqing/Ploting_scripts) - R scripts for image ploting
- [tyh-19/data-processing](https://github.com/tyh-19/data-processing) - scripts of RNAseq data
- [konumat/Trans-Phar](https://github.com/konumat/Trans-Phar) - GWAS-TWAS (Transcriptome-wide association study)-Pharmacological library integration pipeline
- [wilkox/gggenes](https://github.com/wilkox/gggenes) - â¡ï¸ï¸â¡ï¸ï¸â¬…ï¸ï¸â¡ï¸ï¸ Draw gene arrow maps in ggplot2
- [iDEP-SDSU/idep](https://github.com/iDEP-SDSU/idep) - Integrated Differential Expression and Pathway analysis
- [mskilab-org/JaBbA](https://github.com/mskilab-org/JaBbA) - MIP based joint inference of copy number and rearrangement state in cancer whole genome sequence data.
- [KamilSJaron/smudgeplot](https://github.com/KamilSJaron/smudgeplot) - Inference of ploidy and heterozygosity structure using whole genome sequencing data
- [isinaltinkaya/gptchatteR](https://github.com/isinaltinkaya/gptchatteR) - An experimental and unofficial wrapper for interacting with OpenAI GPT models in R.
- [heroalone/HMM-IBD](https://github.com/heroalone/HMM-IBD) - The HMM script and demo for IBD map construction, used in CUBIC population
- [GrosseLab/OneKP-gene-family-evo](https://github.com/GrosseLab/OneKP-gene-family-evo) - Gene family evolution analysis of OneKP capstone paper
- [GregReeves/Reeves2021_MonocotGrafting](https://github.com/GregReeves/Reeves2021_MonocotGrafting) - Data and code sharing for Reeves et al.  "Monocotyledonous plants graft at the embryonic root-shoot interface" Nature (2022) 602, 280â€“286
- [YaoLab-Bioinfo/shinyCircos-V2.0](https://github.com/YaoLab-Bioinfo/shinyCircos-V2.0) - an R/Shiny application for interactive creation of Circos plot
- [amaiacc/MS-brain-reading-genetics](https://github.com/amaiacc/MS-brain-reading-genetics) - 
- [dorolin/PybridASE](https://github.com/dorolin/PybridASE) - Pipeline to study allele-specific expression in hybrids
- [evodify/allele-specific-expression](https://github.com/evodify/allele-specific-expression) - Pipeline to perform the allele-specific expression analysis (based on the work by Skelly et al. 2011)
- [Jiaxin-Fan/ASEP](https://github.com/Jiaxin-Fan/ASEP) - 
- [pjx1990/as_analysis](https://github.com/pjx1990/as_analysis) - A complete Snakemake pipeline for detecting allele specific expression
- [lixiang117423/OsWRKYbyPan-genome](https://github.com/lixiang117423/OsWRKYbyPan-genome) - 
- [xihaoli/STAARpipeline](https://github.com/xihaoli/STAARpipeline) - An R package for performing association analysis of whole-genome/whole-exome sequencing (WGS/WES) studies using STAARpipeline
- [junjunlab/ClusterGVis](https://github.com/junjunlab/ClusterGVis) - One-step to Cluster and Visualize Gene Expression Matrix
- [enblacar/SCpubr](https://github.com/enblacar/SCpubr) - Generate high quality, publication ready visualizations for single cell transcriptomics data.
- [clemgoub/TE-Aid](https://github.com/clemgoub/TE-Aid) - Annotation helper tool for the manual curation of transposable element consensus sequences
- [isglobal-brge/SNPassoc](https://github.com/isglobal-brge/SNPassoc) - Genetic association studies
- [guokai8/gff2gene](https://github.com/guokai8/gff2gene) - Plot gene structure
- [r-lib/slider](https://github.com/r-lib/slider) - Sliding Window Functions
- [shchurch/hawaiian_drosophilidae_expression_2021](https://github.com/shchurch/hawaiian_drosophilidae_expression_2021) - code for future manuscript on tissue-specific expression evolution in Hawaiian _Drosophila_
- [shanwai1234/GPWAS](https://github.com/shanwai1234/GPWAS) - Genome-Phenome Wide Association Study
- [wbaopaul/scATAC-pro](https://github.com/wbaopaul/scATAC-pro) - A comprehensive tool for processing, analyzing and visulizing single cell chromatin accessibility sequencing data
- [QuangOngCM/LDM_RNAseq_ChIPseq_NSCLC](https://github.com/QuangOngCM/LDM_RNAseq_ChIPseq_NSCLC) - Processed data and scripts to reproduce results and figures in "Integrative RNA-seq and H3 trimethylation ChIP-seq analysis of human lung cancer cells isolated by laser-microdissection" paper.
- [cran/intePareto](https://github.com/cran/intePareto) - :exclamation:Â ThisÂ isÂ aÂ read-onlyÂ mirrorÂ ofÂ theÂ CRANÂ RÂ packageÂ repository.  inteParetoÂ â€”Â IntegrativeÂ AnalysisÂ ofÂ RNA-SeqÂ andÂ ChIP-SeqÂ Data
- [Cherry-Kim/scRNA-seq](https://github.com/Cherry-Kim/scRNA-seq) - 
- [victor2410/ChIPpipe](https://github.com/victor2410/ChIPpipe) - 
- [timedreamer/maize_tissue-specific_GRN](https://github.com/timedreamer/maize_tissue-specific_GRN) - The repository source code for maize tissue-specific GRN project
- [Satoyo08/Arabidopsis_H3K4me1](https://github.com/Satoyo08/Arabidopsis_H3K4me1) - Codes used for 'Transcription-coupled and epigenome-encoded mechanisms direct H3K4 methylation (2022)'
- [ricardoi/LeafSpotArea](https://github.com/ricardoi/LeafSpotArea) - An interface for the image processing program ImageJ, which allows a rapid digital image analysis for particle sizes.
- [taiyun/corrplot](https://github.com/taiyun/corrplot) - A visual exploratory tool on correlation matrix
- [Sherry520/maizeG2P](https://github.com/Sherry520/maizeG2P) - 
- [PoisonAlien/trackplot](https://github.com/PoisonAlien/trackplot) - Generate IGV style locus tracks from bigWig files in R
- [MRCIEU/qqman_multiple_colours](https://github.com/MRCIEU/qqman_multiple_colours) - Manhattan plot function which highlights SNPs in different colours
- [fudiyi/AtRTD2-DE-DAS-DTU-pipeline](https://github.com/fudiyi/AtRTD2-DE-DAS-DTU-pipeline) - 
- [nat2bee/repetitive_elements_pipeline](https://github.com/nat2bee/repetitive_elements_pipeline) - Pipeline to identify repetitive elements on non-model species genomes.
- [tavareshugo/publication_Otero2022_PhloemPoleAtlas](https://github.com/tavareshugo/publication_Otero2022_PhloemPoleAtlas) - analysis code for Otero et al 2020
- [Shamir-Lab/INTEND](https://github.com/Shamir-Lab/INTEND) - INTEND (IntegratioN of Transcriptomic and EpigeNomic Data), a novel algorithm for integrating gene expression and DNA methylation datasets covering disjoint sets of samples
- [bayesrx/SpaceX](https://github.com/bayesrx/SpaceX) - SpaceX: Gene Co-expression Network Estimation for Spatial Transcriptomics
- [JiantaoZhao/MGWAS](https://github.com/JiantaoZhao/MGWAS) - 
- [Sh1ne111/mummer-ggplot](https://github.com/Sh1ne111/mummer-ggplot) - Nucmer from Mummer3 followed by ggplot for nicer and more customisable genome alignment plots.
- [Sh1ne111/NAM-genomes](https://github.com/Sh1ne111/NAM-genomes) - Detailed bioinformatics scripts and methods used in the NAM genome paper.
- [mccoy-lab/t2t-variants](https://github.com/mccoy-lab/t2t-variants) - Code associated with the manuscript "A complete reference genome improves analysis of human genetic variation".
- [laurafahey02/Integrated-RNA-seq-and-ChIP-seq-analysis](https://github.com/laurafahey02/Integrated-RNA-seq-and-ChIP-seq-analysis) - 
- [PeggySze/ChinmoST](https://github.com/PeggySze/ChinmoST) - 
- [sk7-dotcom/Data_Integration_Exercise](https://github.com/sk7-dotcom/Data_Integration_Exercise) - CHIP-seq and RNA-seq integration tutorial
- [Rowena-h/FusariumLifestyles](https://github.com/Rowena-h/FusariumLifestyles) - Scripts for Hill et al. (2022) doi:10.1093/molbev/msac085 ğŸŸ£
- [thomasp85/ggforce](https://github.com/thomasp85/ggforce) - Accelerating ggplot2
- [YaoLab-Bioinfo/LIRBase](https://github.com/YaoLab-Bioinfo/LIRBase) - A comprehensive collection of long inverted repeats in 424 eukaryotic genomes
- [showteeth/ggcoverage](https://github.com/showteeth/ggcoverage) - Visualize and annotate genomic coverage with ggplot2
- [showteeth/ggpie](https://github.com/showteeth/ggpie) - ggpie - A ggplot2 extension to create pie, donut and rose pie plot
- [ChongWuLab/CMO](https://github.com/ChongWuLab/CMO) - Cross Methylome Omnibus (CMO) test
- [cran/RAINBOWR](https://github.com/cran/RAINBOWR) - :exclamation:Â ThisÂ isÂ aÂ read-onlyÂ mirrorÂ ofÂ theÂ CRANÂ RÂ packageÂ repository.  RAINBOWRÂ â€”Â Genome-WideÂ AssociationÂ StudyÂ withÂ SNP-SetÂ Methods
- [hahafengxiang/rice_microbiome](https://github.com/hahafengxiang/rice_microbiome) - 
- [timedreamer/riceTFs](https://github.com/timedreamer/riceTFs) - Analyze rice TF RNA-Seq and ChIP-Seq
- [XiaoLuo-boy/ggheatmap](https://github.com/XiaoLuo-boy/ggheatmap) - ggplot2 version of heatmap
- [JeffWeinell/misc.wrappers](https://github.com/JeffWeinell/misc.wrappers) - Collection of wrappers, pipelines, and some original tools for analysis of population and phylogenetic datasets.
- [myles-lewis/glmmSeq](https://github.com/myles-lewis/glmmSeq) - Gene-level general linear mixed model
- [HajkD/LTRpred](https://github.com/HajkD/LTRpred) - De novo annotation of young retrotransposons
- [BaHole/gwas_scripts](https://github.com/BaHole/gwas_scripts) - Codebook from my GWAS cookbook
- [shenlab-sinai/ngsplot](https://github.com/shenlab-sinai/ngsplot) - Quick mining and visualization of NGS data by integrating genomic databases
- [wum5/solanum-gene-expression](https://github.com/wum5/solanum-gene-expression) - A project to compare evolutionary rate of tissue-specific genes among closely related species
- [SamuelHLewis/TEContent](https://github.com/SamuelHLewis/TEContent) - Calculates the absolute and relative TE content of a genome
- [ucl-medical-genomics/ChAMP](https://github.com/ucl-medical-genomics/ChAMP) - DNA methylation analysis Pipeline
- [Shicheng-Guo/methylation-2](https://github.com/Shicheng-Guo/methylation-2) - All the codes for the DNA methylation pipeline using IDAT files from Illumina Human Methylation 450k and EPIC
- [Shicheng-Guo/DNAm-based-age-predictor](https://github.com/Shicheng-Guo/DNAm-based-age-predictor) - A chronological age predictor based on DNA methylation
- [Shicheng-Guo/MethylResolver](https://github.com/Shicheng-Guo/MethylResolver) - Robust method for deconvolving bulk tissue methylation data using least trimmed squares (LTS) regression
- [Shicheng-Guo/EDec](https://github.com/Shicheng-Guo/EDec) - EDec (Epigenomic Deconvolution) is a technique that, starting from methylation and gene expression profiles of bulk tissue samples, infers cell type composition of each input sample as well as DNA met
- [Shicheng-Guo/lyne](https://github.com/Shicheng-Guo/lyne) - Modeling DNA methylation dynamics with approaches from phylogenetics.
- [Shicheng-Guo/methyLiftover](https://github.com/Shicheng-Guo/methyLiftover) - A utility to map bisulfite sequencing data to the Illumina 450K methylation CpG set
- [Shicheng-Guo/methylKit](https://github.com/Shicheng-Guo/methylKit) - R package for DNA methylation analysis
- [Shicheng-Guo/BBMeth](https://github.com/Shicheng-Guo/BBMeth) - BBMeth: An R package for fitting methylation data with a beta-binomial model
- [Shicheng-Guo/methrix](https://github.com/Shicheng-Guo/methrix) - An R :package: for fast and flexible DNA methylation analysis
- [Shicheng-Guo/mDELLFI](https://github.com/Shicheng-Guo/mDELLFI) - low-pass WGBS based DELFI
- [Shicheng-Guo/low-pass-WGBS](https://github.com/Shicheng-Guo/low-pass-WGBS) - 
- [Shicheng-Guo/HCC_WGBS](https://github.com/Shicheng-Guo/HCC_WGBS) - 
- [yutannihilation/gghighlight](https://github.com/yutannihilation/gghighlight) - Highlight points and lines in ggplot2
- [lixiang117423/qPCRtools](https://github.com/lixiang117423/qPCRtools) - 
- [angelovicilab/HAPPI_GWAS](https://github.com/angelovicilab/HAPPI_GWAS) - A genome-wide association study (GWAS) tool written in R
- [Angelovici-Lab/HAPPI.GWAS](https://github.com/Angelovici-Lab/HAPPI.GWAS) - A genome-wide association study (GWAS) tool written in R
- [YinLiLin/CMplot](https://github.com/YinLiLin/CMplot) - ğŸ“Š Circular and Rectangular Manhattan Plot
- [mohsinali1990/My_scripts](https://github.com/mohsinali1990/My_scripts) - 
- [zm-git-dev/peat-microbiome](https://github.com/zm-git-dev/peat-microbiome) - Record of analysis for peat microbiome manuscript
- [zm-git-dev/rna-seq-diff-exprn](https://github.com/zm-git-dev/rna-seq-diff-exprn) - RNA-Sequencing data differential expression analysis pipeline. Performs: genome coverage (via bedtools and HTSeq), generates Circos code and plots, differential expression (via DESeq and NOISeq), stru
- [zm-git-dev/PrimerDesignPipeline](https://github.com/zm-git-dev/PrimerDesignPipeline) - an R wrapper for ecoPCR, ecoPrimers, etc, by Coissac et al. (see http://metabarcoding.org/obitools/doc/scripts/ecoPrimers.html).
- [zm-git-dev/17_03_RSEM_Pipeline](https://github.com/zm-git-dev/17_03_RSEM_Pipeline) - Pipeline
- [zm-git-dev/RNAseq-workflow](https://github.com/zm-git-dev/RNAseq-workflow) - A repository for setting up a RNAseq workflow
- [zm-git-dev/Gscutility](https://github.com/zm-git-dev/Gscutility) - 
- [zm-git-dev/biostar-tutorials](https://github.com/zm-git-dev/biostar-tutorials) - Code for biostar tutorials - mostly R and Perl
- [zm-git-dev/my_WGCNA](https://github.com/zm-git-dev/my_WGCNA) - 
- [zm-git-dev/WGCNA_shiny](https://github.com/zm-git-dev/WGCNA_shiny) - 
- [zm-git-dev/pipeline_QTL](https://github.com/zm-git-dev/pipeline_QTL) - 
- [zm-git-dev/lqq](https://github.com/zm-git-dev/lqq) - 
- [zm-git-dev/rnaseq-pipeline](https://github.com/zm-git-dev/rnaseq-pipeline) - A custom RNAseq pipeline in R
- [zm-git-dev/rna-seq-snakemake](https://github.com/zm-git-dev/rna-seq-snakemake) - Snakemake based pipeline for RNA-Seq analysis
- [zm-git-dev/RNAseq](https://github.com/zm-git-dev/RNAseq) - Scripts for CompBio-athon RNAseq instructional sessions: QC & Alignment, Differential Gene Expression, BLAST & Annotation, WGCNA
- [zm-git-dev/rnaseq_tutorial](https://github.com/zm-git-dev/rnaseq_tutorial) - Informatics for RNA-seq: A web resource for analysis on the cloud. Educational tutorials and working pipelines for RNA-seq analysis including an introduction to: cloud computing, critical file formats
- [zm-git-dev/wanglab.code](https://github.com/zm-git-dev/wanglab.code) - 
- [zm-git-dev/R178_analysis](https://github.com/zm-git-dev/R178_analysis) - 
- [zm-git-dev/BSA_plot](https://github.com/zm-git-dev/BSA_plot) - 
- [zm-git-dev/maize-GAMER](https://github.com/zm-git-dev/maize-GAMER) - The raw proof-of-concept scripts for maize-GAMER project
- [zm-git-dev/ASE_MBASED](https://github.com/zm-git-dev/ASE_MBASED) - Executable R script for ASE with MBASED
- [zm-git-dev/maize_tissue-specific_GRN](https://github.com/zm-git-dev/maize_tissue-specific_GRN) - The repository source code for maize tissue-specific GRN project
- [zm-git-dev/QTLseqr](https://github.com/zm-git-dev/QTLseqr) - QTLseqr is an R package for QTL mapping using NGS Bulk Segregant Analysis
- [zm-git-dev/manuscript-shared-rnaseq](https://github.com/zm-git-dev/manuscript-shared-rnaseq) - R and Rmarkdown code used across multiple RNA-Seq projects.
- [zm-git-dev/bsa-seq](https://github.com/zm-git-dev/bsa-seq) - The analysis pipeline of BSA-seq. Just for information purposes. Not a fully fledged container
- [zm-git-dev/RNA-seq-Pipline](https://github.com/zm-git-dev/RNA-seq-Pipline) - Typo in name/title. I meant "pipeline".
- [zm-git-dev/CircRNAPipeline](https://github.com/zm-git-dev/CircRNAPipeline) - 
- [zm-git-dev/shared_scripts](https://github.com/zm-git-dev/shared_scripts) - shared scripts in Lu Lab @ Tsinghua University
- [zm-git-dev/igv-snapshot](https://github.com/zm-git-dev/igv-snapshot) - Snapshot IGV with batch file
- [zm-git-dev/UGA](https://github.com/zm-git-dev/UGA) - Code used at UGA for research.
- [zm-git-dev/rnaseq-1](https://github.com/zm-git-dev/rnaseq-1) - Pipeline for RNA-seq scripts used by the Essigmann Lab.
- [zm-git-dev/devnetwork](https://github.com/zm-git-dev/devnetwork) - 
- [zm-git-dev/RNAseqWorkshop2018](https://github.com/zm-git-dev/RNAseqWorkshop2018) - 
- [zm-git-dev/creo_pipelines](https://github.com/zm-git-dev/creo_pipelines) - Bioinformatics Pipelines - CREO - UniPG
- [zm-git-dev/RNAseq_pipeline-1](https://github.com/zm-git-dev/RNAseq_pipeline-1) - 
- [zm-git-dev/RNASeqR_analysis_result](https://github.com/zm-git-dev/RNASeqR_analysis_result) - 
- [zm-git-dev/Enrichment_Analysis](https://github.com/zm-git-dev/Enrichment_Analysis) - GO/KEGG Enrichment Analysis with clusterprofiler
- [zm-git-dev/allele-specific-expression](https://github.com/zm-git-dev/allele-specific-expression) - Pipeline to perform the allele-specific expression analysis (based on the work by Skelly et al. 2011)
- [zm-git-dev/MBSA](https://github.com/zm-git-dev/MBSA) - BSA
- [zm-git-dev/qPCR-Pipeline](https://github.com/zm-git-dev/qPCR-Pipeline) - Auto-analysis for Real-Time PCR (qPCR) data
- [zm-git-dev/Amassing](https://github.com/zm-git-dev/Amassing) - Amassing step by step
- [zm-git-dev/Fareeha_RNA-Seq](https://github.com/zm-git-dev/Fareeha_RNA-Seq) - just store some script for my first data analysis
- [zm-git-dev/Teach-Bioinformatics-R-dataviz](https://github.com/zm-git-dev/Teach-Bioinformatics-R-dataviz) - Materials for "Teaching Bioinformatics and R for dataviz course" june-August 2019  in Majorbio
- [zm-git-dev/pancor](https://github.com/zm-git-dev/pancor) - 
- [zm-git-dev/binmapr](https://github.com/zm-git-dev/binmapr) - :exclamation:Â ThisÂ isÂ aÂ read-onlyÂ mirrorÂ ofÂ theÂ CRANÂ RÂ packageÂ repository.  binmaprÂ â€”Â CallÂ MarkerÂ fromÂ SnpÂ DataÂ usingÂ Binmap.Â Homepage:Â https://github.com/xuzhougeng/binmapr  ReportÂ bugsÂ forÂ thisÂ pack
- [zm-git-dev/genome_plots](https://github.com/zm-git-dev/genome_plots) - Genome wide, SV, CNV, SNV rainfall plots
- [zm-git-dev/Tissue_Specificity](https://github.com/zm-git-dev/Tissue_Specificity) - Identifying tissue-specific genes from spatial-temporal transcriptome profiles
- [zm-git-dev/RNA-SEQ--Raw-reads-to-Counts](https://github.com/zm-git-dev/RNA-SEQ--Raw-reads-to-Counts) - A variety of RNA-Seq pipelines are presented:  Tuxedo for bacteria, Tuxedo for Humans, and HISAT2 for Humans. Additionally, you can find scripts for normalization of RNA-Seq Counts and scripts for vis
- [zm-git-dev/syntenyPlotByR](https://github.com/zm-git-dev/syntenyPlotByR) - plot genome alignment synteny
- [zm-git-dev/R-CMplot](https://github.com/zm-git-dev/R-CMplot) - ğŸ“Š Circular and Rectangular Manhattan Plot
- [zm-git-dev/PybridASE](https://github.com/zm-git-dev/PybridASE) - Pipeline to study allele-specific expression in hybrids
- [zm-git-dev/shiny-ggcorrplot2](https://github.com/zm-git-dev/shiny-ggcorrplot2) - An interactive Shiny App for correlation heatmap plots.
- [zm-git-dev/CisTransRegulation](https://github.com/zm-git-dev/CisTransRegulation) - Project description and analytic scripts for cis/trans regulation of fiber development in G. hirsutum
- [zm-git-dev/GenomeQC](https://github.com/zm-git-dev/GenomeQC) - 
- [zm-git-dev/fulltext](https://github.com/zm-git-dev/fulltext) - Search across and get full text for OA & closed journals
- [zm-git-dev/WGCNA-shinyApp](https://github.com/zm-git-dev/WGCNA-shinyApp) - shiny app for WGCNA...
- [zm-git-dev/deepStats](https://github.com/zm-git-dev/deepStats) - deepStats: a stastitical toolbox for deeptools and genomic signals
- [zm-git-dev/BinMap](https://github.com/zm-git-dev/BinMap) - 
- [zm-git-dev/MethylSnake](https://github.com/zm-git-dev/MethylSnake) - A Snakemake pipeline for RRBS data analysis
- [zm-git-dev/GOVS](https://github.com/zm-git-dev/GOVS) - Genome optimization via virtual simulation
- [zm-git-dev/parallel_rTASSEL](https://github.com/zm-git-dev/parallel_rTASSEL) - Parallelized rTASSEL GLM and MLM using GNU parallel
- [zm-git-dev/stress](https://github.com/zm-git-dev/stress) - maize cold/heat stress experiments
- [zm-git-dev/ngsplot](https://github.com/zm-git-dev/ngsplot) - Quick mining and visualization of NGS data by integrating genomic databases
- [zm-git-dev/atlas](https://github.com/zm-git-dev/atlas) - RNA-Seq and Proteomic Atlas of Maize Development
- [zm-git-dev/gwasglue](https://github.com/zm-git-dev/gwasglue) - Linking GWAS data to analytical tools in R
- [zm-git-dev/CNVMotif](https://github.com/zm-git-dev/CNVMotif) - 
- [zm-git-dev/VCFStat](https://github.com/zm-git-dev/VCFStat) - 
- [zm-git-dev/rMVP](https://github.com/zm-git-dev/rMVP) - :postbox: A Memory-efficient, Visualization-enhanced, and Parallel-accelerated Tool For Genome-Wide Association Study
- [zm-git-dev/qPCRtools](https://github.com/zm-git-dev/qPCRtools) - 
- [bernatgel/karyoploteR](https://github.com/bernatgel/karyoploteR) - karyoploteR - An R/Bioconductor package to plot arbitrary data along the genome
- [jnrunge/SPORE](https://github.com/jnrunge/SPORE) - SPORE (Specific Parent-Offspring Relationship Estimation)
- [xuzhougeng/pancor](https://github.com/xuzhougeng/pancor) - 
- [BartlettLab/LRR-RLK_Evolution](https://github.com/BartlettLab/LRR-RLK_Evolution) - code for LRR-RLK evolution paper
- [cran/KMgene](https://github.com/cran/KMgene) - :exclamation:Â ThisÂ isÂ aÂ read-onlyÂ mirrorÂ ofÂ theÂ CRANÂ RÂ packageÂ repository.  KMgeneÂ â€”Â Gene-BasedÂ AssociationÂ AnalysisÂ forÂ ComplexÂ Traits
- [Lintao1987/Scripts](https://github.com/Lintao1987/Scripts) - 
- [stanikae/Calculate-Haplotype-Frequencies](https://github.com/stanikae/Calculate-Haplotype-Frequencies) - This script calculates allelic and genotypic frequencies, as well as perform Fisher's Exact Test to compare representation of genotypes and alleles in different sample groups
- [tavareshugo/2019-07-01-bioinformatics_for_biologists](https://github.com/tavareshugo/2019-07-01-bioinformatics_for_biologists) - 
- [CellFateNucOrg/MoushumiDas_paper](https://github.com/CellFateNucOrg/MoushumiDas_paper) - 
- [biomarble/PlantNGSTools](https://github.com/biomarble/PlantNGSTools) - https://blog.ugeneyun.cn/software/PlantNGSTools.manual.html
- [JameyZhu/shared_scripts](https://github.com/JameyZhu/shared_scripts) - shared scripts in Lu Lab
- [YaoLab-Bioinfo/binQTL.shiny](https://github.com/YaoLab-Bioinfo/binQTL.shiny) - GUI for binQTL
- [biomarble/onekeyBulkSeq](https://github.com/biomarble/onekeyBulkSeq) - one-key pipeline for QTLseq and Mutmap analysis
- [xuzhougeng/binmapr](https://github.com/xuzhougeng/binmapr) - R package for binmap
- [agrosavia-bioinfo/multiGWAS](https://github.com/agrosavia-bioinfo/multiGWAS) - MultiGWAS: A tool for GWAS analysis on tetraploid organisms by integrating results of four GWAS software
- [mchone/gwas_pipeline_plink_fastlmm](https://github.com/mchone/gwas_pipeline_plink_fastlmm) - ä½¿ç”¨plinkå°†æ ¼å¼åŒ–çš„tpedæ–‡ä»¶å’Œtfamæ–‡ä»¶è½¬åŒ–ä¸ºå¯ä»¥ä½¿ç”¨çš„äºŒè¿›åˆ¶bfilesï¼Œå†ä½¿ç”¨fastlmmåšgwas
- [GoreLab/Maize_leaf_cuticle](https://github.com/GoreLab/Maize_leaf_cuticle) - 
- [Fangv1/Functional_haplotype_GWAS](https://github.com/Fangv1/Functional_haplotype_GWAS) - Selecting closely-linked SNPs based on local epistatic effects for haplotype construction improves power of association mapping
- [ramiromagno/gwasrapidd](https://github.com/ramiromagno/gwasrapidd) - gwasrapidd: an R package to query, download and wrangle GWAS Catalog data
- [shangguandong1996/FindIT2](https://github.com/shangguandong1996/FindIT2) - 
- [amhaaning/Combine_GWAS_Results_from_GAPIT](https://github.com/amhaaning/Combine_GWAS_Results_from_GAPIT) - R script that combines all GWAS results for all traits generated by GAPIT and writes two files, one with all marker info and one with marker info for significantly associated markers only.
- [stemangiola/tidySummarizedExperiment](https://github.com/stemangiola/tidySummarizedExperiment) - Brings SummarizedExperiment to tidiverse with no side-effects
- [whweve/IntAssoPlot](https://github.com/whweve/IntAssoPlot) - 
- [thackl/gggenomes](https://github.com/thackl/gggenomes) - A grammar of graphics for comparative genomics
- [samhuairen/summary_GAPIT](https://github.com/samhuairen/summary_GAPIT) - R functions to summary GAPIT GWAS results
- [lipka-lab/Multi-Trait-GWAS-Methods-Reveal-Loci-Associated-with-Maize-Inflorescence-and-Leaf-Architecture](https://github.com/lipka-lab/Multi-Trait-GWAS-Methods-Reveal-Loci-Associated-with-Maize-Inflorescence-and-Leaf-Architecture) - GWAS Results for Multi-Trait GWAS Methods Reveal Loci Associated with Maize Inflorescence and Leaf Architecture
- [cran/gwaRs](https://github.com/cran/gwaRs) - :exclamation:Â ThisÂ isÂ aÂ read-onlyÂ mirrorÂ ofÂ theÂ CRANÂ RÂ packageÂ repository.  gwaRsÂ â€”Â Manhattan,Â Q-Q,Â andÂ PCAÂ PlotsÂ usingÂ 'ggplot2'.Â Homepage:Â https://github.com/LindoNkambule/gwaRs  ReportÂ bugsÂ forÂ thi
- [wzhy2000/Funmap2](https://github.com/wzhy2000/Funmap2) - QTL mapping tools based on Functional Mapping for the longitudinal quantitative traits
- [WangLab-CEMPS/FindIT2_shiny](https://github.com/WangLab-CEMPS/FindIT2_shiny) - 
- [mmollina/MAPpoly](https://github.com/mmollina/MAPpoly) - Genetic maps in autopolyploids
- [ShawnWx2019/WGCNA-shinyApp](https://github.com/ShawnWx2019/WGCNA-shinyApp) - shiny app for WGCNA...
- [lakhanp1/omics_utils](https://github.com/lakhanp1/omics_utils) - util scripts for various omics data analysis
- [easystats/correlation](https://github.com/easystats/correlation) - :link: Methods for Correlation Analysis
- [YaoLab-Bioinfo/binQTL](https://github.com/YaoLab-Bioinfo/binQTL) - QTL mapping with binmap data
- [ksamuk/samuk_et_al_curr_biol_2020](https://github.com/ksamuk/samuk_et_al_curr_biol_2020) - Code and data for Samuk et al. (2020)
- [UMN-BarleyOatSilphium/GenCorPrediction](https://github.com/UMN-BarleyOatSilphium/GenCorPrediction) - Theoretical and empirical evaluation of predictions of genetic correlations
- [debadutta-patra/Venn-Diagram](https://github.com/debadutta-patra/Venn-Diagram) - Script for R. An automated way of plotting venn diagram (upto 8 sets) from a list of genes in xlsx format
- [vqv/ggbiplot](https://github.com/vqv/ggbiplot) - A biplot based on ggplot2
- [ThinkR-open/golem](https://github.com/ThinkR-open/golem) - A Framework for Building Robust Shiny Apps
- [karoliskoncevicius/CEOplots](https://github.com/karoliskoncevicius/CEOplots) - circadian plots
- [karoliskoncevicius/basetheme](https://github.com/karoliskoncevicius/basetheme) - Themes for base plotting system in R
- [lyy005/Aphid_AL4_chromosome_assembly](https://github.com/lyy005/Aphid_AL4_chromosome_assembly) - The analytical pipeline of the AL4 strain genome
- [j-andrews7/OneLinerOmics](https://github.com/j-andrews7/OneLinerOmics) - One liner analyses and visualizations for bulk RNA-seq and ChIP-seq data
- [sinhrks/ggfortify](https://github.com/sinhrks/ggfortify) - Define fortify and autoplot functions to allow ggplot2 to handle some popular R packages.
- [jennybc/code-smells-and-feels](https://github.com/jennybc/code-smells-and-feels) - Talk on code smells and feels and how to change that via refactoring
- [diazlab/CONICS](https://github.com/diazlab/CONICS) - CONICS: COpy-Number analysis In single-Cell RNA-Sequencing
- [yidishui1/RiA2-master](https://github.com/yidishui1/RiA2-master) - ã€ŠRè¯­è¨€å®æˆ˜ï¼ˆç¬¬2ç‰ˆï¼‰ã€‹
- [magiciiboy/xd-project](https://github.com/magiciiboy/xd-project) - Cross-disciplinary evolution of the genomics revolution
- [mkborregaard/codes-for-published-papers](https://github.com/mkborregaard/codes-for-published-papers) - 
- [bioFAM/MOFA2](https://github.com/bioFAM/MOFA2) - Multi-Omics Factor Analysis
- [Shicheng-Guo/heatmap](https://github.com/Shicheng-Guo/heatmap) - 
- [20170505a/bioinformation_tool_zjl](https://github.com/20170505a/bioinformation_tool_zjl) - Commonly use codes of R or Python for biology information analysis.
- [xiaodaocaoren/bioinformation-python](https://github.com/xiaodaocaoren/bioinformation-python) - 
- [jokergoo/simplifyEnrichment](https://github.com/jokergoo/simplifyEnrichment) - Simplify functional enrichment results
- [KJKwon/Rpackages](https://github.com/KJKwon/Rpackages) - R packages How to
- [idelvalle/scripts](https://github.com/idelvalle/scripts) - All used scripts
- [EmilHvitfeldt/paletteer](https://github.com/EmilHvitfeldt/paletteer) - ğŸ¨ğŸ¨ğŸ¨ Collection of most color palettes in a single R package
- [cran/ggstatsplot](https://github.com/cran/ggstatsplot) - :exclamation:Â ThisÂ isÂ aÂ read-onlyÂ mirrorÂ ofÂ theÂ CRANÂ RÂ packageÂ repository.  ggstatsplotÂ â€”Â 'ggplot2'Â BasedÂ PlotsÂ withÂ StatisticalÂ Details.Â Homepage:Â https://indrajeetpatil.github.io/ggstatsplot/,Â https
- [domjacri/git_staRk](https://github.com/domjacri/git_staRk) - Tutorial for Data Analysis in R (Stark 2019 version)
- [caulilin/QTG_Seq](https://github.com/caulilin/QTG_Seq) - 
- [bmansfeld/QTLseqr](https://github.com/bmansfeld/QTLseqr) - QTLseqr is an R package for QTL mapping using NGS Bulk Segregant Analysis
- [emilyli0325/BSA_lifecycle](https://github.com/emilyli0325/BSA_lifecycle) - BSA lifecycle MKK2835xNHP1337
- [czheluo/MBSA](https://github.com/czheluo/MBSA) - Multple methods for BSA Pipeline
- [czheluo/Teach-Bioinformatics-R-dataviz](https://github.com/czheluo/Teach-Bioinformatics-R-dataviz) - Materials for "Teaching Bioinformatics and R for dataviz course" June-August 2019  in Majorbio
- [fasterius/colours](https://github.com/fasterius/colours) - A KTH-based colour palette and a script to show terminal colours.
- [kevinblighe/PCAtools](https://github.com/kevinblighe/PCAtools) - PCAtools: everything Principal Components Analysis
- [a-velt/Mfuzz_RNAseq](https://github.com/a-velt/Mfuzz_RNAseq) - A R script to perform clustering of gene expression time-series RNA-seq data with Mfuzz.
- [guokai8/wiggleplotr](https://github.com/guokai8/wiggleplotr) - A small R package to make sequencing read coverage plots in R.
- [yanxueqing621/maize-GAMER](https://github.com/yanxueqing621/maize-GAMER) - The raw proof-of-concept scripts for maize-GAMER project
- [yaohuan23/shared_scripts](https://github.com/yaohuan23/shared_scripts) - shared scripts in Lu Lab
- [utnesp/NORAD](https://github.com/utnesp/NORAD) - 
- [shenmengyuan/RNA_seq_Biotrainee](https://github.com/shenmengyuan/RNA_seq_Biotrainee) - DEG analysis and KEGG/GO enrichment analysis
- [twbattaglia/RNAseq-workflow](https://github.com/twbattaglia/RNAseq-workflow) - A repository for setting up a RNAseq workflow

## Roff 

- [comery/ENVer-Workshop-2021](https://github.com/comery/ENVer-Workshop-2021) - ç”Ÿç‰©ä¿¡æ¯å­¦å…¥é—¨çš„ç³»åˆ—è¯¾ç¨‹ï¼ŒåŒ…æ‹¬Linux, shell, Python ç¼–ç¨‹ï¼Œå»ºåº“æµ‹åºï¼ŒåŸºå› ç»„å­¦ï¼Œä»¥åŠç³»ç»Ÿè¿›åŒ–ï¼Œå¸¸ç”¨è½¯ä»¶ä½¿ç”¨ç­‰å†…å®¹ã€‚
- [ucr-hpcc/hpcc_modules](https://github.com/ucr-hpcc/hpcc_modules) - Modules for describing software environment on biocluster
- [lifebit-ai/rnaseq](https://github.com/lifebit-ai/rnaseq) - 

## Ruby 

- [gmonfort/svg2pdf](https://github.com/gmonfort/svg2pdf) - Convert SVG files to other formats, pdf, ps, png, jpeg, etc.
- [ixxmu/mp_duty](https://github.com/ixxmu/mp_duty) - æŠ“å–ç½‘ç»œæ–‡ç« åˆ°github issuesä¿å­˜

## Rust 

- [pangenome/vcfbub](https://github.com/pangenome/vcfbub) - use variant nesting information to flter overlapping sites from vg deconstruct output
- [lencx/ChatGPT](https://github.com/lencx/ChatGPT) - ğŸ”® ChatGPT Desktop Application (Mac, Windows and Linux)
- [lixiang117423/orange](https://github.com/lixiang117423/orange) - Cross-platform file search engine that can quickly locate files or folders based on keywords.
- [jguhlin/minimap2-rs](https://github.com/jguhlin/minimap2-rs) - Rust bindings to minimap2 library
- [brentp/echtvar](https://github.com/brentp/echtvar) - using all the bits for echt rapid variant annotation and filtering
- [typst/svg2pdf](https://github.com/typst/svg2pdf) - Converts SVG files to PDF.
- [veeso/termscp](https://github.com/veeso/termscp) - ğŸ–¥  A feature rich terminal UI file transfer and explorer with support for SCP/SFTP/FTP/S3/SMB
- [tolkit/haplotype-windows](https://github.com/tolkit/haplotype-windows) - Calculate SNP density in sliding windows over a VCF.
- [tolkit/fw_plot](https://github.com/tolkit/fw_plot) - Create simple plots of fasta_windows output.
- [tolkit/vcf2msa](https://github.com/tolkit/vcf2msa) - Given a VCF and a reference genome, create a multiple sequence alignment of the samples.
- [zm-git-dev/gfaestus](https://github.com/zm-git-dev/gfaestus) - GFA visualizer, GPU-accelerated using Vulkan
- [GeneDx/pgr-tk](https://github.com/GeneDx/pgr-tk) - PGR-TK: Pangenome Research Tool Kit
- [pinbo/indel_check](https://github.com/pinbo/indel_check) - Check indels and inversions from sam/bam files with split reads

## Shell 

- [pirovc/genome_updater](https://github.com/pirovc/genome_updater) - Bash script to download/update snapshots of files from NCBI genomes repository (refseq/genbank) with track of changes and without redundancy
- [guyleonard/get_ensembl_genomes](https://github.com/guyleonard/get_ensembl_genomes) - A quick and easy way to download the genomes/predicted proteins of taxa available from ENSEMBL Genomes
- [rnnh/bioinfo-notebook](https://github.com/rnnh/bioinfo-notebook) - ğŸ”¬ Bioinformatics Notebook. Scripts for bioinformatics pipelines, with quick start guides for programs and video demonstrations.
- [jinghuazhao/TWAS-pipeline](https://github.com/jinghuazhao/TWAS-pipeline) - TWAS-pipeline
- [ISUgenomics/common_analyses](https://github.com/ISUgenomics/common_analyses) - Repository of common bioinformatics scripts
- [Tong-Chen/s-plot](https://github.com/Tong-Chen/s-plot) - A simple plot library
- [a-thind/IBDVar](https://github.com/a-thind/IBDVar) - A tool for prioritising identity-by-descent (IBD) variants in Whole Genome Sequencing (WGS) data from families with rare heritable diseases.
- [daskrohn/RBD_GWAS](https://github.com/daskrohn/RBD_GWAS) - Genome-wide association study of REM Sleep Behavior Disorder + downstream analyses.
- [karenbobier/Notochthamalus](https://github.com/karenbobier/Notochthamalus) - 
- [zm-git-dev/ChIPLine](https://github.com/zm-git-dev/ChIPLine) - ChIP-seq analysis pipeline
- [songtaogui/easy_bash](https://github.com/songtaogui/easy_bash) - My common bash functions to make pipelining with bash more smooooth ~
- [ay-lab/ChIPLine](https://github.com/ay-lab/ChIPLine) - ChIP-seq analysis pipeline
- [ThomasArn/genoPlotR_wrapper](https://github.com/ThomasArn/genoPlotR_wrapper) - Wrapper for the genoPlotR R package. This wrapper makes phylogeny, genbank files and blast comparisons that is needed for the genoPLotR package
- [linzhi2013/MitoZ](https://github.com/linzhi2013/MitoZ) - MitoZ: A toolkit for assembly, annotation, and visualization of animal mitochondrial genomes
- [xiaolongliang/Assembly_hifiasm_HiC-anchor_3ddna](https://github.com/xiaolongliang/Assembly_hifiasm_HiC-anchor_3ddna) - 
- [HongboDoll/PotatoPanNLRome](https://github.com/HongboDoll/PotatoPanNLRome) - We report a section-wide pan-NLRome for the tuber-bearing section Petota, including the most important non-cereal food crop potato. Evolutionary genomics and functional analyses discover new late blig
- [VGP/vgp-assembly](https://github.com/VGP/vgp-assembly) - VGP repository for the genome assembly working group
- [shenlab-sinai/ngsplotdb](https://github.com/shenlab-sinai/ngsplotdb) - Genome databases generation pipeline for ngs.plot and region_analysis.
- [spundhir/ChIP-seq](https://github.com/spundhir/ChIP-seq) - My Scripts for the analysis of ChIP-seq data
- [GCA-VH-lab/uorf4u](https://github.com/GCA-VH-lab/uorf4u) - uorf4u is a bioinformatics tool for conserved upstream ORF annotation.
- [bcgsc/rAMPage](https://github.com/bcgsc/rAMPage) - rAMPage: Rapid AMP Annotation and Gene Estimation
- [iranee/qnap-alist-webdav](https://github.com/iranee/qnap-alist-webdav) - ä¸€æ¬¾æŒ‚è½½å¤šä¸ªäº‘ç›˜çš„å·¥å…·
- [sunyumail93/PipeRNAseq](https://github.com/sunyumail93/PipeRNAseq) - A comprehensive pipeline for RNAseq data analysis
- [jacobimarsh/cqProt-003](https://github.com/jacobimarsh/cqProt-003) - 
- [xinkaitong/1k-pig-genomes](https://github.com/xinkaitong/1k-pig-genomes) - Accurate haplotype construction and detection of selection signatures enabled by 889 high quality pig genome sequences
- [Roslin-Aquaculture/GWAS_GCTA](https://github.com/Roslin-Aquaculture/GWAS_GCTA) - 
- [wjwei-handsome/sentieon](https://github.com/wjwei-handsome/sentieon) - extremely fast SNP Calling
- [hsadasivan/mm2-ax](https://github.com/hsadasivan/mm2-ax) - 
- [Maize-Genetics-and-Genomics-Database/Pan-Genome](https://github.com/Maize-Genetics-and-Genomics-Database/Pan-Genome) - Pan-genome methods used by MaizeGDB for the pan-gene tab
- [bioinformatics-ca/HTG_2021](https://github.com/bioinformatics-ca/HTG_2021) - CBWâ€™s High Throughput Genomics Analysis 2021
- [BGI-biotools/HaplotypeCaller-Acc](https://github.com/BGI-biotools/HaplotypeCaller-Acc) - Accelerated GATK4 HaplotypeCaller
- [shannonekj/genome_assembly](https://github.com/shannonekj/genome_assembly) - Pipeline for genome assembly of eukaryotic organisms...hopefully.
- [LingFelce/MLLT1_project](https://github.com/LingFelce/MLLT1_project) - Code relating to MLLT1 in breast cancer project
- [Rowena-h/MCLfilter](https://github.com/Rowena-h/MCLfilter) - A tool to filter MCL output into single-copy orthologous clusters
- [Rowena-h/BUSCOfilter](https://github.com/Rowena-h/BUSCOfilter) - A tool to filter BUSCO output into single-copy gene folders for phylogenomics
- [briannho/Shilatifard-Bioinformatics-Internship](https://github.com/briannho/Shilatifard-Bioinformatics-Internship) - A repository of shell, python, and r scripts to process and analyze ChIP-Seq and RNA-Seq data for the Shilatifard Lab at Northwestern University.
- [hzauMaLAB/SNP-CALLING-PIPLINE](https://github.com/hzauMaLAB/SNP-CALLING-PIPLINE) - This is the initial version created by JYChu
- [hbliu/Kidney_Epi_Pri](https://github.com/hbliu/Kidney_Epi_Pri) - Kidney Epigenome and Transcriptome-based multi-stage Prioritization
- [gotouerina/Colinear_mummer_NGenomeSyn](https://github.com/gotouerina/Colinear_mummer_NGenomeSyn) - Script used for NGenomeSyn and Mummer
- [songtaogui/ZMP_blast2go](https://github.com/songtaogui/ZMP_blast2go) - Zeamap blast2go pipeline
- [teresagr20/ChIP-seq_pipeline](https://github.com/teresagr20/ChIP-seq_pipeline) - CSAR - ChIP-Seq Analysis in aRabidopsis thaliana
- [manyu90/Chip-Seq-pipeline](https://github.com/manyu90/Chip-Seq-pipeline) - Pipeline to process Chip Seq data with additional steps added to remove Chr M and P (in plants)
- [songtaogui/pan-Zea_construct](https://github.com/songtaogui/pan-Zea_construct) - Workflow to construct linear representation of pan genome from deep WGS data and public assemblies
- [Shicheng-Guo/CpG_Me](https://github.com/Shicheng-Guo/CpG_Me) - A whole-genome bisulfite sequencing (WGBS) pipeline for the analysis and QC of DNA methylation that goes from from raw reads to a CpG count matrix
- [shuliliu/cattleGTEx](https://github.com/shuliliu/cattleGTEx) - The cattle Genotype-Tissue Expression atlas v1
- [KevCui/download-github-stars](https://github.com/KevCui/download-github-stars) - :star2: Download Github user's stars information to local md/json file
- [zm-git-dev/rnaSeqPipeline](https://github.com/zm-git-dev/rnaSeqPipeline) - This is a very basic RNA-seq pipeline I use for initial C. elegans miSeq and HiSeq data analysis
- [zm-git-dev/GATK-pipeline](https://github.com/zm-git-dev/GATK-pipeline) - A shell script which implements GATK pipeline for variant calling.
- [zm-git-dev/RNA_analysis](https://github.com/zm-git-dev/RNA_analysis) - 
- [zm-git-dev/GATK](https://github.com/zm-git-dev/GATK) - Variant Discovery through GATK Pipeline
- [zm-git-dev/bpgepr](https://github.com/zm-git-dev/bpgepr) - School project, building a database on RNA sequences
- [zm-git-dev/GATK-Pipeline-1](https://github.com/zm-git-dev/GATK-Pipeline-1) - Genome (DNA-seq) Analysis Toolkit
- [zm-git-dev/hisat2-pipeline](https://github.com/zm-git-dev/hisat2-pipeline) - Pipeline for generating hisat2 reference
- [zm-git-dev/GATK_pipelines](https://github.com/zm-git-dev/GATK_pipelines) - 
- [zm-git-dev/workflows](https://github.com/zm-git-dev/workflows) - HPC workflows for NGS data analysis
- [zm-git-dev/SNP_calling_GATK](https://github.com/zm-git-dev/SNP_calling_GATK) - 
- [zm-git-dev/RNA_Seq_protocol-1](https://github.com/zm-git-dev/RNA_Seq_protocol-1) - RNA-Seq protocol of HISAT2-StringTie-Ballgown
- [zm-git-dev/RNA_Seq_protocol](https://github.com/zm-git-dev/RNA_Seq_protocol) - RNA-Seq protocol of HISAT2-StringTie-Ballgown
- [zm-git-dev/NGS-pipeline](https://github.com/zm-git-dev/NGS-pipeline) - By study this, it won't be costly or  time-consuming to customize a NGS data analysis pipeline
- [zm-git-dev/awk-grep-sed](https://github.com/zm-git-dev/awk-grep-sed) - 
- [zm-git-dev/VCFextractor](https://github.com/zm-git-dev/VCFextractor) - VCFextractor.sh is a unix based software for extracting information from VCF files
- [zm-git-dev/RNASEQ_HISAT2_STRINGTIE](https://github.com/zm-git-dev/RNASEQ_HISAT2_STRINGTIE) - mRNA expression analysis using Hisat2 and Stringtie
- [zm-git-dev/NGS-pipelines](https://github.com/zm-git-dev/NGS-pipelines) - The pipelines for NGS(Next-Generation Sequencing) data processing
- [zm-git-dev/SETD2-project](https://github.com/zm-git-dev/SETD2-project) - SETD2-project related scripts, including perl, R and bash
- [zm-git-dev/Soybean_Scripts](https://github.com/zm-git-dev/Soybean_Scripts) - Various scripts I use to generate NGS data
- [zm-git-dev/RNA-seq](https://github.com/zm-git-dev/RNA-seq) - A shell pipeline for automatically analyzing all RNA-seq data in a folder based on STAR-RSEM and GATK
- [zm-git-dev/pipe_align](https://github.com/zm-git-dev/pipe_align) - pipe_align is a pipeline for Transcriptome assembly, quantification, annotation, differentially expression and quality control analysis based on Ilumina short reads from RNA-seq experiments
- [zm-git-dev/Ext-BSA_pipepline](https://github.com/zm-git-dev/Ext-BSA_pipepline) - The Ext-BSA pipeline for bulked sample analysis (BSA).
- [zm-git-dev/RecipABCD](https://github.com/zm-git-dev/RecipABCD) - Best reciprocal hits for 4 data sets
- [zm-git-dev/Amplicon](https://github.com/zm-git-dev/Amplicon) - High throughput sequencing -- Amplicon sequencing, mainly include 16S, 18S and ITS for microbiome analysis
- [zm-git-dev/collection](https://github.com/zm-git-dev/collection) - 
- [zm-git-dev/Demo_workflow_fastq_to_vcf](https://github.com/zm-git-dev/Demo_workflow_fastq_to_vcf) - Demo Workflow: from Ion torrent fastq files to SNP frequencies
- [zm-git-dev/bioseq](https://github.com/zm-git-dev/bioseq) - bioinformatics
- [zm-git-dev/pipelines_beatolab](https://github.com/zm-git-dev/pipelines_beatolab) - 
- [zm-git-dev/Bioinfo-pipelines](https://github.com/zm-git-dev/Bioinfo-pipelines) - Sample bioinfo script collection.
- [zm-git-dev/NGS-Pacbio](https://github.com/zm-git-dev/NGS-Pacbio) - 
- [zm-git-dev/notes](https://github.com/zm-git-dev/notes) - 
- [zm-git-dev/RNASeq_pipeline](https://github.com/zm-git-dev/RNASeq_pipeline) - 
- [zm-git-dev/DSCI512_RNAseqAnalyzers](https://github.com/zm-git-dev/DSCI512_RNAseqAnalyzers) - A simple set of wrappers and tools for RNA-seq analysis. These tools were designed for the DSCI512 RNA-seq analysis class
- [zm-git-dev/rnaseq-3](https://github.com/zm-git-dev/rnaseq-3) - RNAseq pipeline (HISAT2 + featureCounts + DESeq2)
- [zm-git-dev/molgenis-pipelines](https://github.com/zm-git-dev/molgenis-pipelines) - Pipelines for NGS, imputation, gwas, ...
- [zm-git-dev/bio_tools](https://github.com/zm-git-dev/bio_tools) - Useful bioinformatic scripts
- [zm-git-dev/Mimulus_genomic_landscape](https://github.com/zm-git-dev/Mimulus_genomic_landscape) - 
- [zm-git-dev/AllEnricher](https://github.com/zm-git-dev/AllEnricher) - A comprehensive gene set function enrichment tool for multiple species.
- [zm-git-dev/LRSDAY](https://github.com/zm-git-dev/LRSDAY) - LRSDAY: Long-read Sequencing Data Analysis for Yeasts
- [zm-git-dev/ChIP-seq](https://github.com/zm-git-dev/ChIP-seq) - Scripts to analyze ChIP-seq data
- [zm-git-dev/lab](https://github.com/zm-git-dev/lab) - 
- [zm-git-dev/simu](https://github.com/zm-git-dev/simu) - Fast and Efficient Tool to Simulate Summary Statistics from Genome-Wide Association Studies
- [zm-git-dev/2020STARProtocols_pipeline](https://github.com/zm-git-dev/2020STARProtocols_pipeline) - This is the pipeline of 2020 STAR Protocols
- [zm-git-dev/onestep_RNAseq](https://github.com/zm-git-dev/onestep_RNAseq) - RNAseq pipeline with shell scripts
- [zm-git-dev/progressiveMauve_aligner](https://github.com/zm-git-dev/progressiveMauve_aligner) - aligns query to multiple subjects and visualized as a single figure (work in progress)
- [zm-git-dev/ops_script](https://github.com/zm-git-dev/ops_script) - è¿ç»´ç›¸å…³è„šæœ¬
- [zm-git-dev/pan-Zea_construct](https://github.com/zm-git-dev/pan-Zea_construct) - Workflow to construct linear representation of pan genome from deep WGS data and public assemblies
- [zm-git-dev/genoPlotR_wrapper](https://github.com/zm-git-dev/genoPlotR_wrapper) - Wrapper for the genoPlotR R package. This wrapper makes phylogeny, genbank files and blast comparisons that is needed for the genoPLotR package
- [zm-git-dev/cnv-study-nsphs](https://github.com/zm-git-dev/cnv-study-nsphs) - A pipeline to call CNVs using CNVnator in NSHPHS and perform a CN-GWAS
- [zm-git-dev/Siphonophore_Expression](https://github.com/zm-git-dev/Siphonophore_Expression) - 
- [zm-git-dev/Provannid_host-symbiont_popgen](https://github.com/zm-git-dev/Provannid_host-symbiont_popgen) - 
- [zm-git-dev/GSAbot](https://github.com/zm-git-dev/GSAbot) - Telegram bot which sends alerts when new papers, articles, books, etc. related to your keywords are released on Google Scholar or arXiv
- [zm-git-dev/chenchen](https://github.com/zm-git-dev/chenchen) - molecular breedingï¼›Bioinformatics
- [zm-git-dev/labpipelines](https://github.com/zm-git-dev/labpipelines) - 
- [zm-git-dev/useful-scripts](https://github.com/zm-git-dev/useful-scripts) - common useful script
- [zm-git-dev/SAP_WGS](https://github.com/zm-git-dev/SAP_WGS) - Whole-genome sequencing of the Sorghum Asssociation Panel
- [zm-git-dev/Pipelines-Wrappers](https://github.com/zm-git-dev/Pipelines-Wrappers) - ATACseq, ChIPseq, RNAseq, WGS, PLAR and cisVar pipeline wrappers
- [zm-git-dev/Churros](https://github.com/zm-git-dev/Churros) - 
- [zm-git-dev/BS-seq_Analysis](https://github.com/zm-git-dev/BS-seq_Analysis) - 
- [zm-git-dev/pggb](https://github.com/zm-git-dev/pggb) - the pangenome graph builder
- [malonge/CallSV](https://github.com/malonge/CallSV) - A long-read SV calling pipeline
- [paulphys/github-stars-download](https://github.com/paulphys/github-stars-download) - git clone all of your starred repositories on github
- [Bio-protocol/Gene_Annotation_Pipeline](https://github.com/Bio-protocol/Gene_Annotation_Pipeline) - 
- [grst/rstudio-server-conda](https://github.com/grst/rstudio-server-conda) - Run Rstudio Server in a conda environment
- [ytdai/NKTCL_project](https://github.com/ytdai/NKTCL_project) - Source code for NKTCL project
- [nylander/FastEAR](https://github.com/nylander/FastEAR) - FastEAR - Fast(er) Extraction of Alignment Regions from FASTA
- [MatthewHaas/synteny_plot_tutorial](https://github.com/MatthewHaas/synteny_plot_tutorial) - 
- [taylorpaisie/variant_calling_pipeline](https://github.com/taylorpaisie/variant_calling_pipeline) - My variant calling pipeline for bacteria
- [vanreality/GCPan_Variation_Analysis](https://github.com/vanreality/GCPan_Variation_Analysis) - GCPan genome variants analysis
- [mgijax/mgv_installer](https://github.com/mgijax/mgv_installer) - Installer for mgv and mgv-data in the MGI environment.
- [VistaSohrab/TEfinder](https://github.com/VistaSohrab/TEfinder) - A bioinformatics tool for detecting novel transposable element insertions
- [liencau/eGWAS](https://github.com/liencau/eGWAS) - 
- [lv06025158/InDel-Markers-Development-Platform](https://github.com/lv06025158/InDel-Markers-Development-Platform) - An end-to-end computational solution to identify genome-wide InDel markers under a graphic pipeline environment
- [mcstitzer/maize_v4_TE_annotation](https://github.com/mcstitzer/maize_v4_TE_annotation) - scripts and intermediate files used to annotate TEs in Jiao et al.
- [Baizhenxing-0410/follow-me-install-kubernetes-cluster](https://github.com/Baizhenxing-0410/follow-me-install-kubernetes-cluster) - å’Œæˆ‘ä¸€æ­¥æ­¥éƒ¨ç½² kubernetes é›†ç¾¤
- [rcuadrat/GWAS_pipeline](https://github.com/rcuadrat/GWAS_pipeline) - 
- [wlz0726/Population_Genomics_Scripts](https://github.com/wlz0726/Population_Genomics_Scripts) - Scripts of population genomics analysis
- [tavareshugo/BioPipelines](https://github.com/tavareshugo/BioPipelines) - personal scripts for common bioinformatic pipelines
- [peterthorpe5/Methods_M.cerasi_R.padi_genome_assembly](https://github.com/peterthorpe5/Methods_M.cerasi_R.padi_genome_assembly) - Methods for genome paper
- [dchebotarov/gatk4-var-calling-pipeline](https://github.com/dchebotarov/gatk4-var-calling-pipeline) - A pipeline for variant calling based on best practices for GATK 4.0
- [smirarab/1kp](https://github.com/smirarab/1kp) - Data and scripts used in the oneKP capstone analyses
- [srp33/bsaseq_pipeline_docker](https://github.com/srp33/bsaseq_pipeline_docker) - 
- [drvenki/gatk_basic](https://github.com/drvenki/gatk_basic) - A very simple and basic shell script to run GATK best practice pipeline
- [reedliu/NGS-codes](https://github.com/reedliu/NGS-codes) - This dir deposits common NGS pipeline tested and filtered by ReedLiu
- [erg55/Gen220-Phylo-and-Evol-project-1](https://github.com/erg55/Gen220-Phylo-and-Evol-project-1) - Team Members: Eric Gordon, Kaleigh Russell, Hoang Vuong, David Haisten
- [wk8910/assemble_pipeline](https://github.com/wk8910/assemble_pipeline) - Assemble and following pipeline used in wisent genome paper.
- [genomeannotation/OrthoMCL-docker](https://github.com/genomeannotation/OrthoMCL-docker) - An OrthoMCL setup in a docker image
- [rtmag/vivek](https://github.com/rtmag/vivek) - vivek collab
- [renyiwu/bioseq](https://github.com/renyiwu/bioseq) - bioinformatics
- [mayabrandi/RNA_analysis](https://github.com/mayabrandi/RNA_analysis) - 
- [paulineauffret/Annotation_transcriptome_platax](https://github.com/paulineauffret/Annotation_transcriptome_platax) - 
- [Zhang-lab/WGBS_analysis](https://github.com/Zhang-lab/WGBS_analysis) - whole genome bisulfite sequencing analysis
- [RamsinghLab/WGBS](https://github.com/RamsinghLab/WGBS) - Packages for the WGBS Pipeline, localized and  basespace nativeApp
- [VIB-PSB/PlantRRBS](https://github.com/VIB-PSB/PlantRRBS) - Code base for RRBS and WGBS analyzes in Plants
- [kerenzhou062/SETD2-project](https://github.com/kerenzhou062/SETD2-project) - SETD2-project related scripts, including perl, R and bash
- [zm-git-dev/Test](https://github.com/zm-git-dev/Test) - 
- [dongwei1220/SNPhylo](https://github.com/dongwei1220/SNPhylo) - A pipeline to generate a phylogenetic tree from huge SNP data
- [zm-git-dev/RNASeq_ref_pipe](https://github.com/zm-git-dev/RNASeq_ref_pipe) - Run RNASeq analysis with reference genome based on the Tophat-Cufflinks pipeline
- [giuliospinozzi/creo_pipelines](https://github.com/giuliospinozzi/creo_pipelines) - Bioinformatics Pipelines - CREO - UniPG
- [PLStenger/Pearl_Oyster_Colour_Population_Genomics](https://github.com/PLStenger/Pearl_Oyster_Colour_Population_Genomics) - 
- [caballero/biobuilds](https://github.com/caballero/biobuilds) - Curated collection of open-source bioinformatics tools
- [thlee/SNPhylo](https://github.com/thlee/SNPhylo) - A pipeline to generate a phylogenetic tree from huge SNP data
- [AndersenLab/hisat2-pipeline](https://github.com/AndersenLab/hisat2-pipeline) - Pipeline for generating hisat2 reference

## Smalltalk 

- [hernanmd/BioScripts](https://github.com/hernanmd/BioScripts) - A collection of miscellaneous bioinformatics scripts

## TeX 

- [MCG-NKU/NSFC-LaTex](https://github.com/MCG-NKU/NSFC-LaTex) - 
- [Ruzim/NSFC-application-template-latex](https://github.com/Ruzim/NSFC-application-template-latex) - å›½å®¶è‡ªç„¶ç§‘å­¦åŸºé‡‘ç”³è¯·ä¹¦æ­£æ–‡ï¼ˆé¢ä¸Šé¡¹ç›®ï¼‰LaTeX æ¨¡æ¿ï¼ˆéå®˜æ–¹ï¼‰
- [sjackman/thesis](https://github.com/sjackman/thesis) - PhD Thesis: Efficient Assembly of Large Genomes
- [mszep/pandoc_resume](https://github.com/mszep/pandoc_resume) - The Markdown Resume
- [cellgeni/scRNA.seq.course](https://github.com/cellgeni/scRNA.seq.course) - Analysis of single cell RNA-seq data course
- [HassanLab/rdn-wdp-paper](https://github.com/HassanLab/rdn-wdp-paper) - Repository for RDN-WDP paper
- [grmwld/phd_thesis](https://github.com/grmwld/phd_thesis) - PhD thesis in latex
- [showteeth/Awesome-CV](https://github.com/showteeth/Awesome-CV) - :page_facing_up: Awesome CV is LaTeX template for your outstanding job application
- [eneskemalergin/My_CV](https://github.com/eneskemalergin/My_CV) - Online version of my CV created by Latex
- [zm-git-dev/MethylationAnalysisWorkflow](https://github.com/zm-git-dev/MethylationAnalysisWorkflow) - A cross-package Bioconductor workflow for analysing methylation array data
- [zm-git-dev/dissertation](https://github.com/zm-git-dev/dissertation) - Zhian Kamvar's Ph. D. dissertation from Oregon State University
- [zm-git-dev/RNAsik-pipe](https://github.com/zm-git-dev/RNAsik-pipe) - RNAsik - more than just a pipeline
- [zm-git-dev/graph-genomics-review](https://github.com/zm-git-dev/graph-genomics-review) - Pangenome graphs (review article on graph-based pangenomic methods)
- [zm-git-dev/thuthesis](https://github.com/zm-git-dev/thuthesis) - LaTeX Thesis Template for Tsinghua University
- [zm-git-dev/modern-latex-cv](https://github.com/zm-git-dev/modern-latex-cv) - A professional and modern CV in LaTex
- [jimmejardine/qiqqa-open-source](https://github.com/jimmejardine/qiqqa-open-source) - The open-sourced version of the award-winning Qiqqa research management tool for Windows
- [eturkes/resume](https://github.com/eturkes/resume) - My resume, built using the Awesome CV LaTeX template
- [openbiox/Cookbook-for-R-Chinese](https://github.com/openbiox/Cookbook-for-R-Chinese) - :books:Cookbook for R  ä¸­æ–‡ç¿»è¯‘
- [ellisztamas/fecundity_components](https://github.com/ellisztamas/fecundity_components) - Analysis of fitness components, including seed size and number, in *A. thaliana*
- [vgteam/graph-genomics-review](https://github.com/vgteam/graph-genomics-review) - Pangenome graphs (review article on graph-based pangenomic methods)

## TypeScript 

- [balena-io/etcher](https://github.com/balena-io/etcher) - Flash OS images to SD cards & USB drives, safely and easily.
- [YangJianshun/LINKVIEW2](https://github.com/YangJianshun/LINKVIEW2) - DNAåºåˆ—æ¯”å¯¹ç»“æœ å¯è§†åŒ–å±•ç¤ºå·¥å…·
- [Licoy/ChatGPT-Midjourney](https://github.com/Licoy/ChatGPT-Midjourney) - ğŸ­ ä¸€é”®æ‹¥æœ‰ä½ è‡ªå·±çš„ ChatGPT+Midjourney ç½‘é¡µæœåŠ¡ | Own your own ChatGPT+Midjourney web service with one click
- [Zhouqh504/ChatGPT-Next-Web](https://github.com/Zhouqh504/ChatGPT-Next-Web) - One-Click to deploy well-designed ChatGPT web UI on Vercel. ä¸€é”®æ‹¥æœ‰ä½ è‡ªå·±çš„ ChatGPT ç½‘é¡µæœåŠ¡ã€‚
- [cmdcolin/awesome-genome-visualization](https://github.com/cmdcolin/awesome-genome-visualization) - A list of interesting genome browser or genome-browser-like implementations
- [xiangsx/gpt4free-ts](https://github.com/xiangsx/gpt4free-ts) - Providing a free OpenAI GPT-4 API !   This is a replication project for the typescript version of xtekky/gpt4free
- [ztjhz/BetterChatGPT](https://github.com/ztjhz/BetterChatGPT) - An amazing UI for OpenAI's ChatGPT (Website + Windows + MacOS + Linux)
- [openai-translator/openai-translator](https://github.com/openai-translator/openai-translator) - åŸºäº ChatGPT API çš„åˆ’è¯ç¿»è¯‘æµè§ˆå™¨æ’ä»¶å’Œè·¨å¹³å°æ¡Œé¢ç«¯åº”ç”¨    -    Browser extension and cross-platform desktop application for translation based on ChatGPT API.
- [wong2/chatgpt-google-extension](https://github.com/wong2/chatgpt-google-extension) - This project is deprecated. Check my new project ChatHub:
- [fuergaosi233/wechat-chatgpt](https://github.com/fuergaosi233/wechat-chatgpt) - Use ChatGPT On Wechat via wechaty
- [zm-git-dev/gcv](https://github.com/zm-git-dev/gcv) - Federating genomes with love (and synteny derived from functional annotations)
- [ethanwillis/zotero-scihub](https://github.com/ethanwillis/zotero-scihub) - A plugin that will automatically download PDFs of zotero items from sci-hub

## UnrealScript 

- [zm-git-dev/Microbial-Interkingdom-Suppl](https://github.com/zm-git-dev/Microbial-Interkingdom-Suppl) - Scripts and tables for Duran & Thiergart et al. 2018 (unpublished)

## Vue 

- [SouthGreenPlatform/panache](https://github.com/SouthGreenPlatform/panache) - Panache is a web-based interface designed for the visualization of linearized pangenomes. It can be used to show presence/absence information of pangenomic blocks of sequence or genes in a browser-lik
- [shenshenwu5/resume](https://github.com/shenshenwu5/resume) - :star:æ”¹æ”¹æ•°æ®åˆ›ä½œå¤šæ¬¾ä¸ªäººç®€å†
- [Hacker233/resume-design](https://github.com/Hacker233/resume-design) - ä¸€æ¬¾å¼€æºç®€å†è®¾è®¡ç”Ÿæˆå™¨ï¼Œå†…ç½®ä¸¤æ¬¾è®¾è®¡å™¨ï¼Œå¤šç§å…è´¹æ¨¡æ¿é€‰æ‹©ï¼Œè¿˜å¯ä»¥è‡ªå®šä¹‰æ¨¡æ¿ã€ä¸»é¢˜ç­‰ç­‰ï¼Œæ”¯æŒå¯¼å‡ºPDFã€JSONæ•°æ®ã€‚
- [zm-git-dev/mgv](https://github.com/zm-git-dev/mgv) - Multiple Genome Viewer
- [mgijax/mgv](https://github.com/mgijax/mgv) - Multiple Genome Viewer

## WDL 

- [gatk-workflows/gatk4-rnaseq-germline-snps-indels](https://github.com/gatk-workflows/gatk4-rnaseq-germline-snps-indels) - Workflows for processing RNA data for germline short variant discovery with GATK v4 and related tools
- [gpc-gr/reseq-v3](https://github.com/gpc-gr/reseq-v3) - 

## sed 

- [zm-git-dev/genome_assembly](https://github.com/zm-git-dev/genome_assembly) - Annotation of the Zizania palustris genome


## License

[![CC0](http://mirrors.creativecommons.org/presskit/buttons/88x31/svg/cc-zero.svg)](https://creativecommons.org/publicdomain/zero/1.0/)

To the extent possible under law, [zm-git-dev](https://github.com/zm-git-dev) has waived all copyright and related or neighboring rights to this work.

